<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en-US" xml:lang="en-US">
<head>
  <meta charset="utf-8" />
  <meta name="generator" content="pandoc" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" />
  <meta name="author" content="Julika Radecke" />
  <meta name="author" content="Raphaela Seeger" />
  <meta name="author" content="Anna Kádková" />
  <meta name="author" content="Ulrike Laugks" />
  <meta name="author" content="Amin Khosrozadeh" />
  <meta name="author" content="Kenneth N. Goldie" />
  <meta name="author" content="Vladan Lučić" />
  <meta name="author" content="Jakob B. Sørensen" />
  <meta name="author" content="Benoît Zuber" />
  <meta name="dcterms.date" content="2022-06-14" />
  <meta name="keywords" content="synapse, cryo-electron tomography, SNARE, synaptic vesicles" />
  <title>Morphofunctional changes at the active zone during synaptic vesicle exocytosis</title>
  <style>
    code{white-space: pre-wrap;}
    span.smallcaps{font-variant: small-caps;}
    span.underline{text-decoration: underline;}
    div.column{display: inline-block; vertical-align: top; width: 50%;}
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
    ul.task-list{list-style: none;}
    div.csl-bib-body { }
    div.csl-entry {
      clear: both;
    }
    .hanging div.csl-entry {
      margin-left:2em;
      text-indent:-2em;
    }
    div.csl-left-margin {
      min-width:2em;
      float:left;
    }
    div.csl-right-inline {
      margin-left:2em;
      padding-left:1em;
    }
    div.csl-indent {
      margin-left: 2em;
    }
  </style>
  <!--[if lt IE 9]>
    <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv-printshiv.min.js"></script>
  <![endif]-->
  <!--
  Manubot generated metadata rendered from header-includes-template.html.
  Suggest improvements at https://github.com/manubot/manubot/blob/main/manubot/process/header-includes-template.html
  -->
  <meta name="dc.format" content="text/html" />
  <meta name="dc.title" content="Morphofunctional changes at the active zone during synaptic vesicle exocytosis" />
  <meta name="citation_title" content="Morphofunctional changes at the active zone during synaptic vesicle exocytosis" />
  <meta property="og:title" content="Morphofunctional changes at the active zone during synaptic vesicle exocytosis" />
  <meta property="twitter:title" content="Morphofunctional changes at the active zone during synaptic vesicle exocytosis" />
  <meta name="dc.date" content="2022-06-14" />
  <meta name="citation_publication_date" content="2022-06-14" />
  <meta name="dc.language" content="en-US" />
  <meta name="citation_language" content="en-US" />
  <meta name="dc.relation.ispartof" content="Manubot" />
  <meta name="dc.publisher" content="Manubot" />
  <meta name="citation_journal_title" content="Manubot" />
  <meta name="citation_technical_report_institution" content="Manubot" />
  <meta name="citation_author" content="Julika Radecke" />
  <meta name="citation_author_institution" content="Institute of Anatomy, University of Bern, Bern, Switzerland" />
  <meta name="citation_author_institution" content="Department of Neuroscience, Faculty of Health and Medical Sciences, 2200 Copenhagen N, University of Copenhagen, Copenhagen, Denmark" />
  <meta name="citation_author_institution" content="Diamond Light Source Ltd, Didcot, Oxfordshire, United Kingdom" />
  <meta name="citation_author_orcid" content="0000-0002-5815-5537" />
  <meta name="citation_author" content="Raphaela Seeger" />
  <meta name="citation_author_institution" content="Institute of Anatomy, University of Bern, Bern, Switzerland" />
  <meta name="citation_author_institution" content="Graduate School for Cellular and Biomedical Sciences, University of Bern" />
  <meta name="citation_author" content="Anna Kádková" />
  <meta name="citation_author_institution" content="Department of Neuroscience, University of Copenhagen, Blegdamsvej 3B, 2200 Copenhagen N, Denmark" />
  <meta name="citation_author_orcid" content="0000-0001-6648-9679" />
  <meta name="citation_author" content="Ulrike Laugks" />
  <meta name="citation_author_institution" content="Max-Planck-Institute of Biochemistry, Am Klopferspitz 18, 82152 Martinsried, Germany" />
  <meta name="citation_author_orcid" content="0000-0003-4175-4354" />
  <meta name="citation_author" content="Amin Khosrozadeh" />
  <meta name="citation_author_institution" content="Institute of Anatomy, University of Bern, Bern, Switzerland" />
  <meta name="citation_author_institution" content="Graduate School for Cellular and Biomedical Sciences, University of Bern" />
  <meta name="citation_author" content="Kenneth N. Goldie" />
  <meta name="citation_author_institution" content="BioEM Lab, Biozentrum, University of Basel, Basel, Switzerland" />
  <meta name="citation_author_orcid" content="0000-0002-7405-0049" />
  <meta name="citation_author" content="Vladan Lučić" />
  <meta name="citation_author_institution" content="Max-Planck-Institute of Biochemistry, Am Klopferspitz 18, 82152 Martinsried, Germany" />
  <meta name="citation_author_orcid" content="0000-0003-3698-7436" />
  <meta name="citation_author" content="Jakob B. Sørensen" />
  <meta name="citation_author_institution" content="Department of Neuroscience, University of Copenhagen, Blegdamsvej 3B, 2200 Copenhagen N, Denmark" />
  <meta name="citation_author_orcid" content="0000-0001-5465-3769" />
  <meta name="citation_author" content="Benoît Zuber" />
  <meta name="citation_author_institution" content="Institute of Anatomy, University of Bern, Bern, Switzerland" />
  <meta name="citation_author_orcid" content="0000-0001-7725-5579" />
  <link rel="canonical" href="https://aseedb.github.io/synaptic_tomo_ms/" />
  <meta property="og:url" content="https://aseedb.github.io/synaptic_tomo_ms/" />
  <meta property="twitter:url" content="https://aseedb.github.io/synaptic_tomo_ms/" />
  <meta name="citation_fulltext_html_url" content="https://aseedb.github.io/synaptic_tomo_ms/" />
  <meta name="citation_pdf_url" content="https://aseedb.github.io/synaptic_tomo_ms/manuscript.pdf" />
  <link rel="alternate" type="application/pdf" href="https://aseedb.github.io/synaptic_tomo_ms/manuscript.pdf" />
  <link rel="alternate" type="text/html" href="https://aseedb.github.io/synaptic_tomo_ms/v/ddc223741526b9f6b562dbd671c443bf2c1e3b74/" />
  <meta name="manubot_html_url_versioned" content="https://aseedb.github.io/synaptic_tomo_ms/v/ddc223741526b9f6b562dbd671c443bf2c1e3b74/" />
  <meta name="manubot_pdf_url_versioned" content="https://aseedb.github.io/synaptic_tomo_ms/v/ddc223741526b9f6b562dbd671c443bf2c1e3b74/manuscript.pdf" />
  <meta property="og:type" content="article" />
  <meta property="twitter:card" content="summary_large_image" />
  <meta property="og:image" content="https://github.com/aseedb/synaptic_tomo_ms/raw/ddc223741526b9f6b562dbd671c443bf2c1e3b74/thumbnail.png" />
  <meta property="twitter:image" content="https://github.com/aseedb/synaptic_tomo_ms/raw/ddc223741526b9f6b562dbd671c443bf2c1e3b74/thumbnail.png" />
  <link rel="icon" type="image/png" sizes="192x192" href="https://manubot.org/favicon-192x192.png" />
  <link rel="mask-icon" href="https://manubot.org/safari-pinned-tab.svg" color="#ad1457" />
  <meta name="theme-color" content="#ad1457" />
  <!-- end Manubot generated metadata -->
</head>
<body>
<header id="title-block-header">
<h1 class="title">Morphofunctional changes at the active zone during synaptic vesicle exocytosis</h1>
</header>
<p><small><em>
This manuscript
(<a href="https://aseedb.github.io/synaptic_tomo_ms/v/ddc223741526b9f6b562dbd671c443bf2c1e3b74/">permalink</a>)
was automatically generated
from <a href="https://github.com/aseedb/synaptic_tomo_ms/tree/ddc223741526b9f6b562dbd671c443bf2c1e3b74">aseedb/synaptic_tomo_ms@ddc2237</a>
on June 14, 2022.
</em></small></p>
<h2 id="authors">Authors</h2>
<ul>
<li><p><strong>Julika Radecke</strong><sup><a href="#equal_contribution">*</a></sup><br>
<img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-5815-5537">0000-0002-5815-5537</a>
· <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/julikaradecke">julikaradecke</a><br>
<small>
Institute of Anatomy, University of Bern, Bern, Switzerland; Department of Neuroscience, Faculty of Health and Medical Sciences, 2200 Copenhagen N, University of Copenhagen, Copenhagen, Denmark; Diamond Light Source Ltd, Didcot, Oxfordshire, United Kingdom
· Funded by Swiss National Science Foundation, P2BEP3_172242
</small></p></li>
<li><p><strong>Raphaela Seeger</strong><sup><a href="#equal_contribution">*</a></sup><br>
· <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/elatella">elatella</a><br>
<small>
Institute of Anatomy, University of Bern, Bern, Switzerland; Graduate School for Cellular and Biomedical Sciences, University of Bern
</small></p></li>
<li><p><strong>Anna Kádková</strong><br>
<img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-6648-9679">0000-0001-6648-9679</a><br>
<small>
Department of Neuroscience, University of Copenhagen, Blegdamsvej 3B, 2200 Copenhagen N, Denmark
</small></p></li>
<li><p><strong>Ulrike Laugks</strong><br>
<img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0003-4175-4354">0000-0003-4175-4354</a><br>
<small>
Max-Planck-Institute of Biochemistry, Am Klopferspitz 18, 82152 Martinsried, Germany
</small></p></li>
<li><p><strong>Amin Khosrozadeh</strong><br>
· <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/ameen-khosrowzadeh">ameen-khosrowzadeh</a><br>
<small>
Institute of Anatomy, University of Bern, Bern, Switzerland; Graduate School for Cellular and Biomedical Sciences, University of Bern
</small></p></li>
<li><p><strong>Kenneth N. Goldie</strong><br>
<img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-7405-0049">0000-0002-7405-0049</a><br>
<small>
BioEM Lab, Biozentrum, University of Basel, Basel, Switzerland
</small></p></li>
<li><p><strong>Vladan Lučić</strong><br>
<img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0003-3698-7436">0000-0003-3698-7436</a>
· <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/vladanl">vladanl</a><br>
<small>
Max-Planck-Institute of Biochemistry, Am Klopferspitz 18, 82152 Martinsried, Germany
</small></p></li>
<li><p><strong>Jakob B. Sørensen</strong><sup><a href="#correspondence">✉</a></sup><br>
<img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-5465-3769">0000-0001-5465-3769</a>
· <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/JBSorensen">JBSorensen</a><br>
<small>
Department of Neuroscience, University of Copenhagen, Blegdamsvej 3B, 2200 Copenhagen N, Denmark
· Funded by Novo Nordisk Fonden, NNF17OC0028516.; Carlsbergfondet, CF17-0875; Independent Research Fond Denmark, 8020-00228A; Lundbeckfonden, R277-2018-802
</small></p></li>
<li><p><strong>Benoît Zuber</strong><sup><a href="#correspondence">✉</a></sup><br>
<img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-7725-5579">0000-0001-7725-5579</a>
· <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/aseedb">aseedb</a><br>
<small>
Institute of Anatomy, University of Bern, Bern, Switzerland
· Funded by Swiss National Science Foundation, 179520; ERA-NET NEURON, NEURON-119
</small></p></li>
</ul>
<div id="correspondence">
<p>✉ Address correspondence to <a href="mailto:benoit.zuber@ana.unibe.ch" class="email">benoit.zuber@ana.unibe.ch</a> and <a href="mailto:jakobbs@sund.ku.dk" class="email">jakobbs@sund.ku.dk</a>.</p>
</div>
<div id="equal_contribution">
<p>* These authors contributed equally.</p>
</div>
<h2 class="page_break_before" id="abstract">Abstract</h2>
<p>The fusion of synaptic vesicles (SVs) with the plasma membrane (PM) proceeds through intermediate steps that remain poorly resolved.
Additionally, the effect of persistent high or low exocytosis activity on intermediate steps remains unknown.
Through time-resolved cryo-electron tomography, we ordered events into a sequence.
Following stimulation, proximal tethered SVs rapidly form additional tethers with the PM.
Simultaneously, fusion initiation occurs by membrane curvature (‘buckling’) of the SV and PM.
It is followed by the formation of a fusion pore, and the collapse of SV membrane.
At this time, membrane-proximal, but not membrane-distal, vesicles lose their interconnections, allowing them to move towards the PM.
Two mutations of SNAP-25 that arrests or disinhibit spontaneous release, respectively, both caused a loss of interconnectors, while the disinhibiting mutant also caused a loss of membrane proximal multiple-tethered SVs.
Overall, tether formation and connector dissolution is triggered by stimulation and respond to the spontaneous fusion rate.
These morphological observations likely correspond to the transition of SVs from one functional pool to another.</p>
<h2 id="introduction">Introduction</h2>
<p>For normal brain function such as movement coordination or memory formation, communication between neurons is essential.
In the central nervous system, neurons communicate through the release of neurotransmitters at synapses.
This process relies on synaptic vesicle (SV) exocytosis, i.e. the fusion of neurotransmitter-filled SVs with the plasma membrane (PM).
SV exocytosis involves a sequence of steps <span class="citation" data-cites="qCdQVB7d 6kHFuPDC">[<a href="#ref-qCdQVB7d" role="doc-biblioref">1</a>,<a href="#ref-6kHFuPDC" role="doc-biblioref">2</a>]</span>.
The vesicle is docked to the active zone (AZ) PM and the exocytosis machinery goes through a maturation process, termed priming, after which the SV is ready to fuse.
These SVs form the readily releasable pool (RRP).
Finally, a calcium influx triggers fusion of the SV with the PM.
Docked SVs are defined as those in very close proximity or direct contact with the PM as observed by electron microscopy (EM), whereas priming refers to SV ability to undergo exocytosis immediately upon stimulation.
Whether every docked SV is also primed has been debated <span class="citation" data-cites="qCdQVB7d ly3VumnN f0VSTJjV">[<a href="#ref-qCdQVB7d" role="doc-biblioref">1</a>,<a href="#ref-ly3VumnN" role="doc-biblioref">3</a>,<a href="#ref-f0VSTJjV" role="doc-biblioref">4</a>]</span>.
A high-pressure freezing/freeze-substitution EM study of synapses has indicated that vesicles which are in direct contact with the PM, i.e. docked, are also primed and belong to the RRP and that this situation occurs downstream of vesicle tethering <span class="citation" data-cites="f0VSTJjV">[<a href="#ref-f0VSTJjV" role="doc-biblioref">4</a>]</span>.
From a molecular perspective, priming involves several proteins, including the SNARE complex (SNAP-25, syntaxin-1, and synaptobrevin-2), Munc13, Munc18, synaptotagmin-1, and complexin <span class="citation" data-cites="6kHFuPDC A7bPuSTw">[<a href="#ref-6kHFuPDC" role="doc-biblioref">2</a>,<a href="#ref-A7bPuSTw" role="doc-biblioref">5</a>]</span>.
All three SNAREs form a highly stable tight four-helix bundle, known as trans-SNARE complex.
The surfaces of the SV and the PM, are both negatively charged and therefore tend to repulse each other.
The formation of the trans-SNARE complex counteracts this repulsion and brings the SV and the PM in high proximity <span class="citation" data-cites="qdaYzBLt">[<a href="#ref-qdaYzBLt" role="doc-biblioref">6</a>]</span>.
Evidence has suggested that the SNARE complex is only partially zipped in primed vesicles <span class="citation" data-cites="2XC5tAoG">[<a href="#ref-2XC5tAoG" role="doc-biblioref">7</a>]</span>.
Furthermore, various studies have suggested that the formation of at least three SNARE complexes provides the necessary energy for a vesicle to become fusion-competent <span class="citation" data-cites="1ErVrgCCc yI8145n3 12xIhQNRh">[<a href="#ref-1ErVrgCCc" role="doc-biblioref">8</a>,<a href="#ref-yI8145n3" role="doc-biblioref">9</a>,<a href="#ref-12xIhQNRh" role="doc-biblioref">10</a>]</span>.
Yet in the absence of cytoplasmic Ca<sup>2+</sup>, minimal spontaneous exocytosis takes place.
When the presynaptic terminal gets depolarized by an action potential, Ca<sup>2+</sup> flows in the cytoplasm and binds to synaptotagmin-1, which is localized at the SV surface.
Upon Ca<sup>2+</sup> binding, synaptotagmin-1 was proposed to insert between the head groups of the PM anionic phospholipids and trigger membrane curvature and destabilization, leading first to hemifusion and subsequently to fusion <span class="citation" data-cites="qeT2GTd8">[<a href="#ref-qeT2GTd8" role="doc-biblioref">11</a>]</span>.
Interestingly, much of the trans-SNARE bundle surface is negatively charged.
This contributes to the electrostatic barrier that minimizes spontaneous fusion.
Synaptotagmin-1 can then act as an electrostatic switch that triggers exocytosis.
Introducing negatively charged side chains by site-directed mutagenesis reduces the rate of spontaneous and evoked exocytosis, whereas introducing more positive side chains enhances the rate of spontaneous exocytosis and depletes the RRP. <span class="citation" data-cites="AVAZzvH0">[<a href="#ref-AVAZzvH0" role="doc-biblioref">12</a>]</span></p>
<p>Cryo-electron tomography (cryo-ET), which preserves samples to atomic resolution, revealed that under resting conditions, no SV is in direct contact with the PM and the majority of AZ-proximal SVs are connected to the PM by a variable number of short tethers <span class="citation" data-cites="XQJ3R1HJ K10hpz3n">[<a href="#ref-XQJ3R1HJ" role="doc-biblioref">13</a>,<a href="#ref-K10hpz3n" role="doc-biblioref">14</a>]</span>.
The observed gap between the SV and the PM is consistent with the model of an electrostatic barrier formed by the negative charges of the SV, the PM, and the trans-SNARE bundle <span class="citation" data-cites="AVAZzvH0">[<a href="#ref-AVAZzvH0" role="doc-biblioref">12</a>]</span>.
In synaptosomes treated with hypertonic-sucrose solution, which depletes the RRP, the majority of tethered vesicles had only 1 or 2 tethers <span class="citation" data-cites="4Xeb8inT UZqYppzb XQJ3R1HJ">[<a href="#ref-XQJ3R1HJ" role="doc-biblioref">13</a>,<a href="#ref-4Xeb8inT" role="doc-biblioref">15</a>,<a href="#ref-UZqYppzb" role="doc-biblioref">16</a>]</span>.
This observation suggested that the RRP consists of SV, which are linked to the PM by 3 or more tethers.
The RRP, as identified by morphological criteria, only represents a minority of AZ-proximal vesicles.
This is in agreement with previous reports.
In one of them, the term pre-primed pool was used for the few vesicles (~1 vesicle at hippocampal synapses) that are rapidly released and another publication showed that the RRP is made up of only 10-20% of SVs located on the AZ (equal to ~1 vesicle on hippocampal synapses) <span class="citation" data-cites="QkLxmBRs ff66mwER">[<a href="#ref-QkLxmBRs" role="doc-biblioref">17</a>,<a href="#ref-ff66mwER" role="doc-biblioref">18</a>]</span>.
The ensemble of proximal vesicles that are not in the RRP have been termed non-RRP and presumably belong to the recycling pool that releases more slowly <span class="citation" data-cites="XQJ3R1HJ XzTcFx4M">[<a href="#ref-XQJ3R1HJ" role="doc-biblioref">13</a>,<a href="#ref-XzTcFx4M" role="doc-biblioref">19</a>]</span>.
Farther away from the AZ, partially intermixed with the recycling pool, is the reserve pool containing vesicles that only release upon high frequency stimulation.
Vesicles in the reserve pool are tightly clustered and well inter-connected by structures that were termed connectors <span class="citation" data-cites="XQJ3R1HJ XzTcFx4M">[<a href="#ref-XQJ3R1HJ" role="doc-biblioref">13</a>,<a href="#ref-XzTcFx4M" role="doc-biblioref">19</a>]</span>.
It should be noted, that the molecular nature of connectors is not known and is possibly heterogenous.
Synapsin has been proposed as a molecular constituent.
However, since the deletion of all forms of synapsin does not lead to the complete absence of connectors, it is clear that not all connectors contain synapsin <span class="citation" data-cites="1MRtACTR VYo5PIO">[<a href="#ref-1MRtACTR" role="doc-biblioref">20</a>,<a href="#ref-VYo5PIO" role="doc-biblioref">21</a>]</span>.
The second row of SVs near the active zone (45-75 nm from AZ), immediately after the proximal vesicles (&lt;45nm from AZ), is called the intermediate region.
Resting state intermediate SVs are less densely packed and also less connected than proximal SVs <span class="citation" data-cites="K10hpz3n">[<a href="#ref-K10hpz3n" role="doc-biblioref">14</a>]</span>.
This suggests that, after exocytosis of RRP SVs, intermediate SVs could be rapidly recruited in the RRP by diffusion <span class="citation" data-cites="S9w67bLc">[<a href="#ref-S9w67bLc" role="doc-biblioref">22</a>]</span>.
Synaptic activity enhances the mobility of a fraction of SVs, whereas it induces synapsin dissociation from SVs in a synapsin phosphorylation-dependent manner <span class="citation" data-cites="2nauTxqh if88f9zh">[<a href="#ref-2nauTxqh" role="doc-biblioref">23</a>,<a href="#ref-if88f9zh" role="doc-biblioref">24</a>]</span>.
The same mobility enhancement can be achieved through inhibition of synapsin dephosphorylation, which leads to synapsin dissociation from SVs, or by knocking out all three synapsin forms <span class="citation" data-cites="43WYZu5a 1AX9UEqJU 9xDm8GpQ">[<a href="#ref-43WYZu5a" role="doc-biblioref">25</a>,<a href="#ref-1AX9UEqJU" role="doc-biblioref">26</a>,<a href="#ref-9xDm8GpQ" role="doc-biblioref">27</a>]</span>.
Interestingly, ribbon synapses do not express synapsin and show higher SV mobility than conventional synapses <span class="citation" data-cites="BuOLXzHq">[<a href="#ref-BuOLXzHq" role="doc-biblioref">28</a>]</span>.
It is therefore conceivable that inter-SV connectors restrain SV diffusion and that synaptic activity influences the level of inter-SV connectivity and thereby their mobility.</p>
<p>To investigate this hypothesis and to better understand the impact of depolarization and synaptic activity on SV tethering, we designed two sets of cryo-ET experiments.
On the one hand, we compared the morphology of wild-type rat synaptosomes in resting state and a few milliseconds after depolarization.
On the other hand, to study the consequences of increased or decreased spontaneous synaptic activity, we imaged synapses in mouse neuronal culture expressing either wild-type SNAP-25, a more positively charged SNAP-25 mutant (4K mutant), or a more negatively charged mutant of SNAP-25 (4E mutant) <span class="citation" data-cites="AVAZzvH0">[<a href="#ref-AVAZzvH0" role="doc-biblioref">12</a>]</span>
The more positively charged SNAP-25 mutant, which is constitutively active, showed no triple-tethered SV <span class="citation" data-cites="AVAZzvH0">[<a href="#ref-AVAZzvH0" role="doc-biblioref">12</a>]</span>.
This confirmed the morphological definition of the RRP.
Our experiments revealed an immediate formation of additional tethers between proximal RRP vesicles and the PM after depolarization.
Shortly after exocytosis the level of inter-SV connectivity was decreased among SVs situated in a 25 to 75-nm distance range from the AZ PM.
Altogether, our results indicate a regulation through connectors of SV mobility and their recruitment at the AZ PM.</p>
<h2 id="results">Results</h2>
<p>To analyze the morphological changes occurring in the presynpase shortly after stimulation, we pursued a time-resolved cryo-electron tomography approach.
A 52-mM KCl containing buffer was sprayed with an atomizer to depolarize synaptosomes and stimulate exocytosis milliseconds before vitrification.
The spray droplet size was optimized by cutting a 1-ml pipet tip to a diameter matching an EM grid (3 mm) and fixed to the atomizer glass outlet to disperse the spray (Figure <a href="#fig:workflow">1</a>A1).
Furthermore, to achieve the shortest possible delay between spraying and freezing, the nozzle was set 1-2 mm above the liquid ethane container.
This generated many small spray droplets spread throughout the grid (Figure <a href="#fig:workflow">1</a>A2-A4).
Even if sprayed droplets were well distributed throughout the grid, not all synaptosomes were in contact with exocytosis-triggering KCl solution.
Given the very low throughput of cryo-electron tomography, we followed a correlative light and electron microscopy approach.
By cryo-fluorescence microscopy, we identified areas where synaptosomes fluorescently labeled by calcein blue and spray droplets labeled by fluorescine were colocalized.
Additionally, phase contrast imaging enabled quality control of the frozen EM grid with respect to ice contamination and ice cracks, as shown previously <span class="citation" data-cites="ySZJ84X1">[<a href="#ref-ySZJ84X1" role="doc-biblioref">29</a>]</span>.
9 control and 9 stimulated synaptosome tomograms were analyzed (Supplementary Figure {fig:suppl_tomogram_slices}A and B, Supplementary Movie <a href="#vid:synaptosome">S1</a>).
We restricted our analysis to synaptosomes that possessed a smooth PM, free of signs of rupturing and that had a mitochondrion, as we considered these factors essential for synaptosome function.</p>
<p>In addition, we manipulated the electrostatic state of the SNARE complex through mutated SNAP-25 protein introduced using lentiviral vectors into primary SNAP-25 knockout neurons grown on EM grids <span class="citation" data-cites="AVAZzvH0">[<a href="#ref-AVAZzvH0" role="doc-biblioref">12</a>]</span> (Figure <a href="#fig:workflow">1</a>B1-B4).
The “4E” mutation contains four glutamic acid substitutions, whereas the “4K” contains four lysine substitutions; all mutations are placed in the second SNARE-motif of SNAP-25 and were shown to decrease and increase the rate of spontaneous miniature release, respectively <span class="citation" data-cites="AVAZzvH0">[<a href="#ref-AVAZzvH0" role="doc-biblioref">12</a>]</span>.
Optimization of Primary neurons culturing conditions allowed us to establish a protocol, which provides functional synapses thin enough for direct imaging by cryo-ET.
Astrocytes were added to 12 well plates and were grown for 2 days.
After 2 days, the medium was exchanged to a medium that favors neuronal growth and impedes astrocyte growth.
At the same time a droplet of the neuronal suspension was added onto a flame sterilized EM grid and incubated for 30 min at 37 °C, hereafter the grids were placed into the 12 well plates containing the astrocytes.
Neurons were grown for 10-14 days until plunge freezing and were then analyzed at a Titan Krios by cryo-ET (Supplementary Figure {fig:suppl_tomogram_slices}C and D, Supplementary Movie <a href="#vid:neuron">S2</a>.
Thereby, we could image chronically overactive or inactive synapses and relate presynaptic architectural modifications to different functional states.</p>
<div id="fig:workflow" class="fignos">
<figure>
<img src="images/workflow.svg" style="width:15cm" alt="Figure 1: Experimental models. A1) Glass atomizer used to disperse depolarizing solution on the EM grid milliseconds before the grid is vitrified. A2) Spray droplets imaged with the GFP filter set. A3) Synaptosomes imaged with the DAPI filter set. A4) Overlay of spray droplets (green) and synaptosomes (red). Scale bars, 20 μm. B1) Schematic drawing of a 6-well petridish depicting astrocytes (pink stars) growing at the bottom of the petridish and EM grids (black round grid overlaying the astrocytes) with neurons growing on the grids. B2) Gridsquare overview with neurons growing over it; scale bar = 5 µm. B3) Medium magnification overview of neurons growing over R2/1 holes; scale bar = 500 nm. B4) One slice of a tomogram depicting the synapse and its respective postsynapse; scale bar = 100 nm." /><figcaption aria-hidden="true"><span>Figure 1:</span> <strong>Experimental models.</strong> A1) Glass atomizer used to disperse depolarizing solution on the EM grid milliseconds before the grid is vitrified. A2) Spray droplets imaged with the GFP filter set. A3) Synaptosomes imaged with the DAPI filter set. A4) Overlay of spray droplets (green) and synaptosomes (red). Scale bars, 20 μm. B1) Schematic drawing of a 6-well petridish depicting astrocytes (pink stars) growing at the bottom of the petridish and EM grids (black round grid overlaying the astrocytes) with neurons growing on the grids. B2) Gridsquare overview with neurons growing over it; scale bar = 5 µm. B3) Medium magnification overview of neurons growing over R2/1 holes; scale bar = 500 nm. B4) One slice of a tomogram depicting the synapse and its respective postsynapse; scale bar = 100 nm.</figcaption>
</figure>
</div>
<h3 id="increased-membrane-curvature-at-the-onset-of-exocytosis">Increased membrane curvature at the onset of exocytosis</h3>
<p>We analyzed the morphology of SVs fusing with the AZ PM.
Synaptosomes of a single grid have not all been stimulated for the same duration.
Some synaptosomes have been in contact with the stimulating solution from the moment the droplet touched the grid, while others were not at all in contact with the stimulating solution because no droplet landed close enough to them.
Importantly, some synaptosomes were located near the point of impact of a droplet, in which case, it took some time for KCl to diffuse until the concentration around these synaptosomes rose sufficiently to trigger exocytosis.
Therefore, the time interval between triggering exocytosis and freezing ranged between 0 ms and the interval between spray droplets hitting the grid and freezing, which was comprised between 7 and 35 ms depending on the experiments (see <span class="citation" data-cites="17b1KeHkw">[<a href="#ref-17b1KeHkw" role="doc-biblioref">30</a>]</span>).
This time window, and the diffusional delay of the 52 mM KCl, made it possible to identify synaptosomes at different stages of fusion.</p>
<p>Synaptosomes from both control and sprayed grids were thoroughly analyzed for signs of exocytosis, which consisted of morphological changes of the AZ PM and the tethered SV occurring upon stimulation, which are described hereafter.
These signs were only detected in synaptosomes from sprayed grids and are presented in the most parsimonious chronological order.
Upon stimulation, both the vesicle membrane and the PM were slightly bent towards each other (Figure <a href="#fig:membrane_morphology">2</a>B1-B3; orange arrows).
These structures, which have previously been reported in liposomes but not in synapses, have been referred to as membrane curvature events <span class="citation" data-cites="qeT2GTd8">[<a href="#ref-qeT2GTd8" role="doc-biblioref">11</a>]</span>.
Control synaptosomes (i.e. not sprayed) on the other hand, had a straight PM, and no SV membrane was buckled (Figure <a href="#fig:membrane_morphology">2</a>A).
Following membrane bending, we observed contacts between vesicles and the PM bilayer where both membranes lose their clear contours (Figure <a href="#fig:membrane_morphology">2</a>C1 &amp; C2; pink arrows).
This was followed by further transitioning states prior to and during pore opening (Figure <a href="#fig:membrane_morphology">2</a>D-F; blue arrows).
In the next observed fusion state, the vesicle was wide open (Figure <a href="#fig:membrane_morphology">2</a>G), followed by almost completely collapsed vesicles where only a small bump on the PM remained visible (Figure <a href="#fig:membrane_morphology">2</a>H).
These structures were not observed in any of the non-sprayed control datasets.</p>
<div id="fig:membrane_morphology" class="fignos">
<figure>
<img src="images/vesicle_morphology.svg" style="width:15cm" alt="Figure 2: SV exocytosis morphology. Tomographic slice of non-stimulated (A) and stimulated rat synaptosomes (B-H). A) Image of a 2.2-nm thick tomographic slice showing a non-stimulated with SVs at the AZ and a straight PM. B1) Membrane curvature event, 2.2-nm thick tomographic slice. B2) Membrane curvature event, 6.5-nm thick tomographic slice. B3) Membrane curvature event, 2.24 nm thick tomographic slice. Orange arrows showing membrane curvature event. C1,C2) Lipid perturbations of PM and SV, 22-nm thick tomographic slices. The space between SV and PM is denser than in the non-stimulated synaptosomes (see pink arrow). D-F) Vesicles with a pore opening that might be on the way to full collapse fusion, 33-nm thick tomographic slice thickness: 22 nm (D), 30.8 (E), 33 nm (F). G) Wide pore opening, most likely on the way to full collapse fusion, 2.2-nm tomographic slice. H) Remaining bump at the end of full collapse fusion, 11-nm thick tomographic slice. Scale bar, 50 nm." /><figcaption aria-hidden="true"><span>Figure 2:</span> <strong>SV exocytosis morphology.</strong> Tomographic slice of non-stimulated (A) and stimulated rat synaptosomes (B-H). A) Image of a 2.2-nm thick tomographic slice showing a non-stimulated with SVs at the AZ and a straight PM. B1) Membrane curvature event, 2.2-nm thick tomographic slice. B2) Membrane curvature event, 6.5-nm thick tomographic slice. B3) Membrane curvature event, 2.24 nm thick tomographic slice. Orange arrows showing membrane curvature event. C1,C2) Lipid perturbations of PM and SV, 22-nm thick tomographic slices. The space between SV and PM is denser than in the non-stimulated synaptosomes (see pink arrow). D-F) Vesicles with a pore opening that might be on the way to full collapse fusion, 33-nm thick tomographic slice thickness: 22 nm (D), 30.8 (E), 33 nm (F). G) Wide pore opening, most likely on the way to full collapse fusion, 2.2-nm tomographic slice. H) Remaining bump at the end of full collapse fusion, 11-nm thick tomographic slice. Scale bar, 50 nm.</figcaption>
</figure>
</div>
<p>Stimulated synaptosome datasets were divided into early and late fusion stages, respectively, based on the morphology of SV and AZ PM.
Synapses showing membrane bending and direct lipid contact between SV and PM without an open pore were classified as early fusion (Figure <a href="#fig:membrane_morphology">2</a>B-E).
Those with an open pore or a remaining small bump of a fully collapsed vesicle were classified as late fusion (Figure <a href="#fig:membrane_morphology">2</a>F-H).</p>
<h3 id="synaptic-vesicle-distribution-is-impacted-by-synaptic-activity">Synaptic vesicle distribution is impacted by synaptic activity</h3>
<p>Non-sprayed rat synaptosomes as well as WT-SNAP-25 mouse cultured neuron synapses showed typical SV distribution, as observed in previous cryo-ET studies (Figure <a href="#fig:vesicle_distribution">3</a>)<span class="citation" data-cites="XQJ3R1HJ">[<a href="#ref-XQJ3R1HJ" role="doc-biblioref">13</a>]</span>.
Vesicle occupancy in WT-SNAP-25 synapses was 0.09 in the most proximal zone (0-25 nm from the AZ PM), and peaked to 0.18 in 25-50 nm zone. It then dropped to 0.08 in the intermediate zone (50-75 nm) and rose steadily more distally to reach a plateau of ~0.16 spanning the range of 150 to 250 nm distance.
Finally, SV occupancy gradually decreased as the distance from the AZ increased (Figure <a href="#fig:vesicle_distribution">3</a>A).</p>
<div id="fig:vesicle_distribution" class="fignos">
<figure>
<img src="images/vesicle_occupancy_and_distance.png" style="width:15cm" alt="Figure 3: SV distribution. (A, B) Vesicle occupancy expressed as fraction of cytosol volume occupied by vesicles as a function of distance to AZ in (A) cultured neurons and (B) synaptosomes. Each solid line represents the mean occupancy value for the experimental group, while the shaded areas depict the 95% confidence interval. (C, D) Distance of proximal SVs from the AZ. Horizontal line: mean; whiskers: 95% confidence interval. Statistical test: t-test." /><figcaption aria-hidden="true"><span>Figure 3:</span> <strong>SV distribution.</strong> (A, B) Vesicle occupancy expressed as fraction of cytosol volume occupied by vesicles as a function of distance to AZ in (A) cultured neurons and (B) synaptosomes. Each solid line represents the mean occupancy value for the experimental group, while the shaded areas depict the 95% confidence interval. (C, D) Distance of proximal SVs from the AZ. Horizontal line: mean; whiskers: 95% confidence interval. Statistical test: t-test.</figcaption>
</figure>
</div>
<p>The absolute values differ between WT cultured mouse neurons and non-stimulated rat synaptosomes but the SV occupancy distribution follows the same pattern.
The difference in absolute values can likely be attributed to the different experimental and animal models used.
Sprayed synaptosomes that were showing early signs of exocytosis had a nearly identical SV occupancy pattern as non-sprayed synaptosomes (Figure <a href="#fig:vesicle_distribution">3</a>B, dark blue and gray, respectively).
However, when SV full collapse figures were apparent, SV occupancy in the proximal zone was significantly reduced, whereas SV occupancy further away from the AZ PM was unchanged (Figure <a href="#fig:vesicle_distribution">3</a>B, red).
This is consistent with some membrane proximal SVs having engaged in exocytosis, while none of the recycling and reserve pool SVs have.
In order to investigate the consequences of chronic high or low synaptic activity, we investigated the 4E and 4K mutants (Figure <a href="#fig:vesicle_distribution">3</a>A, green and gold, respectively).
In the most proximal 50 nm, SV were significantly less concentrated in the constitutively depressed 4E mutant than in the WT.
However, they were significantly more abundant between 75 and 100 nm.
Furthermore, proximal SVs were on average more distant to the AZ PM in the 4E mutant than in the WT (Figure <a href="#fig:vesicle_distribution">3</a>C).
These observations are consistent with the increased distance between SVs and the PM induced by the repulsion between negative charges present in SNAP-25 and on the AZ/PM.
In the most distal zones, SV occupancy gradually decreased in the 4E mutant and was significantly lower than in the WT over most of the 250 to 750 nm distance range.
The decrease may reflect deleterious effects associated with abnormally low synaptic activity.
The 4K mutant displayed a significantly decreased SV occupancy in comparison to the WT in the most proximal 50 nm.
4K proximal SVs were in average more located further away from the AZ PM than WT proximal SVs.
Furthermore, proximal SVs were in average more distant to the AZ PM in the 4K mutant than in the WT (Figure <a href="#fig:vesicle_distribution">3</a>C).
This can be readily attributed to the high probability of spontaneous exocytosis generated by the additional positive charges of the SNARE bundle.
Between 50 nm and 75 nm away from the AZ PM, SV occupancy was higher in the 4K mutant than in the WT, consistent with recycling pool SVs being recruited.
From 100 to 250 nm, SV occupancy dropped steadily, in contrary to the WT, and from a distance of 170 nm, it was significantly lower.
Yet, beyond 250 nm, it rose linearly until 450 nm, becoming indistinguishable from WT occupancy, and then experienced a sharp increase, peaking to 0.2 at a distance of 550 nm, significantly higher than WT occupancy, before dropping quickly and significantly lower than WT occupancy.</p>
<h3 id="proximal-vesicles-form-additional-tethers-following-stimulation">Proximal vesicles form additional tethers following stimulation</h3>
<p>We investigated the tethering state of proximal SVs (i.e. the SVs whose center is located within 45 nm of the AZ PM) prior to and following stimulation in synaptosomes.
In non-sprayed synaptosomes, 54% of the proximal vesicles were tethered, which is in agreement with previous results (Supplementary Figure <a href="#fig:suppl_histograms">S2</a>B) <span class="citation" data-cites="XQJ3R1HJ">[<a href="#ref-XQJ3R1HJ" role="doc-biblioref">13</a>]</span>.</p>
<p>Interestingly, in the early fusion group the fraction of tethered proximal vesicles significantly increased to 80% (P&lt;0.05, χ2 test).
In the late fusion group, however, 53% of the proximal vesicles were tethered, which is not significantly different to the non-sprayed group.
The average number of tethers per proximal SV followed the same pattern. Proximal SVs had 0.89 ± 0.12 tethers in the non-sprayed group (Figure <a href="#fig:tethers">4</a>D).
This parameter rose to 2.09 ± 0.33 in the early fusion group, while it returned to 1.00 ± 0.20 in the late fusion group.</p>
<div id="fig:tethers" class="fignos">
<figure>
<img src="images/vesicle_tethers.png" style="width:15cm" alt="Figure 4: Proximal SV tethering. (A, B) Fraction of proximal SVs that are triple tethered. Each bar shows the overall fraction of all proximal SVs from a given experimental condition. Each dot represents the value of an individual synapse. Statistical test: χ2 test. (C, D) Number of tethers per proximal SV. Each dot represents an individual SV. The vertical line represents the mean value, and the horizontal whiskers correspond to the 95% confidence interval. Statistical test: t-test." /><figcaption aria-hidden="true"><span>Figure 4:</span> <strong>Proximal SV tethering.</strong> (A, B) Fraction of proximal SVs that are triple tethered. Each bar shows the overall fraction of all proximal SVs from a given experimental condition. Each dot represents the value of an individual synapse. Statistical test: χ<sup>2</sup> test. (C, D) Number of tethers per proximal SV. Each dot represents an individual SV. The vertical line represents the mean value, and the horizontal whiskers correspond to the 95% confidence interval. Statistical test: t-test.</figcaption>
</figure>
</div>
<p>We then analyzed whether the decreased occupancy in the late fusion group was associated with a decreased number of triple-tethered SVs, (defined as SV with at least three tethers) which as mentioned in the introduction are suggested to belong to the RRP.
In resting, non-sprayed synapses 8% of the proximal SVs were triple-tethered (Figure <a href="#fig:tethers">4</a>B).
Surprisingly, the fraction of triple-tethered proximal SVs drastically increased to 29% in the early fusion group (P&lt;0.001, χ2 test).
The fraction decreased to 13% in the late fusion group.
This suggests that upon stimulation some proximal SVs very rapidly acquire new tethers.
Using our definition of the RRP (vesicles that are triple-tethered) this would indicate that the RRP rapidly increases after stimulation and more vesicles become primed for exocytosis.
Furthermore, the lower proximal vesicle occupancy in the late fusion group indicates that under our stimulation conditions, replenishing vesicles to the proximal zone is slower than their release.</p>
<p>The situation in the WT-SNAP-25 neurons was similar to unstimulated synaptosomes.
53% of the all proximal SVs were tethered and 17% of all proximal SVs belonged were triple-tethered (Supplementary Figure <a href="#fig:suppl_histograms">S2</a>A and Figure <a href="#fig:tethers">4</a>A.
On average, proximal SVs had 1.17 ± 0.23 tethers.
The corresponding values for the 4E mutants were not significantly different (15% and 0.96 ± 0.18, respectively).
However, in all 4K mutant datasets there was not a single SV that was part of the RRP, i.e. triple-tethered.
Consistently, the number of tethers per proximal SV was significantly lower in the 4K mutant than in the WT (Figure <a href="#fig:tethers">4</a>C).
These results are in line with physiological measurements that have shown that the RRP is depleted in the chronically spontaneously active 4K mutant, and they provide additional evidence that RRP-vesicles have at least 3 tethers. <span class="citation" data-cites="AVAZzvH0">[<a href="#ref-AVAZzvH0" role="doc-biblioref">12</a>]</span>.</p>
<h3 id="synaptic-activity-modifies-inter-sv-connectivity">Synaptic activity modifies inter-SV connectivity</h3>
<p>The majority of SV are linked to other SVs via molecular bridges previously termed connectors connectors <span class="citation" data-cites="XQJ3R1HJ K10hpz3n">[<a href="#ref-XQJ3R1HJ" role="doc-biblioref">13</a>,<a href="#ref-K10hpz3n" role="doc-biblioref">14</a>]</span>. The function and composition of connectors are not clear yet.
It was earlier proposed that connectors limit SV dispersion and allow SV mobilization for release.
It is generally assumed that synapsin is involved in connector formation and may be one of its components.
It has been suggested that connectors reduce SV mobility and maintain a local high SV concentration in the presynapse.
The connectivity level of an individual SV might be one of the factors defining the pool to which the SV belongs.
To shed some light on the role of connectors, we analyzed SV connectivity in our datasets.
We focused most of our analysis to the SVs located at distance of the AZ PM lower than 250 nm.
Furthermore, we defined 4 distance groups: proximal (0-45 nm), intermediate (45-75 nm), distal 1 (75-150 nm), distal 2 (150-250 nm), as in previous studies <span class="citation" data-cites="XQJ3R1HJ nkmPPJfq">[<a href="#ref-XQJ3R1HJ" role="doc-biblioref">13</a>,<a href="#ref-nkmPPJfq" role="doc-biblioref">31</a>]</span>.
We first analyzed synaptosomes.
In non-sprayed synaptosomes datasets, approximately 70% of the proximal and intermediate SVs were connected to other vesicles.
In distal 1 and 2 regions, this value rose to 84 and 87%, respectively.
Similarly, the number of connectors per vesicles significantly increased from the proximal region (1.63 ± 0.13) to the distal 1 region (2.57 ± 0.09, P&lt;0.01) and the distal 2 region (2.78 ± 0.10, P&lt;0.0001) (Figure <a href="#fig:connectors">5</a>B).
Sprayed early and late fusion synapses showed a similar pattern, with significantly more connectors per SV in distal 1 region than in proximal region for the late fusion group, and more connectors per SV in distal 2 region for both early and late fusion groups.
Nonetheless, proximal SV connectivity and number of connectors per SV dropped in the late fusion group to 53% (P&lt;0.09, χ<sup>2</sup> test) and 1.12 ± 0.15 (P&lt;0.05), respectively (Figure <a href="#fig:connectors">5</a>D and B).
This decrease originates from tethered SVs.
Indeed, 1.65 ± 0.23 connectors per tethered SVs were found in the non-sprayed group. This number increased to 2.58 ± 0.45 in the early fusion group (P&lt;0.05) and dropped to 0.85 ± 0.25 in the late fusion group (P&lt;0.05), whereas no significant differences were found in the numbers of connnectors per proximal non-tethered SVs (Figure <a href="#fig:connectors">5</a>H).
Furthermore, the decrease in proximal connectivity was only observed in non triple-tethered SVs, of which only 51.5% of the vesicles were connected as opposed to 72.3% before stimulation (P&lt;0.01, χ<sup>2</sup> test; Supplementary Figure <a href="#fig:suppl_histograms">S2</a>J).
Consistently, the number of connectors per non triple-tethered proximal SV went from 1.64±0.17 in the non-sprayed group, rose significantly to 2.69 ± 0.54 in the early fusion group (P&lt;0.05) and dropped to 0.9±0.19 in the late fusion group (P&lt;0.05) (Figure <a href="#fig:connectors">5</a>F).
Taken together, our observations indicate that following depolarization, the number of connectors per proximal single- or double-tethered vesicles (i.e. non-RRP) first increases and then decreases to a value lower than the initial one.
We have seen earlier that the fraction of tethered proximal SVs does not differ between non-sprayed and late fusion synaptosomes.
Thus, our data suggest that establishing connectivity is a slower process than tethering.
We hypothesize that given the free space made in the proximal region after some SVs have fused, non-connected vesicles from the intermediate region diffuse to the proximal zone and become tethered to the AZ PM.
Only subsequently, these newly tethered vesicles get interconnected.
Furthermore, we have observed that connectors remained present between fusing SV and neighbor SV (Supplementary Figure <a href="#fig:suppl_connected_slices">S3</a>).
This, in addition to passive diffusion, can contribute to replenishing the RRP.</p>
<div id="fig:connectors" class="fignos">
<figure>
<img src="images/vesicle_connectors.png" style="width:15cm" alt="Figure 5: SV connectivity. (A, B) Number of connectors per SV as a function of their distance to the AZ PM for mouse neurons (A) and rat synaptosomes (B) Each solid line represents the average value of all SVs belonging to a particular experimental condition. Shaded areas represent 95%-confidence interval ranges. (C, D) Fraction of connected vesicles as a function of distance to the AZ PM for mouse neurons (C) and rat synaptosomes (D). (E, F) Number of connectors per proximal SV belonging or not to the RRP for mouse neurons (E) and rat synaptosomes (F). (G, H) Number of connectors per tethered or non-tethered proximal SV for mouse neurons (G) and rat synaptosomes (H). *: P&lt;0.05." /><figcaption aria-hidden="true"><span>Figure 5:</span> <strong>SV connectivity.</strong> (A, B) Number of connectors per SV as a function of their distance to the AZ PM for mouse neurons (A) and rat synaptosomes (B) Each solid line represents the average value of all SVs belonging to a particular experimental condition. Shaded areas represent 95%-confidence interval ranges. (C, D) Fraction of connected vesicles as a function of distance to the AZ PM for mouse neurons (C) and rat synaptosomes (D). (E, F) Number of connectors per proximal SV belonging or not to the RRP for mouse neurons (E) and rat synaptosomes (F). (G, H) Number of connectors per tethered or non-tethered proximal SV for mouse neurons (G) and rat synaptosomes (H). *: P&lt;0.05.</figcaption>
</figure>
</div>
<p>We then analyzed SNAP-25 neurons.
For SNAP-25-WT, similarly to non-sprayed synaptosome, the fraction of connected SVs was significantly higher in the distal 2 region than in the proximal region (p&lt;0.01), albeit the absolute values were overall lower than in synaptosomes (Figure <a href="#fig:connectors">5</a>C).
Consistently, the number of connectors per SV in SNAP-25-WT synapses increased from 1.95 ± 0.38 in the proximal region to 3.23 ± 0.21 in the distal 2 region (Figure <a href="#fig:connectors">5</a>A, P&lt;0.01).
The fraction of connected SVs in the distal 2 region was significantly lower in the 4E and 4K mutant than in the WT (p&lt;0.05).
This was supported by a significantly lower number of connectors per SV in the distal 1 region for the 4K mutant versus the WT (P&lt;0.05), and in the distal 2 region for the 4E mutant versus the WT (P&lt;0.0001) as well as for the 4K mutant versus the WT (P&lt;0.0001, Figure <a href="#fig:connectors">5</a>A).
The number of connectors per proximal SV was not affected by the mutations (Figure <a href="#fig:connectors">5</a>E and G).
These results indicate that prolonged abnormal exocytotic activity is correlated with severe changes in intervesicular connectivity in the distal region.</p>
<h2 id="discussion">Discussion</h2>
<p>Due to its transient nature, SV exocytosis has been difficult to characterize morphologically.
A number of questions remain partially unresolved to this date.
In particular, it has been suggested that following Ca<sup>2+</sup> entry, the insertion of synaptotagmin-1 into the membrane induces an increase in membrane curvature, which lowers the energy barrier of fusion.
Such membrane deformations have been observed in biochemically reconstituted models of exocytosis but have not yet been reported in functional synapses <span class="citation" data-cites="qeT2GTd8 PB9esFqe">[<a href="#ref-qeT2GTd8" role="doc-biblioref">11</a>,<a href="#ref-PB9esFqe" role="doc-biblioref">32</a>]</span>.
Moreover, it is not clear whether the membrane deformation occurs subsequently to Ca<sup>2+</sup> influx or if primed SVs and their PM counterpart present such deformation <span class="citation" data-cites="PB9esFqe">[<a href="#ref-PB9esFqe" role="doc-biblioref">32</a>]</span>.
The optimal sample preservation delivered by cryo-ET makes it possible to investigate the role of tethers located between SVs and the AZ PM and the function of inter-SV connectors.
Combining cryo-ET with spray-mixing plunge-freezing enabled us to investigate the morphological changes occurring immediately after depolarization.</p>
<h3 id="membrane-curvature-increases-following-depolarization">Membrane curvature increases following depolarization</h3>
<p>Depolarization through spraying droplets of KCl solution on synaptosomes milliseconds before freezing allowed us to capture snapshots of exocytosis (Figure <a href="#fig:membrane_morphology">2</a>B1-B3).
We observed that the curvature of some PM regions facing some SVs increased following depolarization.
The SV facing such a PM buckling also seemed to get kinked.
These deformations were not seen in non-sprayed synaptosomes.
This indicates that in functional synapses exocytosis starts with a Ca<sup>2+</sup>-dependent membrane deformation.
The latter may be caused by the intercalation of synaptotagmin-1 C2A and C2B domains between membrane head groups.
A recent biophysical study indicated that C2A and C2B preferably insert in SV membrane and PM, respectively <span class="citation" data-cites="6ISfFKuj">[<a href="#ref-6ISfFKuj" role="doc-biblioref">33</a>]</span>.
It may also be due to the tension/force induced by SNARE-complex zippering <span class="citation" data-cites="1HOYj7n0K">[<a href="#ref-1HOYj7n0K" role="doc-biblioref">34</a>]</span>.
Subsequent snapshots showed a fuzzy contact point between the SV and the PM, which likely corresponds to lipid splaying or the merging of the two membranes.
Membrane fusion then occurred and yielded classical Ω-figures with variable neck diameters.
Finally, nearly fully collapsed SVs were imaged.
Overall our observations support the standard model of full collapse SNARE-dependent membrane fusion <span class="citation" data-cites="8Axaq8nk TrqX98bo">[<a href="#ref-8Axaq8nk" role="doc-biblioref">35</a>,<a href="#ref-TrqX98bo" role="doc-biblioref">36</a>]</span> and reveal details of exocytosis early stage, prior to actual membrane fusion.</p>
<h3 id="sv-local-concentration-correlates-with-sv-connectivity">SV local concentration correlates with SV connectivity</h3>
<p>SV local concentration - a.k.a SV occupancy - is tightly correlated with the distance from the AZ PM. Under resting conditions, SV occupancy reaches a local maximum at the distance approximately 25 nm and a local minimum at approximately 75 nm from the AZ PM, before rising again with the distance increasing (Figure <a href="#fig:vesicle_distribution">3</a>A and B), in agreement with previous reports <span class="citation" data-cites="XQJ3R1HJ">[<a href="#ref-XQJ3R1HJ" role="doc-biblioref">13</a>]</span>.
By definition, all SVs in the proximal region are directly facing the PM.
Their high concentration can be attributed to the fact that more than 50% of them are tethered to the PM.
On the other hand, the number of connectors per SV and SV connectivity gradually increases with the distance from the AZ (Figure <a href="#fig:connectors">5</a>A-D).
This increase correlates with the increase in occupancy.
Thus, we may hypothesize that SV local concentration is a function of their level of tethering to the PM and of connection with other SVs.
Interestingly, under short stimulation of a few ms, SV occupancy only decreases in the proximal region, as a consequence of the fusion of SVs with the PM (Figure <a href="#fig:vesicle_distribution">3</a>B).
In order to further assess the relation between SV tethering, connectivity, and occupancy, we analyzed synapses bearing expressing either WT SNAP-25, a more positively charged mutant (4K), or a more negatively charged mutant (4E) <span class="citation" data-cites="AVAZzvH0">[<a href="#ref-AVAZzvH0" role="doc-biblioref">12</a>]</span>.
The 4K mutant has a decreased energy barrier to membrane fusion and causes constitutively active exocytosis, whereas the 4E mutant shows a decreased exocytotic activity because of a higher energy barrier to membrane fusion.
Both mutants had a significantly decreased proximal SV occupancy (Figure <a href="#fig:vesicle_distribution">3</a>A).
In the case of the 4K mutant, this was probably due to the high frequency of spontaneous exocytosis.
On the other hand, the 4E mutant, because of its additive negative charges, tends to repel SVs from the PM (Figure <a href="#fig:vesicle_distribution">3</a>C), which can explain their decreased proximal occupancy.
Over a narrow distance range, around 75 nm, SV occupancy of both mutants was significantly higher than that of the WT.
Further away, SV occupancy was most significantly lower for the 4E mutants than for the WT.
The 4K mutant followed the same trend but had a massive increase of SV occupancy between 500 and 600 nm, before falling again.
Interestingly, the number of connector per SV follows a similar pattern as SV occupancy in both mutants.
In the 4E mutant, it remains significantly lower than in the WT from approximately 200 nm and beyond.
In the 4K mutant, this value is significantly lower than in the WT from approximately 100 nm but it then rises close to WT values from approximately 400 nm onward.
This increased occupancy may be due to the recycling of spontaneously fusing SVs.
Our data show that strong disturbances in exocytotic activity lead to profound differences in SV occupancy and SV connectivity.
We note that a correlation exists between SV connectivity and concentration.
Future studies will be necessary to assess whether SV concentration depends on the SV connectivity and to decipher the molecular mechanism influencing these parameters.</p>
<h3 id="snap-25-4k-mutant-further-supports-the-rrp-morphological-definition">SNAP-25 4K mutant further supports the RRP morphological definition</h3>
<p>Previously, we showed that the number of tethers of a SV defines whether its exocytosis can be induced by treatment with a hyperosmotic sucrose solution, which corresponds to a definition of the RRP <span class="citation" data-cites="XQJ3R1HJ K10hpz3n">[<a href="#ref-XQJ3R1HJ" role="doc-biblioref">13</a>,<a href="#ref-K10hpz3n" role="doc-biblioref">14</a>]</span>.
We reported that SVs with at least 3 tethers belong to the RRP, according to this definition.
In order to further assess this model, we analyzed synapses of neurons expressing the SNAP-25 mutants.
17% of the WT proximal SVs had 3 tethers or more.
Critically, the 4K mutant had none such SV.
As the RRP (assessed with hyperosmotic sucrose treatment) in this mutant was formerly shown through functional assays to be depleted, our present observation further supports our morphological definition of the RRP <span class="citation" data-cites="AVAZzvH0">[<a href="#ref-AVAZzvH0" role="doc-biblioref">12</a>]</span>.
15% of the proximal SVs had 3 tethers or more in the 4E mutant, which is very similar to the WT situation, while this mutant was shown to possess a normal-sized RRP.
Our observations are also consistent with a number of studies that have concluded that SV exocytosis requires a minimum of three SNARE complexes <span class="citation" data-cites="1ErVrgCCc yI8145n3 12xIhQNRh">[<a href="#ref-1ErVrgCCc" role="doc-biblioref">8</a>,<a href="#ref-yI8145n3" role="doc-biblioref">9</a>,<a href="#ref-12xIhQNRh" role="doc-biblioref">10</a>]</span>.</p>
<h3 id="depolarization-rapidly-induces-additional-tethering-in-proximal-vesicles">Depolarization rapidly induces additional tethering in proximal vesicles</h3>
<p>We compared SV tethering before and shortly after depolarization.
Our observations are schematically summarized in Figure <a href="#fig:model">6</a>.
Interestingly, the fraction of proximal SVs that were tethered increased by 50% shortly after stimulation, in synapses showing early signs of exocytosis.
Simultaneously, the number of tethers per proximal SV more than doubled and, the fraction of proximal SVs with 3 or more tethers tripled.
In presynaptic terminals presenting more advanced stages of exocytosis (Ω-figures), all these measurements returned to pre-stimulation values.
These data indicate that immediately after the onset of stimulation a quick and massive increase in tethering occurs.
This phenomenon was resolved in our measurements, because the spraying of synaptosomes with an intermediate K<sup>+</sup>-concentration made it possible to isolate synaptosomes in an early stage of fusion, which would have been missed during either strong or chronic stimulation, which would deplete primed vesicles.</p>
<p>The phenomenon of rapid, depolarization-induced tethering leads to some free proximal SVs becoming tethered to the AZ PM, while some previously single- or double-tethered SVs gained the additional tether(s) that according to our definition of the RRP (as triple-tethered vesicles) would be expected to render them releasable <span class="citation" data-cites="XQJ3R1HJ">[<a href="#ref-XQJ3R1HJ" role="doc-biblioref">13</a>]</span>.
There are several important implications of this finding.
First, the increase of the number of tethers during the initial membrane contact - in excess of the three tethers formed during priming - might help overcome the fusion barrier.
Functional reconstruction led to the suggestion that SNARE-complexes primarily form downstream of Ca<sup>2+</sup>-influx <span class="citation" data-cites="q1FEacfR">[<a href="#ref-q1FEacfR" role="doc-biblioref">37</a>]</span>, whereas mutagenesis studies in cells supported the notion that SNARE-complexes had already formed before arrival of the Ca<sup>2+</sup>-trigger, i.e. during priming <span class="citation" data-cites="dh8gQ3IE">[<a href="#ref-dh8gQ3IE" role="doc-biblioref">38</a>]</span>.
In fact, both notions might be partly correct, as the formation of a low number of SNARE-complexes might lead to a stable primed state, defined by a valley in the energy landscape due to the dual inhibitory/stimulatory features of the SNARE-complex <span class="citation" data-cites="me9lnSLC AVAZzvH0">[<a href="#ref-AVAZzvH0" role="doc-biblioref">12</a>,<a href="#ref-me9lnSLC" role="doc-biblioref">39</a>]</span>, whereas more SNARE-complexes might form dynamically after triggering, during membrane fusion itself.
Accordingly, in in vitro fusion assays additional SNARE-complexes, above those required for fusion pore formation, leads to fusion pore stabilization and release of larger cargos <span class="citation" data-cites="12xIhQNRh 4GAxfKCY">[<a href="#ref-12xIhQNRh" role="doc-biblioref">10</a>,<a href="#ref-4GAxfKCY" role="doc-biblioref">40</a>]</span>.
Second, vesicles that have not formed three tethers before stimulation might fuse with delayed kinetics during triggering, which accounts for the variable exocytosis kinetics among SVs <span class="citation" data-cites="GwRRPuyV HUxTgHP7 mKOZu7Ak pGojwbzO">[<a href="#ref-GwRRPuyV" role="doc-biblioref">41</a>,<a href="#ref-HUxTgHP7" role="doc-biblioref">42</a>,<a href="#ref-mKOZu7Ak" role="doc-biblioref">43</a>,<a href="#ref-pGojwbzO" role="doc-biblioref">44</a>]</span>.
Superprimed vesicles are expected to have formed the largest number of tethers before stimulation <span class="citation" data-cites="HUxTgHP7 GwRRPuyV">[<a href="#ref-GwRRPuyV" role="doc-biblioref">41</a>,<a href="#ref-HUxTgHP7" role="doc-biblioref">42</a>]</span>.
Third, overlapping protein complexes might be involved in priming and triggering, depending on the timing of their formation.
Accordingly, triggering that stimulates tether-formation might also stimulate priming for those vesicles that were not tethered before stimulation.
Indeed, a number of recent publications have suggested that some SVs can get primed extremely quickly in response to Ca<sup>2+</sup> influx <span class="citation" data-cites="gmOeAcFB a4RbcT3S QTV8mtNy 13WUlCCxy pGojwbzO">[<a href="#ref-pGojwbzO" role="doc-biblioref">44</a>,<a href="#ref-gmOeAcFB" role="doc-biblioref">45</a>,<a href="#ref-a4RbcT3S" role="doc-biblioref">46</a>,<a href="#ref-QTV8mtNy" role="doc-biblioref">47</a>,<a href="#ref-13WUlCCxy" role="doc-biblioref">48</a>]</span>.</p>
<div id="fig:model" class="fignos">
<figure>
<img src="images/model_cartoon.svg" style="width:15cm" alt="Figure 6: Model depicting a synapse transitioning from resting state to early and late fusion states. Tethering and connectivity changes upon synapse stimulation are depicted. Proximal non triple-tethered vesicles (black proximal SVs) gain additional tethers and some of them become triple-tethered (yellow SVs) shortly after stimulation. Primed vesicles then fuse with the plasma membrane (late fusion) and leave an empty space in the AZ cytoplasm. The number of connectors (depicted in blue) per proximal SV increases in late fusion tripled-tethered vesicles. The red arrow shows a vesicle initially located in the intermediate region, which diffuses to the proximal region in the late fusion state. Tethers are shown in green." /><figcaption aria-hidden="true"><span>Figure 6:</span> <strong>Model depicting a synapse transitioning from resting state to early and late fusion states.</strong> Tethering and connectivity changes upon synapse stimulation are depicted. Proximal non triple-tethered vesicles (black proximal SVs) gain additional tethers and some of them become triple-tethered (yellow SVs) shortly after stimulation. Primed vesicles then fuse with the plasma membrane (late fusion) and leave an empty space in the AZ cytoplasm. The number of connectors (depicted in blue) per proximal SV increases in late fusion tripled-tethered vesicles. The red arrow shows a vesicle initially located in the intermediate region, which diffuses to the proximal region in the late fusion state. Tethers are shown in green.</figcaption>
</figure>
</div>
<h3 id="conclusion">Conclusion</h3>
<p>Our study revealed fine morphological changes occurring in the presynaptic terminal immediately after the onset of exocytosis, as well as in chronically active or inactive synapses.
It indicates increased SV tethering induced the rise in presynaptic Ca<sup>2+</sup>, potentially corresponding to SV superpriming, and preceding SV fusion.
It also highlights modifications of proximal SV interconnections in response to evoked exocytosis, as well as more drastic modifications of distal SV interconnections in chronically active synapses and in inactive synapses.
These changes likely affect SV mobility and recruitement at the AZ.</p>
<h2 id="materials-and-methods">Materials and methods</h2>
<h3 id="constructs-and-viruses">Constructs and viruses</h3>
<p>SNAP-25B was N-terminally fused to GFP and cloned into a pLenti construct with a CMV promoter <span class="citation" data-cites="n3fuhRE">[<a href="#ref-n3fuhRE" role="doc-biblioref">49</a>]</span>.
Mutations were made using the QuikChange II XL kit (Agilent).
The mutations were verified by sequencing and have been published before <span class="citation" data-cites="AVAZzvH0">[<a href="#ref-AVAZzvH0" role="doc-biblioref">12</a>]</span>.
The preparation of lentiviral particles followed standard protocols.</p>
<h3 id="animals">Animals</h3>
<p>Synaptosomes were prepared from adult male or female Wistar rats obtained from the central animal facilities of the Department of Biomedical Research of the University of Bern.
Adult male or female Wistar rats at an age of 6-8 weeks were slightly stunned by CO<sub>2</sub> and quickly decapitated with a guillotine.
The procedures used were in accordance with the Swiss Veterinary Law guidelines.
Heterozygous SNAP-25 KO C57/Bl6-mice were routinely backcrossed to Bl6 to generate new heterozygotes.
The strain was kept in the heterozygous condition and timed heterozygous crosses and caesarean section were used to recover knockout embryos at embryonic day 18 (E18).
Pregnant females were killed by cervical dislocation; embryos of either sex were collected and killed by decapitation. Permission to keep and breed SNAP-25 mice was obtained from the Danish Animal Experiments Inspectorate and followed institutional guidelines as overseen by the Institutional Animal Care and Use Committee (IACUC).
Newborn (P0-P2) CD1 outbred mice of either sex were used to create astrocytic cultures and for that were killed by decapitation.</p>
<h3 id="synaptosome-preparation">Synaptosome preparation</h3>
<p>Rat synaptosomes were prepared as previously described <span class="citation" data-cites="7sGZgtQa">[<a href="#ref-7sGZgtQa" role="doc-biblioref">50</a>]</span>, with some modifications.
The cerebral cortex and the hippocampi were removed in sucrose buffer (SEH: 0.32 M sucrose, 1 mM EDTA, 10 mM HEPES; HEPES, #H4034, Sigma-Aldrich Corporate Offices. St. Louis, MO, USA) on ice.
Homogenization of the tissue was done in SEH with a Potter-Elvehjem grinder (#358011, Wheaton. Millville, New Jersey, USA), four strokes at the bottom and 6 from top to bottom were applied to the tissue at a speed of 800 turns/min as described in <span class="citation" data-cites="7sGZgtQa">[<a href="#ref-7sGZgtQa" role="doc-biblioref">50</a>]</span>.
The whole process from decapitation to homogenization was done within 2-3 min, to obtain functional synaptosomes.
Homogenized tissue was then centrifuged at 1000 g for 10 min at 4°C to remove meninges and blood vessels.
The resulting supernatant containing synaptosomes, but also gliosomes and mitochondria was then added to a discontinuous, isoosmotic Percoll (#P1644, Sigma) gradient with 5%, 10% and 23% in 0.32 M sucrose, 1 mM EDTA in centrifuge tubes (#344060, Beckman Coulter).
The samples were spun in an ultracentrifuge (rotor: SW 40 Ti; Beckman Coulter. Nyon, Switzerland) at 16400 rpm for 12 min at 4°C.
The layer with the highest amount of functional synaptosomes was between 10-23 % <span class="citation" data-cites="7sGZgtQa">[<a href="#ref-7sGZgtQa" role="doc-biblioref">50</a>]</span>.
The layer was carefully taken out and diluted 1:10 in HEPES buffered medium (HBM; 140 mM NaCl, 5 mM KCl, 5 mM NaHCO<sub>3</sub>, 1.2 mM Na<sub>2</sub>HPO<sub>4</sub>, 1 mM MgCl<sub>2</sub>, 10 mM Glucose, 20 mM HEPES).
The obtained solution was further spun with an ultracentrifuge (rotor 45 Ti; Beckman Coulter) at 11200 rpm for 20 min at 4°C.
The pellet was carefully and quickly aspirated with a Pasteur pipette to avoid mixture with the solution and then diluted in HBM.</p>
<h3 id="preparation-of-astrocytic-and-neuronal-culture">Preparation of astrocytic and neuronal culture</h3>
<p>The procedure has been published before <span class="citation" data-cites="5DZLPDB7">[<a href="#ref-5DZLPDB7" role="doc-biblioref">51</a>]</span>.
Glial cells were ready to be used after 10 days.
Once they were triturated and counted with a Buerker chamber, 100,000 cells/ml were plated onto untreated 12-well plates containing DMEM supplemented with 10% foetal bovine serum (FBS), 10000 IU penicillin, 10 mg streptomycin and 1% MEM non-essential amino acids (DMEM+10% FBS).
Astrocytes were isolated from CD1 outbred mice (P0-P2).
Pups were killed by decapitation and heads were placed in HBSS-HEPES medium (HBSS supplemented with 1 M HEPES).
The cortices were isolated from the brains and the meninges were removed (dura, pia and arachnoid mater).
The cortices were chopped into smaller fragments and transferred to a tube containing 0.25% trypsin dissolved in Dulbecco’s MEM (DMEM).
Fragments were incubated for 15 min at 37°C.
Subsequently, inactivation medium (12.5 mg albumin + 12.5 mg trypsin-inhibitor in DMEM+10% FBS) was added and the tissue washed with HBSS-HEPES.
Tissue was triturated until a smooth cloudy suspension appeared.
Cells were plated in 75 cm<sup>2</sup> flasks with pre-warmed DMEM, one hemisphere per flask, and stored at 37°C with 5% CO<sub>2</sub>. Glial cells were ready to be used after 10 days.
Glial cells were washed with pre-warmed HBSS-HEPES.
Trypsin was added and the flasks were incubated at 37°C for 10 min.
Cells were triturated and counted with a Buerker chamber before plating 100,000 cells/ml on untreated 12-well plates containing DMEM+10% FBS.
After 2 days, neurons were plated.</p>
<p>Hippocampal neurons were isolated from E18 SNAP-25 KO of either sex.
The SNAP-25 KO pups were obtained by pairing two heterozygote animals, and the embryos were recovered at E18 by caesarean section.
Pups were selected based on the absence of motion after tactile stimulation and bloated neck <span class="citation" data-cites="uPPPsUOM">[<a href="#ref-uPPPsUOM" role="doc-biblioref">52</a>]</span>; the genotype was confirmed by PCR in all cases.
The pups were killed by decapitation and heads were put in HBSS-HEPES medium. The cortices were isolated from the brains and the meninges were removed.
The hippocampi were cut from the cortices before being transferred to a tube containing 0.25% trypsin dissolved in HBSS-HEPES solution.
Fragments were incubated for 20 min at 37°C.
Afterwards, the tissue was washed with HBSS-HEPES.
The hippocampi were triturated and the cell count was determined with a Buerker chamber.
20 µl of solution containing 250,000 cells/ml were plated onto the flame sterilized gold R2/2 or R2/1 EM grids as previously described in <span class="citation" data-cites="5DZLPDB7">[<a href="#ref-5DZLPDB7" role="doc-biblioref">51</a>]</span>.
Following a 30-min incubation at 37°C, the grid was transferred into the 12-well plate containing the astrocytes and medium was replaced with NB medium (Neurobasal with 2% B-27, 1 M HEPES, 0.26% lutamax, 14.3 mM β-mercaptoethanol, 10000 IU penicillin, 10 mg streptomycin) for the E18 pups.
Between 4 h and 1 day later, lentiviral particles carrying either SNAP-25-WT, SNAP-25-4E, or SNAP-25-4K constructs were added to the culture <span class="citation" data-cites="AVAZzvH0">[<a href="#ref-AVAZzvH0" role="doc-biblioref">12</a>]</span>.
The cultures were incubated for 12 to 14 days before being plunge frozen.</p>
<h3 id="plunge-freezing-and-spray-mixing">Plunge freezing and spray-mixing</h3>
<p>Rat synaptosomes were prepared for plunge freezing and spray-mixing as follows.
The following steps from incubation to plunge freezing were all done at room temperature (RT), equivalent to 23-25°C.
The synaptosomal solution was incubated with calcein blue AM (#C1429, Molecular Probes-Thermo Fisher Scientific. Waltham, MA, USA) 30 min prior to plunge freezing to visualize the cytosol of functional – esterase containing – cellular compartments such as synaptosomes.
Additionally, 1.3 mM CaCl<sub>2</sub> and 10 nm gold fiducials were added (gold fiducials, #s10110/8. AURION Immuno Gold Reagents &amp; Accessories. Wageningen, The Netherlands).
CaCl<sub>2</sub> is necessary to trigger exocytosis and gold fiducials are important to align the acquired tilt series for tomogram reconstruction.
The sprayed solution contained 1 mM CaCl<sub>2</sub> and 52 mM KCl in HBM to depolarize synaptosomes and trigger exocytosis.
It also contained fluorescein (#46955, Sigma) to trace the spray droplets on the EM grid in cryo-FM.
The synaptosomal solution was applied to a 200-mesh lacey finder carbon film grid (#AGS166-H2. Agar Scientific. Elektron Technology UK Ltd. Stansted, UK).
Excess liquid on the grid was removed by blotting with a filter paper and the grid was immediately plunge frozen in liquid ethane with a homebuilt plunge freezer and was sprayed on the fly.
The plunge freezer and the spraying device (atomizer) were computer controlled with a LabView script (National Instruments Corporation. Mopac Expwy Austin, TX, USA).
The spraying device was set similarly to the device in <span class="citation" data-cites="17b1KeHkw">[<a href="#ref-17b1KeHkw" role="doc-biblioref">30</a>]</span>.
Nitrogen gas pressure necessary to drive spraying was set to 2.5 bar.
The grid was set to pass in front of the spray nozzle at a distance of 3-4 mm.
The plunge freezer was accelerated to 0.75 m/s and the minimum spray delay was ~7 ms.
The atomizer sprays scattered droplets of various size on the EM grid.
During the time lapse between spraying and freezing the content of the droplets spreads by diffusion.
KCl diffuses approximately 4x faster than fluorescein.
Cryo-ET imaging was done within the diffusion distance of KCl but outside of the visible spray droplet because the center of the spray droplet would usually be too thick for imaging.
This reduces the effective stimulation duration to anything between 0 ms and less than the given spray-freeze delay.
Moreover, through diffusion, KCl concentration rapidly rises and then decreases.
Hence synaptosomes are not permanently depolarized.</p>
<p>After 12 to 14 days of incubation grids with mouse neurons were plunge frozen with a Vitrobot (Thermofisher Scientific, Mark IV) with a blot time of 3 s and a blot force of -10.
Wait time and drain time were not used.
Humidity was set to 100% at 4°C.
4 µl undiluted 10 nm BSA gold tracer (Aurion) was added directly onto the grid prior to plunge freezing.</p>
<h3 id="cryo-fluorescence-microscopy">Cryo-fluorescence microscopy</h3>
<p>After plunge freezing, rat synaptsome samples were imaged at the fluorescent microscope under cryo conditions, with a Zeiss Axio Scope.A1, equipped with an AxioCam MRm camera (Carl Zeiss AG, Germany), and a fluorescence lamp (HXP 120 C).
The correlative microscopy stage (#CMS196, Linkam Scientific Instruments, UK) was cooled down to -190°C by liquid nitrogen and the frozen EM grid was placed into the chamber of the cryostage on a bridge that was partially submerged in liquid nitrogen and was close to the objective, where the temperature was around -150°C.
The filter set used for imaging fluorescein was #38 (#000000-1031-346, Zeiss) (BP 470/40, FT 495, BP 525/50; corresponds to GFP) and the one for calcein blue AM was #49 (#488049-9901-000, Zeiss) (G 365, FT 395, BP 445/50; corresponds to DAPI).
The objective used was either a 10x (#420941-9911, NA = 0.25 Ph1, Zeiss) or a 50x (#422472-9900, NA = 0.55 Dic, Zeiss), the acquisition software used was AxioVision (AxioVs40x64 V 4.8.3.0, Zeiss) and the processing software was ZEN lite (Zeiss).</p>
<h3 id="cryo-electron-microscopy">Cryo-electron microscopy</h3>
<p>Following cryo-FM, the rat synaptosome grids were mounted in a cryo-holder (Gatan, Pleasonton, CA, USA) and transferred to a Tecnai F20 (FEI, Eindhoven, The Netherlands) which was set to low dose conditions, operated at 200 kV, and equipped with a field emission gun.
Images were recorded with a 2k x 2k CCD camera (Gatan) mounted after a GIF Tridiem post-column filter (Gatan) operated in zero-loss mode.
The sample was kept at about -180°C.
Tilt series were acquired using SerialEM <span class="citation" data-cites="19ZFerhph">[<a href="#ref-19ZFerhph" role="doc-biblioref">53</a>]</span> for automated acquisition recorded typically from -50° to 50° with a 2° angular increment and an unbinned pixel size of 0.75 or 1.2 nm.
Due to sample thickness (400-700 nm), tomograms were usually not recorded with higher tilt angles.
Defocus was set between -8 to -12 µm and the total electron dose used was about 80-100 e<sup>-</sup>/Å<sup>2</sup>.
Some tomograms were acquired at a Titan Krios equipped with a K2 direct electron detector (Gatan) without energy filter.
The K2 camera was operated in superresolution counting mode and between 8-40 frames per tilt angle were taken.
Tilt series were acquired using the Latitude software (Gatan) for automated acquisition recorded typically from -60° to 60° with a 2° angular increment and an unbinned pixel size of 0.6 nm.
Defocus was set between -8 to -12 µm and the total electron dose used was about 80-100 e<sup>-</sup>/Å<sup>2</sup>.
Prior to image processing the frames at each tilt angle, frames were aligned and averaged in 2dx MC_Automator <span class="citation" data-cites="7UkCfakv">[<a href="#ref-7UkCfakv" role="doc-biblioref">54</a>]</span> with motioncor <span class="citation" data-cites="4rVTIFyw">[<a href="#ref-4rVTIFyw" role="doc-biblioref">55</a>]</span>.
3D reconstruction was done in IMOD <span class="citation" data-cites="136NHHp17">[<a href="#ref-136NHHp17" role="doc-biblioref">56</a>]</span>.
The alignments were done using the automated fiducial tracking function and the 3D reconstructions were done using the weighted back projection followed by a nonlinear anisotropic diffusion (NAD) filtering.
Following tomogram reconstruction only synaptosomes that fulfilled the following criteria were used: 1) even and non-broken PM, 2) synaptic cleft still attached to the presynapse, 3) spherical vesicles, and 4) a mitochondrion in the presynapse necessary to cover the energy demands of the synapse.
These criteria indicate that the synaptosome is functional <span class="citation" data-cites="NtQ7WfPG">[<a href="#ref-NtQ7WfPG" role="doc-biblioref">57</a>]</span>.</p>
<p>Cultured mouse neurons tilt series were acquired at a Titan Krios, equipped with a Falcon 3 direct electron detector (Thermofisher Scientific) without energy filter.
The Falcon camera was operated in linear mode.
Tilt series were acquired using the TEM Tomography software (TFS) for automated acquisition recorded typically from -60° to 60° with a 2° angular increment and an unbinned pixel size of 0.37 nm.
Defocus was set between -6 to -10 µm and the total electron dose used was about 80-100 e<sup>-</sup>/Å<sup>2</sup>. Tomogram reconstruction was done as for synaptosome datasets.</p>
<h3 id="manual-and-automatic-segmentation-procedures">Manual and automatic segmentation procedures</h3>
<p>Manual segmentation of SVs, mitochondria, and the active zone PM was done in IMOD (Supplementary Figure {fig:suppl_tomogram_slices}B and D, Supplementary <a href="vid:synaptosome">Movies S1</a> and <a href="vid:neuron">S2</a>).
The boundary marked the region to be analyzed by Pyto <span class="citation" data-cites="1HtRUUZQi">[<a href="#ref-1HtRUUZQi" role="doc-biblioref">58</a>]</span>.
The analysis by Pyto was essentially the same as described previously <span class="citation" data-cites="XQJ3R1HJ">[<a href="#ref-XQJ3R1HJ" role="doc-biblioref">13</a>]</span> <span class="citation" data-cites="1HtRUUZQi">[<a href="#ref-1HtRUUZQi" role="doc-biblioref">58</a>]</span>.
In short, the segmented area is divided in 1 voxel thick layers parallel to the active zone for distance calculations.
A hierarchical connectivity segmentation detects densities interconnecting vesicles (so-called connectors) and densities connecting vesicles to the active zone PM (so-called tethers) (Supplementary Figure {fig:suppl_tomogram_slices}B and D, Supplementary <a href="vid:synaptosome">Movies S1</a> &amp; <a href="vid:neuron">S2</a>).
Distance calculations are done with the center of the vesicle.
Mainly default settings were used.
The segmentation procedure is conservative and tends to miss some tethers and connectors because of noise.
Consequently, the numbers of tethers and connectors should not be considered as absolute values but rather to compare experimental groups.
All tomograms analyzed by Pyto were obtained on the same microscope with the same tilt range.
The margin of error for false negatives and positives was found to be less than 10% by comparison with ground truth <span class="citation" data-cites="1HtRUUZQi">[<a href="#ref-1HtRUUZQi" role="doc-biblioref">58</a>]</span>.
As it was done before, an upper limit was set between 2100 and 3200 nm³ on segment volume.
The tomograms that were used for this analysis were binned by a factor of 2 to 3, resulting in voxel sizes between 2.1 and 2.4 nm.
Tether and connector length were calculated using the midpoint method <span class="citation" data-cites="1HtRUUZQi">[<a href="#ref-1HtRUUZQi" role="doc-biblioref">58</a>]</span>.
From the stimulated synaptosomes only those that showed visible signs of exocytosis were used for analysis in Pyto.</p>
<h3 id="data-analysis">Data analysis</h3>
<p>If not stated otherwise data in the text are described as mean ± standard error of the mean (SEM).
Wherever possible, data were presented as box plots with the following settings: orange bar, median; box extremities, lower and upper quartiles; whiskers extend up to 1.5 x interquartile range; dots, outliers.
We used the same statistical tests as in <span class="citation" data-cites="XQJ3R1HJ nkmPPJfq">[<a href="#ref-XQJ3R1HJ" role="doc-biblioref">13</a>,<a href="#ref-nkmPPJfq" role="doc-biblioref">31</a>]</span>.
For normal distributed data, the Student’s t-test was used.
In addition, for data that required to be split into discrete bins (e.g. fraction of connected vesicles by distance to active zone), the χ2 test was used.
To calculate the correlation coefficient for paired samples (such as vesicle distance to active zone and number of tethers), Spearman’s rank correlation was used (ρ-test).
The confidence values were calculated using two-tailed tests and were indicated in the graphs by <em>, P&lt;0.05; <strong>, P&lt;0.01; </strong></em>, P&lt;0.001.
We did not apply statistical methods to predetermine sample size but similar sample sizes as previously reported have been used <span class="citation" data-cites="XQJ3R1HJ">[<a href="#ref-XQJ3R1HJ" role="doc-biblioref">13</a>]</span>.
It was not necessary to apply randomization.</p>
<h3 id="manuscript-preparation">Manuscript preparation</h3>
<p>The manuscript was written with the open and collaborative scientific writing package Manubot <span class="citation" data-cites="YuJbg3zO">[<a href="#ref-YuJbg3zO" role="doc-biblioref">59</a>]</span>.
The source code and data for this manuscript are available at <a href="https://github.com/aseedb/synaptic_tomo_ms" class="uri">https://github.com/aseedb/synaptic_tomo_ms</a>.</p>
<h2 class="page_break_before" id="supplementary-material">Supplementary Material</h2>
<h3 id="supplementary-figures">Supplementary Figures</h3>
<div id="fig:suppl_tomogram_slices" class="fignos">
<figure>
<img src="images/tomogram_slices.png" data-tag="S1" style="width:15cm" alt="Figure S1: (A, B) Tomographic slice without (A) and with (B) segmentation of synaptosome with late fusion events. (C,D) Tomographic slice without (C) and with (D) segmentation of WT SNAP-25 neurons. Segmentation colors: off-white = cell outline; pink = active zone; blue = synaptic vesicles; green = mitochondria; yellow = connectors, red = tethers Scale bar, 100 nm" /><figcaption aria-hidden="true"><span>Figure S1:</span> (A, B) Tomographic slice without (A) and with (B) segmentation of synaptosome with late fusion events.
(C,D) Tomographic slice without (C) and with (D) segmentation of WT SNAP-25 neurons.
Segmentation colors: off-white = cell outline; pink = active zone; blue = synaptic vesicles; green = mitochondria; yellow = connectors, red = tethers
Scale bar, 100 nm</figcaption>
</figure>
</div>
<div id="fig:suppl_histograms" class="fignos">
<figure>
<img src="images/supplementary_graphs.png" data-tag="S2" style="width:15cm" alt="Figure S2: (A, B) Histogram of the number of tethers per proximal SV. (C, D) Histogram of the number of connectors per proximal non-RRP SV. (E, F) Histogram of the number of connectors per RRP SV. (G, H) Histogram of connected SV amongst tethered or non-tethered proximal SVs. (I, J) Histogram of connected SV amongst proximal non-RRP or RRP SVs. (A, C, E, G, I) Synapses in mouse cultured neurons. (B, D, F, H, J) Rat synaptosomes." /><figcaption aria-hidden="true"><span>Figure S2:</span> (A, B) Histogram of the number of tethers per proximal SV.
(C, D) Histogram of the number of connectors per proximal non-RRP SV.
(E, F) Histogram of the number of connectors per RRP SV.
(G, H) Histogram of connected SV amongst tethered or non-tethered proximal SVs.
(I, J) Histogram of connected SV amongst proximal non-RRP or RRP SVs.
(A, C, E, G, I) Synapses in mouse cultured neurons.
(B, D, F, H, J) Rat synaptosomes.</figcaption>
</figure>
</div>
<div id="fig:suppl_connected_slices" class="fignos">
<figure>
<img src="images/supplementary_connected_vesicles.png" data-tag="S3" style="width:8cm" alt="Figure S3: (A, B) Tomographic slices showing tethered connected vesicles, blue arrows highlight the connectors. Scale bar, 50 nm" /><figcaption aria-hidden="true"><span>Figure S3:</span> (A, B) Tomographic slices showing tethered connected vesicles, blue arrows highlight the connectors.
Scale bar, 50 nm</figcaption>
</figure>
</div>
<h3 id="supplementary-movies">Supplementary Movies</h3>
<p id="vid:synaptosome">
<video
  controls
  width="100%"
  alt="Tomogram with segmentation of synaptosome with late fusion events"
>
<source
    src="images/spray1_raw2seg3.mp4"
    type="video/mp4"
  >
</video>
<strong>Movie S1: Tomogram with segmentation of synaptosome with late fusion events.</strong>
off-white = cell outline; pink = active zone; blue = synaptic vesicles; green = mitochondria; light green = cytomatrix; yellow = connectors, red = tethers, scale bar 100nm
</p>
<p id="vid:neuron">
<video
  controls
  width="100%"
  alt="Tomogram with segmentation of WT SNAP-25 neurons"
>
<source
    src="images/neuron73_rawseg2.mp4"
    type="video/mp4"
  >
</video>
<strong>Movie S2: Tomogram with segmentation of WT SNAP-25 neurons.</strong>
off-white= cell outline; pink = active zone; blue = synaptic vesicles; green = mitochondria; light green = cytomatrix; yellow = connectors, red = tethers, scale bar 100nm
</p>
<h2 id="author-contributions">Author contributions</h2>
<p>JR, RS, JBS, and BZ designed the study.
JR, RS, and Anna K performed the experiments.
KG provided access to and assistance at one of the Titan Krios microscopes.
JR, RS, and BZ analyzed the data.
VL, UL, and Amin K contributed to the Pyto analysis.
JR, RS, JBS, and BZ wrote the manuscript with contribution from all authors.
JBS and BZ supervised the project.</p>
<h2 id="acknowledgments">Acknowledgments</h2>
<p>We would like to thank Marek Kamínek for maintaining the electron microscope and supporting its use in Bern, and Tillmann Hanns Pape for support in specimen preparation and electron microscope operation in Copenhagen.
Data was acquired on a machine supported by the <a href="https://www.mic.unibe.ch/">Microscopy Imaging Center (MIC) of the University of Bern</a>, a machine supported by the <a href="https://cfim.ku.dk/publications/">Core Facility for Integrated Microscopy (CFIM) of the University of Copenhagen</a>, and a machine supported by the Center for Cellular Imaging and NanoAnalytics, Biozentrum, University of Basel.
This work was funded through the grants mentioned in the author list.</p>
<h2 class="page_break_before" id="references">References</h2>
<!-- Explicitly insert bibliography here -->
<div id="refs" class="references csl-bib-body" role="doc-bibliography">
<div id="ref-qCdQVB7d" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1. </div><div class="csl-right-inline"><strong>Vesicle Docking in Regulated Exocytosis</strong> <div class="csl-block">Matthijs Verhage, Jakob B Sørensen</div> <em>Traffic</em> (2008-09) <a href="https://doi.org/bjtx2n">https://doi.org/bjtx2n</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1111/j.1600-0854.2008.00759.x">10.1111/j.1600-0854.2008.00759.x</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/18445120">18445120</a></div></div>
</div>
<div id="ref-6kHFuPDC" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2. </div><div class="csl-right-inline"><strong>Neurotransmitter Release: The Last Millisecond in the Life of a Synaptic Vesicle</strong> <div class="csl-block">Thomas C Südhof</div> <em>Neuron</em> (2013-10) <a href="https://doi.org/f5gng4">https://doi.org/f5gng4</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.neuron.2013.10.022">10.1016/j.neuron.2013.10.022</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24183019">24183019</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3866025">PMC3866025</a></div></div>
</div>
<div id="ref-ly3VumnN" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">3. </div><div class="csl-right-inline"><strong>The readily releasable pool of synaptic vesicles</strong> <div class="csl-block">Pascal S Kaeser, Wade G Regehr</div> <em>Current Opinion in Neurobiology</em> (2017-04) <a href="https://doi.org/gbkfsd">https://doi.org/gbkfsd</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.conb.2016.12.012">10.1016/j.conb.2016.12.012</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28103533">28103533</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5447466">PMC5447466</a></div></div>
</div>
<div id="ref-f0VSTJjV" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">4. </div><div class="csl-right-inline"><strong>The Morphological and Molecular Nature of Synaptic Vesicle Priming at Presynaptic Active Zones</strong> <div class="csl-block">Cordelia Imig, Sang-Won Min, Stefanie Krinner, Marife Arancillo, Christian Rosenmund, Thomas C Südhof, JeongSeop Rhee, Nils Brose, Benjamin H Cooper</div> <em>Neuron</em> (2014-10) <a href="https://doi.org/gcvj2v">https://doi.org/gcvj2v</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.neuron.2014.10.009">10.1016/j.neuron.2014.10.009</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25374362">25374362</a></div></div>
</div>
<div id="ref-A7bPuSTw" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">5. </div><div class="csl-right-inline"><strong>The Synaptic Vesicle Release Machinery</strong> <div class="csl-block">Josep Rizo, Junjie Xu</div> <em>Annual Review of Biophysics</em> (2015-06-22) <a href="https://doi.org/gjnb9q">https://doi.org/gjnb9q</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1146/annurev-biophys-060414-034057">10.1146/annurev-biophys-060414-034057</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26098518">26098518</a></div></div>
</div>
<div id="ref-qdaYzBLt" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">6. </div><div class="csl-right-inline"><strong>Crystal structure of a SNARE complex involved in synaptic exocytosis at 2.4 Å resolution</strong> <div class="csl-block">RBryan Sutton, Dirk Fasshauer, Reinhard Jahn, Axel T Brunger</div> <em>Nature</em> (1998-09) <a href="https://doi.org/cwkm8k">https://doi.org/cwkm8k</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/26412">10.1038/26412</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/9759724">9759724</a></div></div>
</div>
<div id="ref-2XC5tAoG" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">7. </div><div class="csl-right-inline"><strong>Sequential N- to C-terminal SNARE complex assembly drives priming and fusion of secretory vesicles</strong> <div class="csl-block">Jakob B Sørensen, Katrin Wiederhold, Emil M Müller, Ira Milosevic, Gábor Nagy, Bert L de Groot, Helmut Grubmüller, Dirk Fasshauer</div> <em>The EMBO Journal</em> (2006-02-23) <a href="https://doi.org/dhpwm3">https://doi.org/dhpwm3</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/sj.emboj.7601003">10.1038/sj.emboj.7601003</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/16498411">16498411</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1409717">PMC1409717</a></div></div>
</div>
<div id="ref-1ErVrgCCc" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">8. </div><div class="csl-right-inline"><strong>Single Vesicle Millisecond Fusion Kinetics Reveals Number of SNARE Complexes Optimal for Fast SNARE-mediated Membrane Fusion</strong> <div class="csl-block">Marta K Domanska, Volker Kiessling, Alexander Stein, Dirk Fasshauer, Lukas K Tamm</div> <em>Journal of Biological Chemistry</em> (2009-11) <a href="https://doi.org/chdxqp">https://doi.org/chdxqp</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1074/jbc.m109.047381">10.1074/jbc.m109.047381</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/19759010">19759010</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2797286">PMC2797286</a></div></div>
</div>
<div id="ref-yI8145n3" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">9. </div><div class="csl-right-inline"><strong>Fast Vesicle Fusion in Living Cells Requires at Least Three SNARE Complexes</strong> <div class="csl-block">Ralf Mohrmann, Heidi de Wit, Matthijs Verhage, Erwin Neher, Jakob B Sørensen</div> <em>Science</em> (2010-10-22) <a href="https://doi.org/c7q87p">https://doi.org/c7q87p</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1126/science.1193134">10.1126/science.1193134</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20847232">20847232</a></div></div>
</div>
<div id="ref-12xIhQNRh" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">10. </div><div class="csl-right-inline"><strong>SNARE Proteins: One to Fuse and Three to Keep the Nascent Fusion Pore Open</strong> <div class="csl-block">Lei Shi, Qing-Tao Shen, Alexander Kiel, Jing Wang, Hong-Wei Wang, Thomas J Melia, James E Rothman, Frédéric Pincet</div> <em>Science</em> (2012-03-16) <a href="https://doi.org/gk8gpf">https://doi.org/gk8gpf</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1126/science.1214984">10.1126/science.1214984</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/22422984">22422984</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3736847">PMC3736847</a></div></div>
</div>
<div id="ref-qeT2GTd8" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">11. </div><div class="csl-right-inline"><strong>Membrane Curvature in Synaptic Vesicle Fusion and Beyond</strong> <div class="csl-block">Harvey T McMahon, Michael M Kozlov, Sascha Martens</div> <em>Cell</em> (2010-03) <a href="https://doi.org/d3gmvv">https://doi.org/d3gmvv</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.cell.2010.02.017">10.1016/j.cell.2010.02.017</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20211126">20211126</a></div></div>
</div>
<div id="ref-AVAZzvH0" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">12. </div><div class="csl-right-inline"><strong>An Electrostatic Energy Barrier for SNARE-Dependent Spontaneous and Evoked Synaptic Transmission</strong> <div class="csl-block">Marvin Ruiter, Anna Kádková, Andrea Scheutzow, Jörg Malsam, Thomas H Söllner, Jakob B Sørensen</div> <em>Cell Reports</em> (2019-02) <a href="https://doi.org/gfv5gd">https://doi.org/gfv5gd</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.celrep.2019.01.103">10.1016/j.celrep.2019.01.103</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30811985">30811985</a></div></div>
</div>
<div id="ref-XQJ3R1HJ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">13. </div><div class="csl-right-inline"><strong>Quantitative analysis of the native presynaptic cytomatrix by cryoelectron tomography</strong> <div class="csl-block">Rubén Fernández-Busnadiego, Benoît Zuber, Ulrike Elisabeth Maurer, Marek Cyrklaff, Wolfgang Baumeister, Vladan Lučić</div> <em>Journal of Cell Biology</em> (2010-01-11) <a href="https://doi.org/b9c26b">https://doi.org/b9c26b</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1083/jcb.200908082">10.1083/jcb.200908082</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20065095">20065095</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2812849">PMC2812849</a></div></div>
</div>
<div id="ref-K10hpz3n" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">14. </div><div class="csl-right-inline"><strong>Molecular architecture of the presynaptic terminal</strong> <div class="csl-block">Benoît Zuber, Vladan Lučić</div> <em>Current Opinion in Structural Biology</em> (2019-02) <a href="https://doi.org/gk8gpd">https://doi.org/gk8gpd</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.sbi.2019.01.008">10.1016/j.sbi.2019.01.008</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30925443">30925443</a></div></div>
</div>
<div id="ref-4Xeb8inT" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">15. </div><div class="csl-right-inline"><strong>Definition of the Readily Releasable Pool of Vesicles at Hippocampal Synapses</strong> <div class="csl-block">Christian Rosenmund, Charles F Stevens</div> <em>Neuron</em> (1996-06) <a href="https://doi.org/c76twv">https://doi.org/c76twv</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/s0896-6273(00)80146-4">10.1016/s0896-6273(00)80146-4</a></div></div>
</div>
<div id="ref-UZqYppzb" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">16. </div><div class="csl-right-inline"><strong>Properties of Synaptic Vesicle Pools in Mature Central Nerve Terminals</strong> <div class="csl-block">Anthony C Ashton, Yuri A Ushkaryov</div> <em>Journal of Biological Chemistry</em> (2005-11) <a href="https://doi.org/cb5883">https://doi.org/cb5883</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1074/jbc.m504137200">10.1074/jbc.m504137200</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/16148008">16148008</a></div></div>
</div>
<div id="ref-QkLxmBRs" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">17. </div><div class="csl-right-inline"><strong>Vesicle release probability and pre‐primed pool at glutamatergic synapses in area CA1 of the rat neonatal hippocampus</strong> <div class="csl-block">Eric Hanse, Bengt Gustafsson</div> <em>The Journal of Physiology</em> (2001-03) <a href="https://doi.org/fq7th3">https://doi.org/fq7th3</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1111/j.1469-7793.2001.0481i.x">10.1111/j.1469-7793.2001.0481i.x</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/11230520">11230520</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2278469">PMC2278469</a></div></div>
</div>
<div id="ref-ff66mwER" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">18. </div><div class="csl-right-inline"><strong>Reluctant Vesicles Contribute to the Total Readily Releasable Pool in Glutamatergic Hippocampal Neurons</strong> <div class="csl-block">KL Moulder</div> <em>Journal of Neuroscience</em> (2005-04-13) <a href="https://doi.org/dwqpxj">https://doi.org/dwqpxj</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1523/jneurosci.5231-04.2005">10.1523/jneurosci.5231-04.2005</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/15829636">15829636</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6724923">PMC6724923</a></div></div>
</div>
<div id="ref-XzTcFx4M" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">19. </div><div class="csl-right-inline"><strong>Synaptic vesicle pools: an update</strong> <div class="csl-block">Denker</div> <em>Frontiers in Synaptic Neuroscience</em> (2010) <a href="https://doi.org/d74dd7">https://doi.org/d74dd7</a> <div class="csl-block">DOI: <a href="https://doi.org/10.3389/fnsyn.2010.00135">10.3389/fnsyn.2010.00135</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21423521">21423521</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3059705">PMC3059705</a></div></div>
</div>
<div id="ref-1MRtACTR" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">20. </div><div class="csl-right-inline"><strong>The cytoskeletal architecture of the presynaptic terminal and molecular structure of synapsin 1.</strong> <div class="csl-block">N Hirokawa, K Sobue, K Kanda, A Harada, H Yorifuji</div> <em>Journal of Cell Biology</em> (1989-01-01) <a href="https://doi.org/ddbqhb">https://doi.org/ddbqhb</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1083/jcb.108.1.111">10.1083/jcb.108.1.111</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/2536030">2536030</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2115350">PMC2115350</a></div></div>
</div>
<div id="ref-VYo5PIO" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">21. </div><div class="csl-right-inline"><strong>Three-Dimensional Architecture of Presynaptic Terminal Cytomatrix</strong> <div class="csl-block">L Siksou, P Rostaing, J-P Lechaire, T Boudier, T Ohtsuka, A Fejtova, H-T Kao, P Greengard, ED Gundelfinger, A Triller, S Marty</div> <em>Journal of Neuroscience</em> (2007-06-27) <a href="https://doi.org/bjw3mv">https://doi.org/bjw3mv</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1523/jneurosci.1773-07.2007">10.1523/jneurosci.1773-07.2007</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/17596435">17596435</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6672225">PMC6672225</a></div></div>
</div>
<div id="ref-S9w67bLc" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">22. </div><div class="csl-right-inline"><strong>Physical determinants of vesicle mobility and supply at a central synapse</strong> <div class="csl-block">Jason Seth Rothman, Laszlo Kocsis, Etienne Herzog, Zoltan Nusser, Robin Angus Silver</div> <em>eLife</em> (2016-08-19) <a href="https://doi.org/f9rmkt">https://doi.org/f9rmkt</a> <div class="csl-block">DOI: <a href="https://doi.org/10.7554/elife.15133">10.7554/elife.15133</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27542193">27542193</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5025287">PMC5025287</a></div></div>
</div>
<div id="ref-2nauTxqh" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">23. </div><div class="csl-right-inline"><strong>Activity-Dependence of Synaptic Vesicle Dynamics</strong> <div class="csl-block">Luca A Forte, Michael W Gramlich, Vitaly A Klyachko</div> <em>The Journal of Neuroscience</em> (2017-09-27) <a href="https://doi.org/gch7j3">https://doi.org/gch7j3</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1523/jneurosci.0383-17.2017">10.1523/jneurosci.0383-17.2017</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28954868">28954868</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5666583">PMC5666583</a></div></div>
</div>
<div id="ref-if88f9zh" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">24. </div><div class="csl-right-inline"><strong>Synapsin dispersion and reclustering during synaptic activity</strong> <div class="csl-block">Ping Chi, Paul Greengard, Timothy A Ryan</div> <em>Nature Neuroscience</em> (2001-10-29) <a href="https://doi.org/cpdwc7">https://doi.org/cpdwc7</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/nn756">10.1038/nn756</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/11685225">11685225</a></div></div>
</div>
<div id="ref-43WYZu5a" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">25. </div><div class="csl-right-inline"><strong>Interactions of synapsin I with small synaptic vesicles: distinct sites in synapsin I bind to vesicle phospholipids and vesicle proteins.</strong> <div class="csl-block">F Benfenati, M Bähler, R Jahn, P Greengard</div> <em>Journal of Cell Biology</em> (1989-05-01) <a href="https://doi.org/d65t4v">https://doi.org/d65t4v</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1083/jcb.108.5.1863">10.1083/jcb.108.5.1863</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/2497106">2497106</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2115532">PMC2115532</a></div></div>
</div>
<div id="ref-1AX9UEqJU" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">26. </div><div class="csl-right-inline"><strong>Visualization of Synaptic Vesicle Movement in Intact Synaptic Boutons Using Fluorescence Fluctuation Spectroscopy</strong> <div class="csl-block">Randolf Jordan, Edward A Lemke, Jurgen Klingauf</div> <em>Biophysical Journal</em> (2005-09) <a href="https://doi.org/cdpkkv">https://doi.org/cdpkkv</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1529/biophysj.105.061663">10.1529/biophysj.105.061663</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/15980175">15980175</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1366711">PMC1366711</a></div></div>
</div>
<div id="ref-9xDm8GpQ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">27. </div><div class="csl-right-inline"><strong>Synapsin Selectively Controls the Mobility of Resting Pool Vesicles at Hippocampal Terminals</strong> <div class="csl-block">A Orenbuch, L Shalev, V Marra, I Sinai, Y Lavy, J Kahn, JJ Burden, K Staras, D Gitler</div> <em>Journal of Neuroscience</em> (2012-03-21) <a href="https://doi.org/f3wq83">https://doi.org/f3wq83</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1523/jneurosci.5058-11.2012">10.1523/jneurosci.5058-11.2012</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/22442064">22442064</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3492757">PMC3492757</a></div></div>
</div>
<div id="ref-BuOLXzHq" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">28. </div><div class="csl-right-inline"><strong>High Mobility of Vesicles Supports Continuous Exocytosis at a Ribbon Synapse</strong> <div class="csl-block">Matthew Holt, Anne Cooke, Andreas Neef, Leon Lagnado</div> <em>Current Biology</em> (2004-02) <a href="https://doi.org/b9g9h2">https://doi.org/b9g9h2</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.cub.2003.12.053">10.1016/j.cub.2003.12.053</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/14761649">14761649</a></div></div>
</div>
<div id="ref-ySZJ84X1" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">29. </div><div class="csl-right-inline"><strong>Correlative microscopy: Bridging the gap between fluorescence light microscopy and cryo-electron tomography</strong> <div class="csl-block">Anna Sartori, Rudolf Gatz, Florian Beck, Alexander Rigort, Wolfgang Baumeister, Juergen M Plitzko</div> <em>Journal of Structural Biology</em> (2007-11) <a href="https://doi.org/cp5krz">https://doi.org/cp5krz</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.jsb.2007.07.011">10.1016/j.jsb.2007.07.011</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/17884579">17884579</a></div></div>
</div>
<div id="ref-17b1KeHkw" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">30. </div><div class="csl-right-inline"><strong>Analysis of transient structures by cryo-microscopy combined with rapid mixing of spray droplets</strong> <div class="csl-block">John Berriman, Nigel Unwin</div> <em>Ultramicroscopy</em> (1994-12) <a href="https://doi.org/ctwp5j">https://doi.org/ctwp5j</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/0304-3991(94)90012-4">10.1016/0304-3991(94)90012-4</a></div></div>
</div>
<div id="ref-nkmPPJfq" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">31. </div><div class="csl-right-inline"><strong>Cryo–electron tomography reveals a critical role of RIM1α in synaptic vesicle tethering</strong> <div class="csl-block">Rubén Fernández-Busnadiego, Shoh Asano, Ana-Maria Oprisoreanu, Eri Sakata, Michael Doengi, Zdravko Kochovski, Magdalena Zürner, Valentin Stein, Susanne Schoch, Wolfgang Baumeister, Vladan Lučić</div> <em>Journal of Cell Biology</em> (2013-05-27) <a href="https://doi.org/f4x2mj">https://doi.org/f4x2mj</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1083/jcb.201206063">10.1083/jcb.201206063</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23712261">23712261</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3664715">PMC3664715</a></div></div>
</div>
<div id="ref-PB9esFqe" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">32. </div><div class="csl-right-inline"><strong>&lt;scp&gt;SNARE&lt;/scp&gt;
and regulatory proteins induce local membrane protrusions to prime docked vesicles for fast calcium‐triggered fusion</strong> <div class="csl-block">Tanmay AM Bharat, Jörg Malsam, Wim JH Hagen, Andrea Scheutzow, Thomas H Söllner, John AG Briggs</div> <em>EMBO reports</em> (2014-02-03) <a href="https://doi.org/f2qg4d">https://doi.org/f2qg4d</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/embr.201337807">10.1002/embr.201337807</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24493260">24493260</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3989697">PMC3989697</a></div></div>
</div>
<div id="ref-6ISfFKuj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">33. </div><div class="csl-right-inline"><strong>Phosphatidylinositol 4,5 Bisphosphate Controls the cis and trans Interactions of Synaptotagmin 1</strong> <div class="csl-block">Sarah B Nyenhuis, Anusa Thapa, David S Cafiso</div> <em>Biophysical Journal</em> (2019-07) <a href="https://doi.org/gnk9cp">https://doi.org/gnk9cp</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.bpj.2019.06.016">10.1016/j.bpj.2019.06.016</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31301806">31301806</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6701194">PMC6701194</a></div></div>
</div>
<div id="ref-1HOYj7n0K" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">34. </div><div class="csl-right-inline"><strong>Single Reconstituted Neuronal SNARE Complexes Zipper in Three Distinct Stages</strong> <div class="csl-block">Ying Gao, Sylvain Zorman, Gregory Gundersen, Zhiqun Xi, Lu Ma, George Sirinakis, James E Rothman, Yongli Zhang</div> <em>Science</em> (2012-09-14) <a href="https://doi.org/f37csk">https://doi.org/f37csk</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1126/science.1224492">10.1126/science.1224492</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/22903523">22903523</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3677750">PMC3677750</a></div></div>
</div>
<div id="ref-8Axaq8nk" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">35. </div><div class="csl-right-inline"><strong>Molecular mechanism of fusion pore formation driven by the neuronal SNARE complex</strong> <div class="csl-block">Satyan Sharma, Manfred Lindau</div> <em>Proceedings of the National Academy of Sciences</em> (2018-11-27) <a href="https://doi.org/gfsd3m">https://doi.org/gfsd3m</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1073/pnas.1816495115">10.1073/pnas.1816495115</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30482862">30482862</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6294955">PMC6294955</a></div></div>
</div>
<div id="ref-TrqX98bo" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">36. </div><div class="csl-right-inline"><strong>Caught in the Act: Visualization of SNARE-Mediated Fusion Events in Molecular Detail</strong> <div class="csl-block">Herre Jelger Risselada, Carsten Kutzner, Helmut Grubmüller</div> <em>ChemBioChem</em> (2011-03-23) <a href="https://doi.org/bwr34v">https://doi.org/bwr34v</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/cbic.201100020">10.1002/cbic.201100020</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21433241">21433241</a></div></div>
</div>
<div id="ref-q1FEacfR" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">37. </div><div class="csl-right-inline"><strong>Synaptotagmin-1 may be a distance regulator acting upstream of SNARE nucleation</strong> <div class="csl-block">Geert van den Bogaart, Shashi Thutupalli, Jelger H Risselada, Karsten Meyenberg, Matthew Holt, Dietmar Riedel, Ulf Diederichsen, Stephan Herminghaus, Helmut Grubmüller, Reinhard Jahn</div> <em>Nature Structural &amp;amp; Molecular Biology</em> (2011-06-05) <a href="https://doi.org/dh26xp">https://doi.org/dh26xp</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/nsmb.2061">10.1038/nsmb.2061</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21642968">21642968</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3130798">PMC3130798</a></div></div>
</div>
<div id="ref-dh8gQ3IE" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">38. </div><div class="csl-right-inline"><strong>Synaptobrevin N-terminally bound to syntaxin–SNAP-25 defines the primed vesicle state in regulated exocytosis</strong> <div class="csl-block">Alexander M Walter, Katrin Wiederhold, Dieter Bruns, Dirk Fasshauer, Jakob B Sørensen</div> <em>Journal of Cell Biology</em> (2010-02-08) <a href="https://doi.org/bsp5dq">https://doi.org/bsp5dq</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1083/jcb.200907018">10.1083/jcb.200907018</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20142423">20142423</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2819690">PMC2819690</a></div></div>
</div>
<div id="ref-me9lnSLC" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">39. </div><div class="csl-right-inline"><strong>Opposing functions of two sub-domains of the SNARE-complex in neurotransmission</strong> <div class="csl-block">Jens P Weber, Kerstin Reim, Jakob B Sørensen</div> <em>The EMBO Journal</em> (2010-06-18) <a href="https://doi.org/dzkrh2">https://doi.org/dzkrh2</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/emboj.2010.130">10.1038/emboj.2010.130</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20562829">20562829</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2928689">PMC2928689</a></div></div>
</div>
<div id="ref-4GAxfKCY" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">40. </div><div class="csl-right-inline"><strong>Dynamics and number of trans-SNARE complexes determine nascent fusion pore properties</strong> <div class="csl-block">Huan Bao, Debasis Das, Nicholas A Courtney, Yihao Jiang, Joseph S Briguglio, Xiaochu Lou, Daniel Roston, Qiang Cui, Baron Chanda, Edwin R Chapman</div> <em>Nature</em> (2018-01-31) <a href="https://doi.org/gczkdz">https://doi.org/gczkdz</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/nature25481">10.1038/nature25481</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29420480">29420480</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5808578">PMC5808578</a></div></div>
</div>
<div id="ref-GwRRPuyV" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">41. </div><div class="csl-right-inline"><strong>Superpriming of synaptic vesicles as a common basis for intersynapse variability and modulation of synaptic strength</strong> <div class="csl-block">Holger Taschenberger, Andrew Woehler, Erwin Neher</div> <em>Proceedings of the National Academy of Sciences</em> (2016-07-18) <a href="https://doi.org/f8w9vc">https://doi.org/f8w9vc</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1073/pnas.1606383113">10.1073/pnas.1606383113</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27432975">27432975</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4978258">PMC4978258</a></div></div>
</div>
<div id="ref-HUxTgHP7" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">42. </div><div class="csl-right-inline"><strong>Superpriming of synaptic vesicles after their recruitment to the readily releasable pool</strong> <div class="csl-block">Jae Sung Lee, Won-Kyung Ho, Erwin Neher, Suk-Ho Lee</div> <em>Proceedings of the National Academy of Sciences</em> (2013-08-26) <a href="https://doi.org/f494g4">https://doi.org/f494g4</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1073/pnas.1314427110">10.1073/pnas.1314427110</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23980146">23980146</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3773744">PMC3773744</a></div></div>
</div>
<div id="ref-mKOZu7Ak" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">43. </div><div class="csl-right-inline"><strong>Merits and Limitations of Vesicle Pool Models in View of Heterogeneous Populations of Synaptic Vesicles</strong> <div class="csl-block">Erwin Neher</div> <em>Neuron</em> (2015-09) <a href="https://doi.org/f7whxq">https://doi.org/f7whxq</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.neuron.2015.08.038">10.1016/j.neuron.2015.08.038</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26402599">26402599</a></div></div>
</div>
<div id="ref-pGojwbzO" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">44. </div><div class="csl-right-inline"><strong>Rapid regulation of vesicle priming explains synaptic facilitation despite heterogeneous vesicle:Ca2+ channel distances</strong> <div class="csl-block">Janus RL Kobbersmed, Andreas T Grasskamp, Meida Jusyte, Mathias A Böhme, Susanne Ditlevsen, Jakob Balslev Sørensen, Alexander M Walter</div> <em>eLife</em> (2020-02-20) <a href="https://doi.org/gnf2f5">https://doi.org/gnf2f5</a> <div class="csl-block">DOI: <a href="https://doi.org/10.7554/elife.51032">10.7554/elife.51032</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32077852">32077852</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7145420">PMC7145420</a></div></div>
</div>
<div id="ref-gmOeAcFB" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">45. </div><div class="csl-right-inline"><strong>Control of Presynaptic Parallel Fiber Efficacy by Activity-Dependent Regulation of the Number of Occupied Release Sites</strong> <div class="csl-block">Hartmut Schmidt</div> <em>Frontiers in Systems Neuroscience</em> (2019-07-17) <a href="https://doi.org/gnk9cw">https://doi.org/gnk9cw</a> <div class="csl-block">DOI: <a href="https://doi.org/10.3389/fnsys.2019.00030">10.3389/fnsys.2019.00030</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31379524">31379524</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6650762">PMC6650762</a></div></div>
</div>
<div id="ref-a4RbcT3S" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">46. </div><div class="csl-right-inline"><strong>Calcium-dependent docking of synaptic vesicles</strong> <div class="csl-block">Melissa Silva, Van Tran, Alain Marty</div> <em>Trends in Neurosciences</em> (2021-07) <a href="https://doi.org/gkzxh9">https://doi.org/gkzxh9</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.tins.2021.04.003">10.1016/j.tins.2021.04.003</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/34049722">34049722</a></div></div>
</div>
<div id="ref-QTV8mtNy" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">47. </div><div class="csl-right-inline"><strong>Synaptic vesicles transiently dock to refill release sites</strong> <div class="csl-block">Grant F Kusick, Morven Chin, Sumana Raychaudhuri, Kristina Lippmann, Kadidia P Adula, Edward J Hujber, Thien Vu, MWayne Davis, Erik M Jorgensen, Shigeki Watanabe</div> <em>Nature Neuroscience</em> (2020-09-28) <a href="https://doi.org/gmqhk9">https://doi.org/gmqhk9</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/s41593-020-00716-1">10.1038/s41593-020-00716-1</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32989294">32989294</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8054220">PMC8054220</a></div></div>
</div>
<div id="ref-13WUlCCxy" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">48. </div><div class="csl-right-inline"><strong>Actin- and Myosin-Dependent Vesicle Loading of Presynaptic Docking Sites Prior to Exocytosis</strong> <div class="csl-block">Takafumi Miki, Gerardo Malagon, Camila Pulido, Isabel Llano, Erwin Neher, Alain Marty</div> <em>Neuron</em> (2016-08) <a href="https://doi.org/f8z4sn">https://doi.org/f8z4sn</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.neuron.2016.07.033">10.1016/j.neuron.2016.07.033</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27537485">27537485</a></div></div>
</div>
<div id="ref-n3fuhRE" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">49. </div><div class="csl-right-inline"><strong>Differential Abilities of SNAP-25 Homologs to Support Neuronal Function</strong> <div class="csl-block">I Delgado-Martinez, RB Nehring, JB Sorensen</div> <em>Journal of Neuroscience</em> (2007-08-29) <a href="https://doi.org/d7mhxn">https://doi.org/d7mhxn</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1523/jneurosci.5092-06.2007">10.1523/jneurosci.5092-06.2007</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/17728451">17728451</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6673127">PMC6673127</a></div></div>
</div>
<div id="ref-7sGZgtQa" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">50. </div><div class="csl-right-inline"><strong>A rapid Percoll gradient procedure for preparation of synaptosomes</strong> <div class="csl-block">Peter R Dunkley, Paula E Jarvie, Phillip J Robinson</div> <em>Nature Protocols</em> (2008-10-16) <a href="https://doi.org/b7zwh8">https://doi.org/b7zwh8</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/nprot.2008.171">10.1038/nprot.2008.171</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/18927557">18927557</a></div></div>
</div>
<div id="ref-5DZLPDB7" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">51. </div><div class="csl-right-inline"><strong>Preparation of Primary Neurons for Visualizing Neurites in a Frozen-hydrated State Using Cryo-Electron Tomography</strong> <div class="csl-block">Sarah H Shahmoradian, Mauricio R Galiano, Chengbiao Wu, Shurui Chen, Matthew N Rasband, William C Mobley, Wah Chiu</div> <em>Journal of Visualized Experiments</em> (2014-02-12) <a href="https://doi.org/gmh9w3">https://doi.org/gmh9w3</a> <div class="csl-block">DOI: <a href="https://doi.org/10.3791/50783">10.3791/50783</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24561719">24561719</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4089403">PMC4089403</a></div></div>
</div>
<div id="ref-uPPPsUOM" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">52. </div><div class="csl-right-inline"><strong>Genetic ablation of the t-SNARE SNAP-25 distinguishes mechanisms of neuroexocytosis</strong> <div class="csl-block">Philip Washbourne, Peter M Thompson, Mario Carta, Edmar T Costa, James R Mathews, Guillermina Lopez-Benditó, Zoltán Molnár, Mark W Becher, CFernando Valenzuela, LDonald Partridge, Michael C Wilson</div> <em>Nature Neuroscience</em> (2001-12-19) <a href="https://doi.org/fbmxgc">https://doi.org/fbmxgc</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/nn783">10.1038/nn783</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/11753414">11753414</a></div></div>
</div>
<div id="ref-19ZFerhph" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">53. </div><div class="csl-right-inline"><strong>Automated electron microscope tomography using robust prediction of specimen movements</strong> <div class="csl-block">David N Mastronarde</div> <em>Journal of Structural Biology</em> (2005-10) <a href="https://doi.org/ff7gzx">https://doi.org/ff7gzx</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.jsb.2005.07.007">10.1016/j.jsb.2005.07.007</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/16182563">16182563</a></div></div>
</div>
<div id="ref-7UkCfakv" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">54. </div><div class="csl-right-inline"><strong>2dx_automator: Implementation of a semiautomatic high-throughput high-resolution cryo-electron crystallography pipeline</strong> <div class="csl-block">Sebastian Scherer, Julia Kowal, Mohamed Chami, Venkata Dandey, Marcel Arheit, Philippe Ringler, Henning Stahlberg</div> <em>Journal of Structural Biology</em> (2014-05) <a href="https://doi.org/f522h3">https://doi.org/f522h3</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.jsb.2014.03.016">10.1016/j.jsb.2014.03.016</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24680783">24680783</a></div></div>
</div>
<div id="ref-4rVTIFyw" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">55. </div><div class="csl-right-inline"><strong>Electron counting and beam-induced motion correction enable near-atomic-resolution single-particle cryo-EM</strong> <div class="csl-block">Xueming Li, Paul Mooney, Shawn Zheng, Christopher R Booth, Michael B Braunfeld, Sander Gubbens, David A Agard, Yifan Cheng</div> <em>Nature Methods</em> (2013-05-05) <a href="https://doi.org/f4zpjf">https://doi.org/f4zpjf</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/nmeth.2472">10.1038/nmeth.2472</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23644547">23644547</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3684049">PMC3684049</a></div></div>
</div>
<div id="ref-136NHHp17" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">56. </div><div class="csl-right-inline"><strong>Computer Visualization of Three-Dimensional Image Data Using IMOD</strong> <div class="csl-block">James R Kremer, David N Mastronarde, JRichard McIntosh</div> <em>Journal of Structural Biology</em> (1996-01) <a href="https://doi.org/d9nfzw">https://doi.org/d9nfzw</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1006/jsbi.1996.0013">10.1006/jsbi.1996.0013</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/8742726">8742726</a></div></div>
</div>
<div id="ref-NtQ7WfPG" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">57. </div><div class="csl-right-inline"><strong>A rapid Percoll gradient procedure for isolation of synaptosomes directly from an S1 fraction: viability of subcellular fractions</strong> <div class="csl-block">Steven M Harrison, Paula E Jarvie, Peter R Dunkley</div> <em>Brain Research</em> (1988-02) <a href="https://doi.org/b5tzcr">https://doi.org/b5tzcr</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/0006-8993(88)91384-4">10.1016/0006-8993(88)91384-4</a></div></div>
</div>
<div id="ref-1HtRUUZQi" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">58. </div><div class="csl-right-inline"><strong>Hierarchical detection and analysis of macromolecular complexes in cryo-electron tomograms using Pyto software</strong> <div class="csl-block">Vladan Lučić, Rubén Fernández-Busnadiego, Ulrike Laugks, Wolfgang Baumeister</div> <em>Journal of Structural Biology</em> (2016-12) <a href="https://doi.org/f9d5t2">https://doi.org/f9d5t2</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.jsb.2016.10.004">10.1016/j.jsb.2016.10.004</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27742578">27742578</a></div></div>
</div>
<div id="ref-YuJbg3zO" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">59. </div><div class="csl-right-inline"><strong>Open collaborative writing with Manubot</strong> <div class="csl-block">Daniel S Himmelstein, Vincent Rubinetti, David R Slochower, Dongbo Hu, Venkat S Malladi, Casey S Greene, Anthony Gitter</div> <em>PLOS Computational Biology</em> (2019-06-24) <a href="https://doi.org/c7np">https://doi.org/c7np</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1371/journal.pcbi.1007128">10.1371/journal.pcbi.1007128</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31233491">31233491</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6611653">PMC6611653</a></div></div>
</div>
</div>
<!-- default theme -->

<style>
  /* import google fonts */
  @import url("https://fonts.googleapis.com/css?family=Open+Sans:400,600,700");
  @import url("https://fonts.googleapis.com/css?family=Source+Code+Pro");

  /* -------------------------------------------------- */
  /* global */
  /* -------------------------------------------------- */

  /* all elements */
  * {
    /* force sans-serif font unless specified otherwise */
    font-family: "Open Sans", "Helvetica", sans-serif;

    /* prevent text inflation on some mobile browsers */
    -webkit-text-size-adjust: none !important;
    -moz-text-size-adjust: none !important;
    -o-text-size-adjust: none !important;
    text-size-adjust: none !important;
  }

  @media only screen {
    /* "page" element */
    body {
      position: relative;
      box-sizing: border-box;
      font-size: 12pt;
      line-height: 1.5;
      max-width: 8.5in;
      margin: 20px auto;
      padding: 40px;
      border-radius: 5px;
      border: solid 1px #bdbdbd;
      box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
      background: #ffffff;
    }
  }

  /* when on screen < 8.5in wide */
  @media only screen and (max-width: 8.5in) {
    /* "page" element */
    body {
      padding: 20px;
      margin: 0;
      border-radius: 0;
      border: none;
      box-shadow: 0 0 20px rgba(0, 0, 0, 0.05) inset;
      background: none;
    }
  }

  /* -------------------------------------------------- */
  /* headings */
  /* -------------------------------------------------- */

  /* all headings */
  h1,
  h2,
  h3,
  h4,
  h5,
  h6 {
    margin: 20px 0;
    padding: 0;
    font-weight: bold;
  }

  /* biggest heading */
  h1 {
    margin: 40px 0;
    text-align: center;
  }

  /* second biggest heading */
  h2 {
    margin-top: 30px;
    padding-bottom: 5px;
    border-bottom: solid 1px #bdbdbd;
  }

  /* heading font sizes */
  h1 {
    font-size: 2em;
  }
  h2 {
    font-size: 1.5em;
  }
  h3 {
    font-size: 1.35em;
  }
  h4 {
    font-size: 1.25em;
  }
  h5 {
    font-size: 1.15em;
  }
  h6 {
    font-size: 1em;
  }

  /* -------------------------------------------------- */
  /* manuscript header */
  /* -------------------------------------------------- */

  /* manuscript title */
  header > h1 {
    margin: 0;
  }

  /* manuscript title caption text (ie "automatically generated on") */
  header + p {
    text-align: center;
    margin-top: 10px;
  }

  /* -------------------------------------------------- */
  /* text elements */
  /* -------------------------------------------------- */

  /* links */
  a {
    color: #2196f3;
    overflow-wrap: break-word;
  }

  /* superscripts and subscripts */
  sub,
  sup {
    /* prevent from affecting line height */
    line-height: 0;
  }

  /* unordered and ordered lists*/
  ul,
  ol {
    padding-left: 20px;
  }

  /* class for styling text semibold */
  .semibold {
    font-weight: 600;
  }

  /* class for styling elements horizontally left aligned */
  .left {
    display: block;
    text-align: left;
    margin-left: auto;
    margin-right: 0;
    justify-content: left;
  }

  /* class for styling elements horizontally centered */
  .center {
    display: block;
    text-align: center;
    margin-left: auto;
    margin-right: auto;
    justify-content: center;
  }

  /* class for styling elements horizontally right aligned */
  .right {
    display: block;
    text-align: right;
    margin-left: 0;
    margin-right: auto;
    justify-content: right;
  }

  /* -------------------------------------------------- */
  /* section elements */
  /* -------------------------------------------------- */

  /* horizontal divider line */
  hr {
    border: none;
    height: 1px;
    background: #bdbdbd;
  }

  /* paragraphs, horizontal dividers, figures, tables, code */
  p,
  hr,
  figure,
  table,
  pre {
    /* treat all as "paragraphs", with consistent vertical margins */
    margin-top: 20px;
    margin-bottom: 20px;
  }

  /* -------------------------------------------------- */
  /* figures */
  /* -------------------------------------------------- */

  /* figure */
  figure {
    max-width: 100%;
    margin-left: auto;
    margin-right: auto;
  }

  /* figure caption */
  figcaption {
    padding: 0;
    padding-top: 10px;
  }

  /* figure image element */
  figure > img,
  figure > svg {
    max-width: 100%;
    display: block;
    margin-left: auto;
    margin-right: auto;
  }

  /* figure auto-number */
  img + figcaption > span:first-of-type,
  svg + figcaption > span:first-of-type {
    font-weight: bold;
    margin-right: 5px;
  }

  /* -------------------------------------------------- */
  /* tables */
  /* -------------------------------------------------- */

  /* table */
  table {
    border-collapse: collapse;
    border-spacing: 0;
    width: 100%;
    margin-left: auto;
    margin-right: auto;
  }

  /* table cells */
  th,
  td {
    border: solid 1px #bdbdbd;
    padding: 10px;
    /* squash table if too wide for page by forcing line breaks */
    overflow-wrap: break-word;
    word-break: break-word;
  }

  /* header row and even rows */
  th,
  tr:nth-child(2n) {
    background-color: #fafafa;
  }

  /* odd rows */
  tr:nth-child(2n + 1) {
    background-color: #ffffff;
  }

  /* table caption */
  caption {
    text-align: left;
    padding: 0;
    padding-bottom: 10px;
  }

  /* table auto-number */
  table > caption > span:first-of-type {
    font-weight: bold;
    margin-right: 5px;
  }

  /* -------------------------------------------------- */
  /* code */
  /* -------------------------------------------------- */

  /* multi-line code block */
  pre {
    padding: 10px;
    background-color: #eeeeee;
    color: #000000;
    border-radius: 5px;
    break-inside: avoid;
    text-align: left;
  }

  /* inline code, ie code within normal text */
  :not(pre) > code {
    padding: 0 4px;
    background-color: #eeeeee;
    color: #000000;
    border-radius: 5px;
  }

  /* code text */
  /* apply all children, to reach syntax highlighting sub-elements */
  code,
  code * {
    /* force monospace font */
    font-family: "Source Code Pro", "Courier New", monospace;
  }

  /* -------------------------------------------------- */
  /* quotes */
  /* -------------------------------------------------- */

  /* quoted text */
  blockquote {
    margin: 0;
    padding: 0;
    border-left: 4px solid #bdbdbd;
    padding-left: 16px;
    break-inside: avoid;
  }

  /* -------------------------------------------------- */
  /* banners */
  /* -------------------------------------------------- */

  /* info banners */
  .banner {
    box-sizing: border-box;
    display: block;
    position: relative;
    width: 100%;
    margin-top: 20px;
    margin-bottom: 20px;
    padding: 20px;
    text-align: center;
  }

  /* paragraph in banner */
  .banner > p {
    margin: 0;
  }

  /* -------------------------------------------------- */
  /* highlight colors */
  /* -------------------------------------------------- */

  .white {
    background: #ffffff;
  }
  .lightgrey {
    background: #eeeeee;
  }
  .grey {
    background: #757575;
  }
  .darkgrey {
    background: #424242;
  }
  .black {
    background: #000000;
  }
  .lightred {
    background: #ffcdd2;
  }
  .lightyellow {
    background: #ffecb3;
  }
  .lightgreen {
    background: #dcedc8;
  }
  .lightblue {
    background: #e3f2fd;
  }
  .lightpurple {
    background: #f3e5f5;
  }
  .red {
    background: #f44336;
  }
  .orange {
    background: #ff9800;
  }
  .yellow {
    background: #ffeb3b;
  }
  .green {
    background: #4caf50;
  }
  .blue {
    background: #2196f3;
  }
  .purple {
    background: #9c27b0;
  }
  .white,
  .lightgrey,
  .lightred,
  .lightyellow,
  .lightgreen,
  .lightblue,
  .lightpurple,
  .orange,
  .yellow,
  .white a,
  .lightgrey a,
  .lightred a,
  .lightyellow a,
  .lightgreen a,
  .lightblue a,
  .lightpurple a,
  .orange a,
  .yellow a {
    color: #000000;
  }
  .grey,
  .darkgrey,
  .black,
  .red,
  .green,
  .blue,
  .purple,
  .grey a,
  .darkgrey a,
  .black a,
  .red a,
  .green a,
  .blue a,
  .purple a {
    color: #ffffff;
  }

  /* -------------------------------------------------- */
  /* buttons */
  /* -------------------------------------------------- */

  /* class for styling links like buttons */
  .button {
    display: inline-flex;
    justify-content: center;
    align-items: center;
    margin: 5px;
    padding: 10px 20px;
    font-size: 0.75em;
    font-weight: 600;
    text-transform: uppercase;
    text-decoration: none;
    letter-spacing: 1px;
    background: none;
    color: #2196f3;
    border: solid 1px #bdbdbd;
    border-radius: 5px;
  }

  /* buttons when hovered */
  .button:hover:not([disabled]),
  .icon_button:hover:not([disabled]) {
    cursor: pointer;
    background: #f5f5f5;
  }

  /* buttons when disabled */
  .button[disabled],
  .icon_button[disabled] {
    opacity: 0.35;
    pointer-events: none;
  }

  /* class for styling buttons containg only single icon */
  .icon_button {
    display: inline-flex;
    justify-content: center;
    align-items: center;
    text-decoration: none;
    margin: 0;
    padding: 0;
    background: none;
    border-radius: 5px;
    border: none;
    width: 20px;
    height: 20px;
    min-width: 20px;
    min-height: 20px;
  }

  /* icon button inner svg image */
  .icon_button > svg {
    height: 16px;
  }

  /* -------------------------------------------------- */
  /* icons */
  /* -------------------------------------------------- */

  /* class for styling icons inline with text */
  .inline_icon {
    height: 1em;
    position: relative;
    top: 0.125em;
  }

  /* -------------------------------------------------- */
  /* references */
  /* -------------------------------------------------- */

  .csl-entry {
    margin-top: 15px;
    margin-bottom: 15px;
  }

  /* -------------------------------------------------- */
  /* print control */
  /* -------------------------------------------------- */

  @media print {
    @page {
      /* suggested printing margin */
      margin: 0.5in;
    }

    /* document and "page" elements */
    html,
    body {
      margin: 0;
      padding: 0;
      width: 100%;
      height: 100%;
    }

    /* "page" element */
    body {
      font-size: 11pt !important;
      line-height: 1.35;
    }

    /* all headings */
    h1,
    h2,
    h3,
    h4,
    h5,
    h6 {
      margin: 15px 0;
    }

    /* figures and tables */
    figure,
    table {
      font-size: 0.85em;
    }

    /* table cells */
    th,
    td {
      padding: 5px;
    }

    /* shrink font awesome icons */
    i.fas,
    i.fab,
    i.far,
    i.fal {
      transform: scale(0.85);
    }

    /* decrease banner margins */
    .banner {
      margin-top: 15px;
      margin-bottom: 15px;
      padding: 15px;
    }

    /* class for centering an element vertically on its own page */
    .page_center {
      margin: auto;
      width: 100%;
      height: 100%;
      display: flex;
      align-items: center;
      vertical-align: middle;
      break-before: page;
      break-after: page;
    }

    /* always insert a page break before the element */
    .page_break_before {
      break-before: page;
    }

    /* always insert a page break after the element */
    .page_break_after {
      break-after: page;
    }

    /* avoid page break before the element */
    .page_break_before_avoid {
      break-before: avoid;
    }

    /* avoid page break after the element */
    .page_break_after_avoid {
      break-after: avoid;
    }

    /* avoid page break inside the element */
    .page_break_inside_avoid {
      break-inside: avoid;
    }
  }

  /* -------------------------------------------------- */
  /* override pandoc css quirks */
  /* -------------------------------------------------- */

  .sourceCode {
    /* prevent unsightly overflow in wide code blocks */
    overflow: auto !important;
  }

  div.sourceCode {
    /* prevent background fill on top-most code block  container */
    background: none !important;
  }

  .sourceCode * {
    /* force consistent line spacing */
    line-height: 1.5 !important;
  }

  div.sourceCode {
    /* style code block margins same as <pre> element */
    margin-top: 20px;
    margin-bottom: 20px;
  }

  /* -------------------------------------------------- */
  /* tablenos */
  /* -------------------------------------------------- */

  /* tablenos wrapper */
  .tablenos {
    width: 100%;
    margin: 20px 0;
  }

  .tablenos > table {
    /* move margins from table to table_wrapper to allow margin collapsing */
    margin: 0;
  }

  @media only screen {
    /* tablenos wrapper */
    .tablenos {
      /* show scrollbar on tables if necessary to prevent overflow */
      overflow-x: auto !important;
    }

    .tablenos th,
    .tablenos td {
      overflow-wrap: unset !important;
      word-break: unset !important;
    }

    /* table in wrapper */
    .tablenos table,
    .tablenos table * {
      /* don't break table words */
      overflow-wrap: normal !important;
    }
  }
</style>
<!-- 
    Plugin Core

    Functions needed for and shared across all first-party plugins.
-->

<script>
  // get element that is target of hash (from link element or url)
  function getHashTarget(link) {
    const hash = link ? link.hash : window.location.hash;
    const id = hash.slice(1);
    let target = document.querySelector(`[id="${id}"]`);
    if (!target) return;

    // if figure or table, modify target to get expected element
    if (id.indexOf("fig:") === 0) target = target.querySelector("figure");
    if (id.indexOf("tbl:") === 0) target = target.querySelector("table");

    return target;
  }

  // get position/dimensions of element or viewport
  function getRectInView(element) {
    let rect = {};
    rect.left = 0;
    rect.top = 0;
    rect.right = document.documentElement.clientWidth;
    rect.bottom = document.documentElement.clientHeight;
    let style = {};

    if (element instanceof HTMLElement) {
      rect = element.getBoundingClientRect();
      style = window.getComputedStyle(element);
    }

    const margin = {};
    margin.left = parseFloat(style.marginLeftWidth) || 0;
    margin.top = parseFloat(style.marginTopWidth) || 0;
    margin.right = parseFloat(style.marginRightWidth) || 0;
    margin.bottom = parseFloat(style.marginBottomWidth) || 0;

    const border = {};
    border.left = parseFloat(style.borderLeftWidth) || 0;
    border.top = parseFloat(style.borderTopWidth) || 0;
    border.right = parseFloat(style.borderRightWidth) || 0;
    border.bottom = parseFloat(style.borderBottomWidth) || 0;

    const newRect = {};
    newRect.left = rect.left + margin.left + border.left;
    newRect.top = rect.top + margin.top + border.top;
    newRect.right = rect.right + margin.right + border.right;
    newRect.bottom = rect.bottom + margin.bottom + border.bottom;
    newRect.width = newRect.right - newRect.left;
    newRect.height = newRect.bottom - newRect.top;

    return newRect;
  }

  // get position of element relative to page
  function getRectInPage(element) {
    const rect = getRectInView(element);
    const body = getRectInView(document.body);

    const newRect = {};
    newRect.left = rect.left - body.left;
    newRect.top = rect.top - body.top;
    newRect.right = rect.right - body.left;
    newRect.bottom = rect.bottom - body.top;
    newRect.width = rect.width;
    newRect.height = rect.height;

    return newRect;
  }

  // get closest element before specified element that matches query
  function firstBefore(element, query) {
    while (element && element !== document.body && !element.matches(query))
      element = element.previousElementSibling || element.parentNode;

    return element;
  }

  // check if element is part of collapsed heading
  function isCollapsed(element) {
    while (element && element !== document.body) {
      if (element.dataset.collapsed === "true") return true;
      element = element.parentNode;
    }
    return false;
  }

  // expand any collapsed parent containers of element if necessary
  function expandElement(element) {
    if (isCollapsed(element)) {
      // accordion plugin
      const heading = firstBefore(element, "h2");
      if (heading) heading.click();
      // details/summary HTML element
      const summary = firstBefore(element, "summary");
      if (summary) summary.click();
    }
  }

  // scroll to and focus element
  function goToElement(element, offset) {
    // expand accordion section if collapsed
    expandElement(element);
    const y =
      getRectInView(element).top -
      getRectInView(document.documentElement).top -
      (offset || 0);

    // trigger any function listening for "onscroll" event
    window.dispatchEvent(new Event("scroll"));
    window.scrollTo(0, y);
    document.activeElement.blur();
    element.focus();
  }

  // get list of elements after a start element up to element matching query
  function nextUntil(element, query, exclude) {
    const elements = [];
    while (((element = element.nextElementSibling), element)) {
      if (element.matches(query)) break;
      if (!element.matches(exclude)) elements.push(element);
    }
    return elements;
  }
</script>
<!--
  Accordion Plugin

  Allows sections of content under h2 headings to be collapsible.
-->

<script type="module">
  // whether to always start expanded ('false'), always start collapsed
  // ('true'), or start collapsed when screen small ('auto')
  const startCollapsed = "auto";

  // start script
  function start() {
    // run through each <h2> heading
    const headings = document.querySelectorAll("h2");
    for (const heading of headings) {
      addArrow(heading);

      // start expanded/collapsed based on option
      if (
        startCollapsed === "true" ||
        (startCollapsed === "auto" && isSmallScreen()) ||
        heading.dataset.collapsed === "true"
      )
        collapseHeading(heading);
      else expandElement(heading);
    }

    // attach hash change listener to window
    window.addEventListener("hashchange", onHashChange);
  }

  // when hash (eg manuscript.html#introduction) changes
  function onHashChange() {
    const target = getHashTarget();
    if (target) goToElement(target);
  }

  // add arrow to heading
  function addArrow(heading) {
    // add arrow button
    const arrow = document.createElement("button");
    arrow.innerHTML = document.querySelector(".icon_angle_down").innerHTML;
    arrow.classList.add("icon_button", "accordion_arrow");
    heading.insertBefore(arrow, heading.firstChild);

    // attach click listener to heading and button
    heading.addEventListener("click", onHeadingClick);
    arrow.addEventListener("click", onArrowClick);
  }

  // determine if on mobile-like device with small screen
  function isSmallScreen() {
    return Math.min(window.innerWidth, window.innerHeight) < 480;
  }

  // when <h2> heading is clicked
  function onHeadingClick(event) {
    // only collapse if <h2> itself is target of click (eg, user did
    // not click on anchor within <h2>)
    if (event.target === this) toggleCollapse(this);
  }

  // when arrow button is clicked
  function onArrowClick() {
    toggleCollapse(this.parentNode);
  }

  // collapse section if expanded, expand if collapsed
  function toggleCollapse(heading) {
    if (heading.dataset.collapsed === "false") collapseHeading(heading);
    else expandElement(heading);
  }

  // elements to exclude from collapse, such as table of contents panel,
  // hypothesis panel, etc
  const exclude = "#toc_panel, div.annotator-frame, #lightbox_overlay";

  // collapse section
  function collapseHeading(heading) {
    heading.setAttribute("data-collapsed", "true");
    const children = getChildren(heading);
    for (const child of children) child.setAttribute("data-collapsed", "true");
  }

  // expand section
  function expandElement(heading) {
    heading.setAttribute("data-collapsed", "false");
    const children = getChildren(heading);
    for (const child of children) child.setAttribute("data-collapsed", "false");
  }

  // get list of elements between this <h2> and next <h2> or <h1>
  // ("children" of the <h2> section)
  function getChildren(heading) {
    return nextUntil(heading, "h2, h1", exclude);
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- angle down icon -->

<template class="icon_angle_down">
  <!-- modified from: https://fontawesome.com/icons/angle-down -->
  <svg width="16" height="16" viewBox="0 0 448 512">
    <path
      fill="currentColor"
      d="M207.029 381.476L12.686 187.132c-9.373-9.373-9.373-24.569 0-33.941l22.667-22.667c9.357-9.357 24.522-9.375 33.901-.04L224 284.505l154.745-154.021c9.379-9.335 24.544-9.317 33.901.04l22.667 22.667c9.373 9.373 9.373 24.569 0 33.941L240.971 381.476c-9.373 9.372-24.569 9.372-33.942 0z"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* accordion arrow button */
    .accordion_arrow {
      margin-right: 10px;
    }

    /* arrow icon when <h2> data-collapsed attribute true */
    h2[data-collapsed="true"] > .accordion_arrow > svg {
      transform: rotate(-90deg);
    }

    /* all elements (except <h2>'s) when data-collapsed attribute true */
    *:not(h2)[data-collapsed="true"] {
      display: none;
    }

    /* accordion arrow button when hovered and <h2>'s when hovered */
    .accordion_arrow:hover,
    h2[data-collapsed="true"]:hover,
    h2[data-collapsed="false"]:hover {
      cursor: pointer;
    }
  }

  /* always hide accordion arrow button on print */
  @media only print {
    .accordion_arrow {
      display: none;
    }
  }
</style>
<!--
  Anchors Plugin

  Adds an anchor next to each of a certain type of element that provides a
  human-readable url to that specific item/position in the document (e.g.
  "manuscript.html#abstract"). It also makes it such that scrolling out of view
  of a target removes its identifier from the url.
-->

<script type="module">
  // which types of elements to add anchors next to, in "document.querySelector"
  // format
  const typesQuery =
    'h1, h2, h3, div[id^="fig:"], div[id^="tbl:"], span[id^="eq:"]';

  // start script
  function start() {
    // add anchor to each element of specified types
    const elements = document.querySelectorAll(typesQuery);
    for (const element of elements) addAnchor(element);

    // attach scroll listener to window
    window.addEventListener("scroll", onScroll);
  }

  // when window is scrolled
  function onScroll() {
    // if url has hash and user has scrolled out of view of hash
    // target, remove hash from url
    const tolerance = 100;
    const target = getHashTarget();
    if (target) {
      if (
        target.getBoundingClientRect().top > window.innerHeight + tolerance ||
        target.getBoundingClientRect().bottom < 0 - tolerance
      )
        history.pushState(null, null, " ");
    }
  }

  // add anchor to element
  function addAnchor(element) {
    let addTo; // element to add anchor button to

    // if figure or table, modify withId and addTo to get expected
    // elements
    if (element.id.indexOf("fig:") === 0) {
      addTo = element.querySelector("figcaption");
    } else if (element.id.indexOf("tbl:") === 0) {
      addTo = element.querySelector("caption");
    } else if (element.id.indexOf("eq:") === 0) {
      addTo = element.querySelector(".eqnos-number");
    }

    addTo = addTo || element;
    const id = element.id || null;

    // do not add anchor if element doesn't have assigned id.
    // id is generated by pandoc and is assumed to be unique and
    // human-readable
    if (!id) return;

    // create anchor button
    const anchor = document.createElement("a");
    anchor.innerHTML = document.querySelector(".icon_link").innerHTML;
    anchor.title = "Link to this part of the document";
    anchor.classList.add("icon_button", "anchor");
    anchor.dataset.ignore = "true";
    anchor.href = "#" + id;
    addTo.appendChild(anchor);
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- link icon -->

<template class="icon_link">
  <!-- modified from: https://fontawesome.com/icons/link -->
  <svg width="16" height="16" viewBox="0 0 512 512">
    <path
      fill="currentColor"
      d="M326.612 185.391c59.747 59.809 58.927 155.698.36 214.59-.11.12-.24.25-.36.37l-67.2 67.2c-59.27 59.27-155.699 59.262-214.96 0-59.27-59.26-59.27-155.7 0-214.96l37.106-37.106c9.84-9.84 26.786-3.3 27.294 10.606.648 17.722 3.826 35.527 9.69 52.721 1.986 5.822.567 12.262-3.783 16.612l-13.087 13.087c-28.026 28.026-28.905 73.66-1.155 101.96 28.024 28.579 74.086 28.749 102.325.51l67.2-67.19c28.191-28.191 28.073-73.757 0-101.83-3.701-3.694-7.429-6.564-10.341-8.569a16.037 16.037 0 0 1-6.947-12.606c-.396-10.567 3.348-21.456 11.698-29.806l21.054-21.055c5.521-5.521 14.182-6.199 20.584-1.731a152.482 152.482 0 0 1 20.522 17.197zM467.547 44.449c-59.261-59.262-155.69-59.27-214.96 0l-67.2 67.2c-.12.12-.25.25-.36.37-58.566 58.892-59.387 154.781.36 214.59a152.454 152.454 0 0 0 20.521 17.196c6.402 4.468 15.064 3.789 20.584-1.731l21.054-21.055c8.35-8.35 12.094-19.239 11.698-29.806a16.037 16.037 0 0 0-6.947-12.606c-2.912-2.005-6.64-4.875-10.341-8.569-28.073-28.073-28.191-73.639 0-101.83l67.2-67.19c28.239-28.239 74.3-28.069 102.325.51 27.75 28.3 26.872 73.934-1.155 101.96l-13.087 13.087c-4.35 4.35-5.769 10.79-3.783 16.612 5.864 17.194 9.042 34.999 9.69 52.721.509 13.906 17.454 20.446 27.294 10.606l37.106-37.106c59.271-59.259 59.271-155.699.001-214.959z"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* anchor button */
    .anchor {
      opacity: 0;
      margin-left: 5px;
    }

    /* anchor buttons within <h2>'s */
    h2 .anchor {
      margin-left: 10px;
    }

    /* anchor buttons when hovered/focused and anything containing an anchor button when hovered */
    *:hover > .anchor,
    .anchor:hover,
    .anchor:focus {
      opacity: 1;
    }

    /* anchor button when hovered */
    .anchor:hover {
      cursor: pointer;
    }
  }

  /* always show anchor button on devices with no mouse/hover ability */
  @media (hover: none) {
    .anchor {
      opacity: 1;
    }
  }

  /* always hide anchor button on print */
  @media only print {
    .anchor {
      display: none;
    }
  }
</style>
<!-- 
    Attributes Plugin

    Allows arbitrary HTML attributes to be attached to (almost) any element.
    Place an HTML comment inside or next to the desired element with the content:
    $attribute="value"
-->

<script type="module">
  // start script
  function start() {
    // get list of comments in document
    const comments = findComments();

    for (const comment of comments)
      if (comment.parentElement)
        addAttributes(comment.parentElement, comment.nodeValue.trim());
  }

  // add html attributes to specified element based on string of
  // html attributes and values
  function addAttributes(element, text) {
    // regex's for finding attribute/value pairs in the format of
    // attribute="value" or attribute='value
    const regex2 = /\$([a-zA-Z\-]+)?=\"(.+?)\"/;
    const regex1 = /\$([a-zA-Z\-]+)?=\'(.+?)\'/;

    // loop through attribute/value pairs
    let match;
    while ((match = text.match(regex2) || text.match(regex1))) {
      // get attribute and value from regex capture groups
      let attribute = match[1];
      let value = match[2];

      // remove from string
      text = text.substring(match.index + match[0].length);

      if (!attribute || !value) break;

      // set attribute of parent element
      try {
        element.setAttribute(attribute, value);
      } catch (error) {
        console.log(error);
      }

      // special case for colspan
      if (attribute === "colspan") removeTableCells(element, value);
    }
  }

  // get list of comment elements in document
  function findComments() {
    const comments = [];

    // iterate over comment nodes in document
    function acceptNode(node) {
      return NodeFilter.FILTER_ACCEPT;
    }
    const iterator = document.createNodeIterator(
      document.body,
      NodeFilter.SHOW_COMMENT,
      acceptNode
    );
    let node;
    while ((node = iterator.nextNode())) comments.push(node);

    return comments;
  }

  // remove certain number of cells after specified cell
  function removeTableCells(cell, number) {
    number = parseInt(number);
    if (!number) return;

    // remove elements
    for (; number > 1; number--) {
      if (cell.nextElementSibling) cell.nextElementSibling.remove();
    }
  }

  // start script on DOMContentLoaded instead of load to ensure this plugins
  // runs before other plugins
  window.addEventListener("DOMContentLoaded", start);
</script>
<!--
  Jump to First Plugin

  Adds a button next to each reference entry, figure, and table that jumps the
  page to the first occurrence of a link to that item in the manuscript.
-->

<script type="module">
  // whether to add buttons next to reference entries
  const references = "true";
  // whether to add buttons next to figures
  const figures = "true";
  // whether to add buttons next to tables
  const tables = "true";

  // start script
  function start() {
    if (references !== "false")
      makeButtons(`div[id^="ref-"]`, ".csl-left-margin", "reference");
    if (figures !== "false")
      makeButtons(`div[id^="fig:"]`, "figcaption", "figure");
    if (tables !== "false") makeButtons(`div[id^="tbl:"]`, "caption", "table");
  }

  // when jump button clicked
  function onButtonClick() {
    const first = getFirstOccurrence(this.dataset.id);
    if (!first) return;

    // update url hash so navigating "back" in history will return user to button
    window.location.hash = this.dataset.id;
    // scroll to link
    const timeout = function () {
      goToElement(first, window.innerHeight * 0.5);
    };
    window.setTimeout(timeout, 0);
  }

  // get first occurrence of link to item in document
  function getFirstOccurrence(id) {
    let query = "a";
    query += '[href="#' + id + '"]';
    // exclude buttons, anchor links, toc links, etc
    query += ":not(.button):not(.icon_button):not(.anchor):not(.toc_link)";
    return document.querySelector(query);
  }

  // add button next to each reference entry, figure, or table
  function makeButtons(query, containerQuery, subject) {
    const elements = document.querySelectorAll(query);
    for (const element of elements) {
      const id = element.id;
      const buttonContainer = element.querySelector(containerQuery);
      const first = getFirstOccurrence(id);

      // if can't find link to reference or place to put button, ignore
      if (!first || !buttonContainer) continue;

      // make jump button
      let button = document.createElement("button");
      button.classList.add("icon_button", "jump_arrow");
      button.title = `Jump to the first occurrence of this ${subject} in the document`;
      const icon = document.querySelector(".icon_angle_double_up");
      button.innerHTML = icon.innerHTML;
      button.dataset.id = id;
      button.dataset.ignore = "true";
      button.addEventListener("click", onButtonClick);
      buttonContainer.prepend(button);
    }
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- angle double up icon -->

<template class="icon_angle_double_up">
  <!-- modified from: https://fontawesome.com/icons/angle-double-up -->
  <svg width="16" height="16" viewBox="0 0 320 512">
    <path
      fill="currentColor"
      d="M177 255.7l136 136c9.4 9.4 9.4 24.6 0 33.9l-22.6 22.6c-9.4 9.4-24.6 9.4-33.9 0L160 351.9l-96.4 96.4c-9.4 9.4-24.6 9.4-33.9 0L7 425.7c-9.4-9.4-9.4-24.6 0-33.9l136-136c9.4-9.5 24.6-9.5 34-.1zm-34-192L7 199.7c-9.4 9.4-9.4 24.6 0 33.9l22.6 22.6c9.4 9.4 24.6 9.4 33.9 0l96.4-96.4 96.4 96.4c9.4 9.4 24.6 9.4 33.9 0l22.6-22.6c9.4-9.4 9.4-24.6 0-33.9l-136-136c-9.2-9.4-24.4-9.4-33.8 0z"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* jump button */
    .jump_arrow {
      position: relative;
      top: 0.125em;
      margin-right: 5px;
    }
  }

  /* always hide jump button on print */
  @media only print {
    .jump_arrow {
      display: none;
    }
  }
</style>
<!-- 
    Lightbox Plugin

    Makes it such that when a user clicks on an image, the image fills the
    screen and the user can pan/drag/zoom the image and navigate between other
    images in the document.
-->

<script type="module">
  // list of possible zoom/scale factors
  const zooms =
    "0.1, 0.25, 0.333333, 0.5, 0.666666, 0.75, 1, 1.25, 1.5, 1.75, 2, 2.5, 3, 3.5, 4, 5, 6, 7, 8";
  // whether to fit image to view ('fit'), display at 100% and shrink if
  // necessary ('shrink'), or always display at 100% ('100')
  const defaultZoom = "fit";
  // whether to zoom in/out toward center of view ('true') or mouse ('false')
  const centerZoom = "false";

  // start script
  function start() {
    // run through each <img> element
    const imgs = document.querySelectorAll("figure > img");
    let count = 1;
    for (const img of imgs) {
      img.classList.add("lightbox_document_img");
      img.dataset.number = count;
      img.dataset.total = imgs.length;
      img.addEventListener("click", openLightbox);
      count++;
    }

    // attach mouse and key listeners to window
    window.addEventListener("mousemove", onWindowMouseMove);
    window.addEventListener("keyup", onKeyUp);
  }

  // when mouse is moved anywhere in window
  function onWindowMouseMove(event) {
    window.mouseX = event.clientX;
    window.mouseY = event.clientY;
  }

  // when key pressed
  function onKeyUp(event) {
    if (!event || !event.key) return;

    switch (event.key) {
      // trigger click of prev button
      case "ArrowLeft":
        const prevButton = document.getElementById("lightbox_prev_button");
        if (prevButton) prevButton.click();
        break;
      // trigger click of next button
      case "ArrowRight":
        const nextButton = document.getElementById("lightbox_next_button");
        if (nextButton) nextButton.click();
        break;
      // close on esc
      case "Escape":
        closeLightbox();
        break;
    }
  }

  // open lightbox
  function openLightbox() {
    const lightbox = makeLightbox(this);
    if (!lightbox) return;

    blurBody(lightbox);
    document.body.appendChild(lightbox);
  }

  // make lightbox
  function makeLightbox(img) {
    // delete lightbox if it exists, start fresh
    closeLightbox();

    // create screen overlay containing lightbox
    const overlay = document.createElement("div");
    overlay.id = "lightbox_overlay";

    // create image info boxes
    const numberInfo = document.createElement("div");
    const zoomInfo = document.createElement("div");
    numberInfo.id = "lightbox_number_info";
    zoomInfo.id = "lightbox_zoom_info";

    // create container for image
    const imageContainer = document.createElement("div");
    imageContainer.id = "lightbox_image_container";
    const lightboxImg = makeLightboxImg(
      img,
      imageContainer,
      numberInfo,
      zoomInfo
    );
    imageContainer.appendChild(lightboxImg);

    // create bottom container for caption and navigation buttons
    const bottomContainer = document.createElement("div");
    bottomContainer.id = "lightbox_bottom_container";
    const caption = makeCaption(img);
    const prevButton = makePrevButton(img);
    const nextButton = makeNextButton(img);
    bottomContainer.appendChild(prevButton);
    bottomContainer.appendChild(caption);
    bottomContainer.appendChild(nextButton);

    // attach top middle and bottom to overlay
    overlay.appendChild(numberInfo);
    overlay.appendChild(zoomInfo);
    overlay.appendChild(imageContainer);
    overlay.appendChild(bottomContainer);

    return overlay;
  }

  // make <img> object that is intuitively draggable and zoomable
  function makeLightboxImg(sourceImg, container, numberInfoBox, zoomInfoBox) {
    // create copy of source <img>
    const img = sourceImg.cloneNode(true);
    img.classList.remove("lightbox_document_img");
    img.removeAttribute("id");
    img.removeAttribute("width");
    img.removeAttribute("height");
    img.style.position = "unset";
    img.style.margin = "0";
    img.style.padding = "0";
    img.style.width = "";
    img.style.height = "";
    img.style.minWidth = "";
    img.style.minHeight = "";
    img.style.maxWidth = "";
    img.style.maxHeight = "";
    img.id = "lightbox_img";

    // build sorted list of zoomSteps
    const zoomSteps = zooms.split(/[^0-9.]/).map((step) => parseFloat(step));
    zoomSteps.sort((a, b) => a - b);

    // <img> object property variables
    let zoom = 1;
    let translateX = 0;
    let translateY = 0;
    let clickMouseX = undefined;
    let clickMouseY = undefined;
    let clickTranslateX = undefined;
    let clickTranslateY = undefined;

    updateNumberInfo();

    // update image numbers displayed in info box
    function updateNumberInfo() {
      numberInfoBox.innerHTML =
        sourceImg.dataset.number + " of " + sourceImg.dataset.total;
    }

    // update zoom displayed in info box
    function updateZoomInfo() {
      let zoomInfo = zoom * 100;
      if (!Number.isInteger(zoomInfo)) zoomInfo = zoomInfo.toFixed(2);
      zoomInfoBox.innerHTML = zoomInfo + "%";
    }

    // move to closest zoom step above current zoom
    const zoomIn = function () {
      for (const zoomStep of zoomSteps) {
        if (zoomStep > zoom) {
          zoom = zoomStep;
          break;
        }
      }
      updateTransform();
    };

    // move to closest zoom step above current zoom
    const zoomOut = function () {
      zoomSteps.reverse();
      for (const zoomStep of zoomSteps) {
        if (zoomStep < zoom) {
          zoom = zoomStep;
          break;
        }
      }
      zoomSteps.reverse();

      updateTransform();
    };

    // update display of <img> based on scale/translate properties
    const updateTransform = function () {
      // set transform
      img.style.transform =
        "translate(" +
        (translateX || 0) +
        "px," +
        (translateY || 0) +
        "px) scale(" +
        (zoom || 1) +
        ")";

      // get new width/height after scale
      const rect = img.getBoundingClientRect();
      // limit translate
      translateX = Math.max(translateX, -rect.width / 2);
      translateX = Math.min(translateX, rect.width / 2);
      translateY = Math.max(translateY, -rect.height / 2);
      translateY = Math.min(translateY, rect.height / 2);

      // set transform
      img.style.transform =
        "translate(" +
        (translateX || 0) +
        "px," +
        (translateY || 0) +
        "px) scale(" +
        (zoom || 1) +
        ")";

      updateZoomInfo();
    };

    // fit <img> to container
    const fit = function () {
      // no x/y offset, 100% zoom by default
      translateX = 0;
      translateY = 0;
      zoom = 1;

      // widths of <img> and container
      const imgWidth = img.naturalWidth;
      const imgHeight = img.naturalHeight;
      const containerWidth = parseFloat(
        window.getComputedStyle(container).width
      );
      const containerHeight = parseFloat(
        window.getComputedStyle(container).height
      );

      // how much zooming is needed to fit <img> to container
      const xRatio = imgWidth / containerWidth;
      const yRatio = imgHeight / containerHeight;
      const maxRatio = Math.max(xRatio, yRatio);
      const newZoom = 1 / maxRatio;

      // fit <img> to container according to option
      if (defaultZoom === "shrink") {
        if (maxRatio > 1) zoom = newZoom;
      } else if (defaultZoom === "fit") zoom = newZoom;

      updateTransform();
    };

    // when mouse wheel is rolled anywhere in container
    const onContainerWheel = function (event) {
      if (!event) return;

      // let ctrl + mouse wheel to zoom behave as normal
      if (event.ctrlKey) return;

      // prevent normal scroll behavior
      event.preventDefault();
      event.stopPropagation();

      // point around which to scale img
      const viewRect = container.getBoundingClientRect();
      const viewX = (viewRect.left + viewRect.right) / 2;
      const viewY = (viewRect.top + viewRect.bottom) / 2;
      const originX = centerZoom === "true" ? viewX : mouseX;
      const originY = centerZoom === "true" ? viewY : mouseY;

      // get point on image under origin
      const oldRect = img.getBoundingClientRect();
      const oldPercentX = (originX - oldRect.left) / oldRect.width;
      const oldPercentY = (originY - oldRect.top) / oldRect.height;

      // increment/decrement zoom
      if (event.deltaY < 0) zoomIn();
      if (event.deltaY > 0) zoomOut();

      // get offset between previous image point and origin
      const newRect = img.getBoundingClientRect();
      const offsetX = originX - (newRect.left + newRect.width * oldPercentX);
      const offsetY = originY - (newRect.top + newRect.height * oldPercentY);

      // translate image to keep image point under origin
      translateX += offsetX;
      translateY += offsetY;

      // perform translate
      updateTransform();
    };

    // when container is clicked
    function onContainerClick(event) {
      // if container itself is target of click, and not other
      // element above it
      if (event.target === this) closeLightbox();
    }

    // when mouse button is pressed on image
    const onImageMouseDown = function (event) {
      // store original mouse position relative to image
      clickMouseX = window.mouseX;
      clickMouseY = window.mouseY;
      clickTranslateX = translateX;
      clickTranslateY = translateY;
      event.stopPropagation();
      event.preventDefault();
    };

    // when mouse button is released anywhere in window
    const onWindowMouseUp = function (event) {
      // reset original mouse position
      clickMouseX = undefined;
      clickMouseY = undefined;
      clickTranslateX = undefined;
      clickTranslateY = undefined;

      // remove global listener if lightbox removed from document
      if (!document.body.contains(container))
        window.removeEventListener("mouseup", onWindowMouseUp);
    };

    // when mouse is moved anywhere in window
    const onWindowMouseMove = function (event) {
      if (
        clickMouseX === undefined ||
        clickMouseY === undefined ||
        clickTranslateX === undefined ||
        clickTranslateY === undefined
      )
        return;

      // offset image based on original and current mouse position
      translateX = clickTranslateX + window.mouseX - clickMouseX;
      translateY = clickTranslateY + window.mouseY - clickMouseY;
      updateTransform();
      event.preventDefault();

      // remove global listener if lightbox removed from document
      if (!document.body.contains(container))
        window.removeEventListener("mousemove", onWindowMouseMove);
    };

    // when window is resized
    const onWindowResize = function (event) {
      fit();

      // remove global listener if lightbox removed from document
      if (!document.body.contains(container))
        window.removeEventListener("resize", onWindowResize);
    };

    // attach the necessary event listeners
    img.addEventListener("dblclick", fit);
    img.addEventListener("mousedown", onImageMouseDown);
    container.addEventListener("wheel", onContainerWheel);
    container.addEventListener("mousedown", onContainerClick);
    container.addEventListener("touchstart", onContainerClick);
    window.addEventListener("mouseup", onWindowMouseUp);
    window.addEventListener("mousemove", onWindowMouseMove);
    window.addEventListener("resize", onWindowResize);

    // run fit() after lightbox atttached to document and <img> Loaded
    // so needed container and img dimensions available
    img.addEventListener("load", fit);

    return img;
  }

  // make caption
  function makeCaption(img) {
    const caption = document.createElement("div");
    caption.id = "lightbox_caption";
    const captionSource = img.nextElementSibling;
    if (captionSource.tagName.toLowerCase() === "figcaption") {
      const captionCopy = makeCopy(captionSource);
      caption.innerHTML = captionCopy.innerHTML;
    }

    caption.addEventListener("touchstart", function (event) {
      event.stopPropagation();
    });

    return caption;
  }

  // make carbon copy of html dom element
  function makeCopy(source) {
    const sourceCopy = source.cloneNode(true);

    // delete elements marked with ignore (eg anchor and jump buttons)
    const deleteFromCopy = sourceCopy.querySelectorAll('[data-ignore="true"]');
    for (const element of deleteFromCopy) element.remove();

    // delete certain element attributes
    const attributes = [
      "id",
      "data-collapsed",
      "data-selected",
      "data-highlighted",
      "data-glow",
    ];
    for (const attribute of attributes) {
      sourceCopy.removeAttribute(attribute);
      const elements = sourceCopy.querySelectorAll("[" + attribute + "]");
      for (const element of elements) element.removeAttribute(attribute);
    }

    return sourceCopy;
  }

  // make button to jump to previous image in document
  function makePrevButton(img) {
    const prevButton = document.createElement("button");
    prevButton.id = "lightbox_prev_button";
    prevButton.title = "Jump to the previous image in the document [←]";
    prevButton.classList.add("icon_button", "lightbox_button");
    prevButton.innerHTML = document.querySelector(".icon_caret_left").innerHTML;

    // attach click listeners to button
    prevButton.addEventListener("click", function () {
      getPrevImg(img).click();
    });

    return prevButton;
  }

  // make button to jump to next image in document
  function makeNextButton(img) {
    const nextButton = document.createElement("button");
    nextButton.id = "lightbox_next_button";
    nextButton.title = "Jump to the next image in the document [→]";
    nextButton.classList.add("icon_button", "lightbox_button");
    nextButton.innerHTML = document.querySelector(
      ".icon_caret_right"
    ).innerHTML;

    // attach click listeners to button
    nextButton.addEventListener("click", function () {
      getNextImg(img).click();
    });

    return nextButton;
  }

  // get previous image in document
  function getPrevImg(img) {
    const imgs = document.querySelectorAll(".lightbox_document_img");

    // find index of provided img
    let index;
    for (index = 0; index < imgs.length; index++) {
      if (imgs[index] === img) break;
    }

    // wrap index to other side if < 1
    if (index - 1 >= 0) index--;
    else index = imgs.length - 1;
    return imgs[index];
  }

  // get next image in document
  function getNextImg(img) {
    const imgs = document.querySelectorAll(".lightbox_document_img");

    // find index of provided img
    let index;
    for (index = 0; index < imgs.length; index++) {
      if (imgs[index] === img) break;
    }

    // wrap index to other side if > total
    if (index + 1 <= imgs.length - 1) index++;
    else index = 0;
    return imgs[index];
  }

  // close lightbox
  function closeLightbox() {
    focusBody();

    const lightbox = document.getElementById("lightbox_overlay");
    if (lightbox) lightbox.remove();
  }

  // make all elements behind lightbox non-focusable
  function blurBody(overlay) {
    const all = document.querySelectorAll("*");
    for (const element of all) element.tabIndex = -1;
    document.body.classList.add("body_no_scroll");
  }

  // make all elements focusable again
  function focusBody() {
    const all = document.querySelectorAll("*");
    for (const element of all) element.removeAttribute("tabIndex");
    document.body.classList.remove("body_no_scroll");
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- caret left icon -->

<template class="icon_caret_left">
  <!-- modified from: https://fontawesome.com/icons/caret-left -->
  <svg width="16" height="16" viewBox="0 0 192 512">
    <path
      fill="currentColor"
      d="M192 127.338v257.324c0 17.818-21.543 26.741-34.142 14.142L29.196 270.142c-7.81-7.81-7.81-20.474 0-28.284l128.662-128.662c12.599-12.6 34.142-3.676 34.142 14.142z"
    ></path>
  </svg>
</template>

<!-- caret right icon -->

<template class="icon_caret_right">
  <!-- modified from: https://fontawesome.com/icons/caret-right -->
  <svg width="16" height="16" viewBox="0 0 192 512">
    <path
      fill="currentColor"
      d="M0 384.662V127.338c0-17.818 21.543-26.741 34.142-14.142l128.662 128.662c7.81 7.81 7.81 20.474 0 28.284L34.142 398.804C21.543 411.404 0 402.48 0 384.662z"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* regular <img> in document when hovered */
    img.lightbox_document_img:hover {
      cursor: pointer;
    }

    .body_no_scroll {
      overflow: hidden !important;
    }

    /* screen overlay */
    #lightbox_overlay {
      display: flex;
      flex-direction: column;
      position: fixed;
      left: 0;
      top: 0;
      right: 0;
      bottom: 0;
      background: rgba(0, 0, 0, 0.75);
      z-index: 3;
    }

    /* middle area containing lightbox image */
    #lightbox_image_container {
      flex-grow: 1;
      display: flex;
      justify-content: center;
      align-items: center;
      overflow: hidden;
      position: relative;
      padding: 20px;
    }

    /* bottom area containing caption */
    #lightbox_bottom_container {
      display: flex;
      justify-content: center;
      align-items: center;
      height: 100px;
      min-height: 100px;
      max-height: 100px;
      background: rgba(0, 0, 0, 0.5);
    }

    /* image number info text box */
    #lightbox_number_info {
      position: absolute;
      color: #ffffff;
      font-weight: 600;
      left: 2px;
      top: 0;
      z-index: 4;
    }

    /* zoom info text box */
    #lightbox_zoom_info {
      position: absolute;
      color: #ffffff;
      font-weight: 600;
      right: 2px;
      top: 0;
      z-index: 4;
    }

    /* copy of image caption */
    #lightbox_caption {
      box-sizing: border-box;
      display: inline-block;
      width: 100%;
      max-height: 100%;
      padding: 10px 0;
      text-align: center;
      overflow-y: auto;
      color: #ffffff;
    }

    /* navigation previous/next button */
    .lightbox_button {
      width: 100px;
      height: 100%;
      min-width: 100px;
      min-height: 100%;
      color: #ffffff;
    }

    /* navigation previous/next button when hovered */
    .lightbox_button:hover {
      background: none !important;
    }

    /* navigation button icon */
    .lightbox_button > svg {
      height: 25px;
    }

    /* figure auto-number */
    #lightbox_caption > span:first-of-type {
      font-weight: bold;
      margin-right: 5px;
    }

    /* lightbox image when hovered */
    #lightbox_img:hover {
      cursor: grab;
    }

    /* lightbox image when grabbed */
    #lightbox_img:active {
      cursor: grabbing;
    }
  }

  /* when on screen < 480px wide */
  @media only screen and (max-width: 480px) {
    /* make navigation buttons skinnier on small screens to make more room for caption text */
    .lightbox_button {
      width: 50px;
      min-width: 50px;
    }
  }

  /* always hide lightbox on print */
  @media only print {
    #lightbox_overlay {
      display: none;
    }
  }
</style>
<!-- 
  Link Highlight Plugin

  Makes it such that when a user hovers or focuses a link, other links that have
  the same target will be highlighted. It also makes it such that when clicking
  a link, the target of the link (eg reference, figure, table) is briefly
  highlighted.
-->

<script type="module">
  // whether to also highlight links that go to external urls
  const externalLinks = "false";
  // whether user must click off to unhighlight instead of just
  // un-hovering
  const clickUnhighlight = "false";
  // whether to also highlight links that are unique
  const highlightUnique = "true";

  // start script
  function start() {
    const links = getLinks();
    for (const link of links) {
      // attach mouse and focus listeners to link
      link.addEventListener("mouseenter", onLinkFocus);
      link.addEventListener("focus", onLinkFocus);
      link.addEventListener("mouseleave", onLinkUnhover);
    }

    // attach click and hash change listeners to window
    window.addEventListener("click", onClick);
    window.addEventListener("touchstart", onClick);
    window.addEventListener("hashchange", onHashChange);

    // run hash change on window load in case user has navigated
    // directly to hash
    onHashChange();
  }

  // when link is focused (tabbed to) or hovered
  function onLinkFocus() {
    highlight(this);
  }

  // when link is unhovered
  function onLinkUnhover() {
    if (clickUnhighlight !== "true") unhighlightAll();
  }

  // when the mouse is clicked anywhere in window
  function onClick(event) {
    unhighlightAll();
  }

  // when hash (eg manuscript.html#introduction) changes
  function onHashChange() {
    const target = getHashTarget();
    if (target) glowElement(target);
  }

  // start glow sequence on an element
  function glowElement(element) {
    const startGlow = function () {
      onGlowEnd();
      element.dataset.glow = "true";
      element.addEventListener("animationend", onGlowEnd);
    };
    const onGlowEnd = function () {
      element.removeAttribute("data-glow");
      element.removeEventListener("animationend", onGlowEnd);
    };
    startGlow();
  }

  // highlight link and all others with same target
  function highlight(link) {
    // force unhighlight all to start fresh
    unhighlightAll();

    // get links with same target
    if (!link) return;
    const sameLinks = getSameLinks(link);

    // if link unique and option is off, exit and don't highlight
    if (sameLinks.length <= 1 && highlightUnique !== "true") return;

    // highlight all same links, and "select" (special highlight) this
    // one
    for (const sameLink of sameLinks) {
      if (sameLink === link) sameLink.setAttribute("data-selected", "true");
      else sameLink.setAttribute("data-highlighted", "true");
    }
  }

  // unhighlight all links
  function unhighlightAll() {
    const links = getLinks();
    for (const link of links) {
      link.setAttribute("data-selected", "false");
      link.setAttribute("data-highlighted", "false");
    }
  }

  // get links with same target
  function getSameLinks(link) {
    const results = [];
    const links = getLinks();
    for (const otherLink of links) {
      if (otherLink.getAttribute("href") === link.getAttribute("href"))
        results.push(otherLink);
    }
    return results;
  }

  // get all links of types we wish to handle
  function getLinks() {
    let query = "a";
    if (externalLinks !== "true") query += '[href^="#"]';
    // exclude buttons, anchor links, toc links, etc
    query += ":not(.button):not(.icon_button):not(.anchor):not(.toc_link)";
    return document.querySelectorAll(query);
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<style>
  @media only screen {
    /* anything with data-highlighted attribute true */
    [data-highlighted="true"] {
      background: #ffeb3b;
    }

    /* anything with data-selected attribute true */
    [data-selected="true"] {
      background: #ff8a65 !important;
    }

    /* animation definition for glow */
    @keyframes highlight_glow {
      0% {
        background: none;
      }
      10% {
        background: #bbdefb;
      }
      100% {
        background: none;
      }
    }

    /* anything with data-glow attribute true */
    [data-glow="true"] {
      animation: highlight_glow 2s;
    }
  }
</style>
<!--
  Table of Contents Plugin

  Provides a "table of contents" (toc) panel on the side of the document that
  allows the user to conveniently navigate between sections of the document.
-->

<script type="module">
  // which types of elements to add links for, in "document.querySelector" format
  const typesQuery = "h1, h2, h3";
  // whether toc starts open. use 'true' or 'false', or 'auto' to
  // use 'true' behavior when screen wide enough and 'false' when not
  const startOpen = "false";
  // whether toc closes when clicking on toc link. use 'true' or
  // 'false', or 'auto' to use 'false' behavior when screen wide
  // enough and 'true' when not
  const clickClose = "auto";
  // if list item is more than this many characters, text will be
  // truncated
  const charLimit = "50";
  // whether or not to show bullets next to each toc item
  const bullets = "false";

  // start script
  function start() {
    // make toc panel and populate with entries (links to document
    // sections)
    const panel = makePanel();
    if (!panel) return;
    makeEntries(panel);
    // attach panel to document after making entries, so 'toc' heading
    // in panel isn't included in toc
    document.body.insertBefore(panel, document.body.firstChild);

    // initial panel state
    if (startOpen === "true" || (startOpen === "auto" && !isSmallScreen()))
      openPanel();
    else closePanel();

    // attach click, scroll, and hash change listeners to window
    window.addEventListener("click", onClick);
    window.addEventListener("scroll", onScroll);
    window.addEventListener("hashchange", onScroll);
    window.addEventListener("keyup", onKeyUp);
    onScroll();

    // add class to push document body down out of way of toc button
    document.body.classList.add("toc_body_nudge");
  }

  // determine if screen wide enough to fit toc panel
  function isSmallScreen() {
    // in default theme:
    // 816px = 8.5in = width of "page" (<body>) element
    // 260px = min width of toc panel (*2 for both sides of <body>)
    return window.innerWidth < 816 + 260 * 2;
  }

  // when mouse is clicked anywhere in window
  function onClick() {
    if (isSmallScreen()) closePanel();
  }

  // when window is scrolled or hash changed
  function onScroll() {
    highlightViewed();
  }

  // when key pressed
  function onKeyUp(event) {
    if (!event || !event.key) return;

    // close on esc
    if (event.key === "Escape") closePanel();
  }

  // find entry of currently viewed document section in toc and highlight
  function highlightViewed() {
    const firstId = getFirstInView(typesQuery);

    // get toc entries (links), unhighlight all, then highlight viewed
    const list = document.getElementById("toc_list");
    if (!firstId || !list) return;
    const links = list.querySelectorAll("a");
    for (const link of links) link.dataset.viewing = "false";
    const link = list.querySelector('a[href="#' + firstId + '"]');
    if (!link) return;
    link.dataset.viewing = "true";
  }

  // get first or previous toc listed element in top half of view
  function getFirstInView(query) {
    // get all elements matching query and with id
    const elements = document.querySelectorAll(query);
    const elementsWithIds = [];
    for (const element of elements) {
      if (element.id) elementsWithIds.push(element);
    }

    // get first or previous element in top half of view
    for (let i = 0; i < elementsWithIds.length; i++) {
      const element = elementsWithIds[i];
      const prevElement = elementsWithIds[Math.max(0, i - 1)];
      if (element.getBoundingClientRect().top >= 0) {
        if (element.getBoundingClientRect().top < window.innerHeight / 2)
          return element.id;
        else return prevElement.id;
      }
    }
  }

  // make panel
  function makePanel() {
    // create panel
    const panel = document.createElement("div");
    panel.id = "toc_panel";
    if (bullets === "true") panel.dataset.bullets = "true";

    // create header
    const header = document.createElement("div");
    header.id = "toc_header";

    // create toc button
    const button = document.createElement("button");
    button.id = "toc_button";
    button.innerHTML = document.querySelector(".icon_th_list").innerHTML;
    button.title = "Table of Contents";
    button.classList.add("icon_button");

    // create header text
    const text = document.createElement("h4");
    text.innerHTML = "Table of Contents";

    // create container for toc list
    const list = document.createElement("div");
    list.id = "toc_list";

    // attach click listeners
    panel.addEventListener("click", onPanelClick);
    header.addEventListener("click", onHeaderClick);
    button.addEventListener("click", onButtonClick);

    // attach elements
    header.appendChild(button);
    header.appendChild(text);
    panel.appendChild(header);
    panel.appendChild(list);

    return panel;
  }

  // create toc entries (links) to each element of the specified types
  function makeEntries(panel) {
    const elements = document.querySelectorAll(typesQuery);
    for (const element of elements) {
      // do not add link if element doesn't have assigned id
      if (!element.id) continue;

      // create link/list item
      const link = document.createElement("a");
      link.classList.add("toc_link");
      switch (element.tagName.toLowerCase()) {
        case "h1":
          link.dataset.level = "1";
          break;
        case "h2":
          link.dataset.level = "2";
          break;
        case "h3":
          link.dataset.level = "3";
          break;
        case "h4":
          link.dataset.level = "4";
          break;
      }
      link.title = element.innerText;
      let text = element.innerText;
      if (text.length > charLimit) text = text.slice(0, charLimit) + "...";
      link.innerHTML = text;
      link.href = "#" + element.id;
      link.addEventListener("click", onLinkClick);

      // attach link
      panel.querySelector("#toc_list").appendChild(link);
    }
  }

  // when panel is clicked
  function onPanelClick(event) {
    // stop click from propagating to window/document and closing panel
    event.stopPropagation();
  }

  // when header itself is clicked
  function onHeaderClick(event) {
    togglePanel();
  }

  // when button is clicked
  function onButtonClick(event) {
    togglePanel();
    // stop header underneath button from also being clicked
    event.stopPropagation();
  }

  // when link is clicked
  function onLinkClick(event) {
    if (clickClose === "true" || (clickClose === "auto" && isSmallScreen()))
      closePanel();
    else openPanel();
  }

  // open panel if closed, close if opened
  function togglePanel() {
    const panel = document.getElementById("toc_panel");
    if (!panel) return;

    if (panel.dataset.open === "true") closePanel();
    else openPanel();
  }

  // open panel
  function openPanel() {
    const panel = document.getElementById("toc_panel");
    if (panel) panel.dataset.open = "true";
  }

  // close panel
  function closePanel() {
    const panel = document.getElementById("toc_panel");
    if (panel) panel.dataset.open = "false";
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- th list icon -->

<template class="icon_th_list">
  <!-- modified from: https://fontawesome.com/icons/th-list -->
  <svg width="16" height="16" viewBox="0 0 512 512" tabindex="-1">
    <path
      fill="currentColor"
      d="M96 96c0 26.51-21.49 48-48 48S0 122.51 0 96s21.49-48 48-48 48 21.49 48 48zM48 208c-26.51 0-48 21.49-48 48s21.49 48 48 48 48-21.49 48-48-21.49-48-48-48zm0 160c-26.51 0-48 21.49-48 48s21.49 48 48 48 48-21.49 48-48-21.49-48-48-48zm96-236h352c8.837 0 16-7.163 16-16V76c0-8.837-7.163-16-16-16H144c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16zm0 160h352c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H144c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16zm0 160h352c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H144c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16z"
      tabindex="-1"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* toc panel */
    #toc_panel {
      box-sizing: border-box;
      position: fixed;
      top: 0;
      left: 0;
      background: #ffffff;
      box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
      z-index: 2;
    }

    /* toc panel when closed */
    #toc_panel[data-open="false"] {
      min-width: 60px;
      width: 60px;
      height: 60px;
      border-right: solid 1px #bdbdbd;
      border-bottom: solid 1px #bdbdbd;
    }

    /* toc panel when open */
    #toc_panel[data-open="true"] {
      min-width: 260px;
      max-width: 480px;
      /* keep panel edge consistent distance away from "page" edge */
      width: calc(((100vw - 8.5in) / 2) - 30px - 40px);
      bottom: 0;
      border-right: solid 1px #bdbdbd;
    }

    /* toc panel header */
    #toc_header {
      box-sizing: border-box;
      display: flex;
      flex-direction: row;
      align-items: center;
      height: 60px;
      margin: 0;
      padding: 20px;
    }

    /* toc panel header when hovered */
    #toc_header:hover {
      cursor: pointer;
    }

    /* toc panel header when panel open */
    #toc_panel[data-open="true"] > #toc_header {
      border-bottom: solid 1px #bdbdbd;
    }

    /* toc open/close header button */
    #toc_button {
      margin-right: 20px;
    }

    /* hide toc list and header text when closed */
    #toc_panel[data-open="false"] > #toc_header > *:not(#toc_button),
    #toc_panel[data-open="false"] > #toc_list {
      display: none;
    }

    /* toc list of entries */
    #toc_list {
      box-sizing: border-box;
      width: 100%;
      padding: 20px;
      position: absolute;
      top: calc(60px + 1px);
      bottom: 0;
      overflow: auto;
    }

    /* toc entry, link to section in document */
    .toc_link {
      display: block;
      padding: 5px;
      position: relative;
      font-weight: 600;
      text-decoration: none;
    }

    /* toc entry when hovered or when "viewed" */
    .toc_link:hover,
    .toc_link[data-viewing="true"] {
      background: #f5f5f5;
    }

    /* toc entry, level 1 indentation */
    .toc_link[data-level="1"] {
      margin-left: 0;
    }

    /* toc entry, level 2 indentation */
    .toc_link[data-level="2"] {
      margin-left: 20px;
    }

    /* toc entry, level 3 indentation */
    .toc_link[data-level="3"] {
      margin-left: 40px;
    }

    /* toc entry, level 4 indentation */
    .toc_link[data-level="4"] {
      margin-left: 60px;
    }

    /* toc entry bullets */
    #toc_panel[data-bullets="true"] .toc_link[data-level]:before {
      position: absolute;
      left: -15px;
      top: -1px;
      font-size: 1.5em;
    }

    /* toc entry, level 2 bullet */
    #toc_panel[data-bullets="true"] .toc_link[data-level="2"]:before {
      content: "\2022";
    }

    /* toc entry, level 3 bullet */
    #toc_panel[data-bullets="true"] .toc_link[data-level="3"]:before {
      content: "\25AB";
    }

    /* toc entry, level 4 bullet */
    #toc_panel[data-bullets="true"] .toc_link[data-level="4"]:before {
      content: "-";
    }
  }

  /* when on screen < 8.5in wide */
  @media only screen and (max-width: 8.5in) {
    /* push <body> ("page") element down to make room for toc icon */
    .toc_body_nudge {
      padding-top: 60px;
    }

    /* toc icon when panel closed and not hovered */
    #toc_panel[data-open="false"]:not(:hover) {
      background: rgba(255, 255, 255, 0.75);
    }
  }

  /* always hide toc panel on print */
  @media only print {
    #toc_panel {
      display: none;
    }
  }
</style>
<!-- 
  Tooltips Plugin

  Makes it such that when the user hovers or focuses a link to a citation or
  figure, a tooltip appears with a preview of the reference content, along with
  arrows to navigate between instances of the same reference in the document.
-->

<script type="module">
  // whether user must click off to close tooltip instead of just un-hovering
  const clickClose = "false";
  // delay (in ms) between opening and closing tooltip
  const delay = "100";

  // start script
  function start() {
    const links = getLinks();
    for (const link of links) {
      // attach hover and focus listeners to link
      link.addEventListener("mouseover", onLinkHover);
      link.addEventListener("mouseleave", onLinkUnhover);
      link.addEventListener("focus", onLinkFocus);
      link.addEventListener("touchend", onLinkTouch);
    }

    // attach mouse, key, and resize listeners to window
    window.addEventListener("mousedown", onClick);
    window.addEventListener("touchstart", onClick);
    window.addEventListener("keyup", onKeyUp);
    window.addEventListener("resize", onResize);
  }

  // when link is hovered
  function onLinkHover() {
    // function to open tooltip
    const delayOpenTooltip = function () {
      openTooltip(this);
    }.bind(this);

    // run open function after delay
    this.openTooltipTimer = window.setTimeout(delayOpenTooltip, delay);
  }

  // when mouse leaves link
  function onLinkUnhover() {
    // cancel opening tooltip
    window.clearTimeout(this.openTooltipTimer);

    // don't close on unhover if option specifies
    if (clickClose === "true") return;

    // function to close tooltip
    const delayCloseTooltip = function () {
      // if tooltip open and if mouse isn't over tooltip, close
      const tooltip = document.getElementById("tooltip");
      if (tooltip && !tooltip.matches(":hover")) closeTooltip();
    };

    // run close function after delay
    this.closeTooltipTimer = window.setTimeout(delayCloseTooltip, delay);
  }

  // when link is focused (tabbed to)
  function onLinkFocus(event) {
    openTooltip(this);
  }

  // when link is touched on touch screen
  function onLinkTouch(event) {
    // attempt to force hover state on first tap always, and trigger
    // regular link click (and navigation) on second tap
    if (event.target === document.activeElement) event.target.click();
    else {
      document.activeElement.blur();
      event.target.focus();
    }
    if (event.cancelable) event.preventDefault();
    event.stopPropagation();
    return false;
  }

  // when mouse is clicked anywhere in window
  function onClick(event) {
    closeTooltip();
  }

  // when key pressed
  function onKeyUp(event) {
    if (!event || !event.key) return;

    switch (event.key) {
      // trigger click of prev button
      case "ArrowLeft":
        const prevButton = document.getElementById("tooltip_prev_button");
        if (prevButton) prevButton.click();
        break;
      // trigger click of next button
      case "ArrowRight":
        const nextButton = document.getElementById("tooltip_next_button");
        if (nextButton) nextButton.click();
        break;
      // close on esc
      case "Escape":
        closeTooltip();
        break;
    }
  }

  // when window is resized or zoomed
  function onResize() {
    closeTooltip();
  }

  // get all links of types we wish to handle
  function getLinks() {
    const queries = [];
    // exclude buttons, anchor links, toc links, etc
    const exclude =
      ":not(.button):not(.icon_button):not(.anchor):not(.toc_link)";
    queries.push('a[href^="#ref-"]' + exclude); // citation links
    queries.push('a[href^="#fig:"]' + exclude); // figure links
    const query = queries.join(", ");
    return document.querySelectorAll(query);
  }

  // get links with same target, get index of link in set, get total
  // same links
  function getSameLinks(link) {
    const sameLinks = [];
    const links = getLinks();
    for (const otherLink of links) {
      if (otherLink.getAttribute("href") === link.getAttribute("href"))
        sameLinks.push(otherLink);
    }

    return {
      elements: sameLinks,
      index: sameLinks.indexOf(link),
      total: sameLinks.length,
    };
  }

  // open tooltip
  function openTooltip(link) {
    // delete tooltip if it exists, start fresh
    closeTooltip();

    // make tooltip element
    const tooltip = makeTooltip(link);

    // if source couldn't be found and tooltip not made, exit
    if (!tooltip) return;

    // make navbar elements
    const navBar = makeNavBar(link);
    if (navBar) tooltip.firstElementChild.appendChild(navBar);

    // attach tooltip to page
    document.body.appendChild(tooltip);

    // position tooltip
    const position = function () {
      positionTooltip(link);
    };
    position();

    // if tooltip contains images, position again after they've loaded
    const imgs = tooltip.querySelectorAll("img");
    for (const img of imgs) img.addEventListener("load", position);
  }

  // close (delete) tooltip
  function closeTooltip() {
    const tooltip = document.getElementById("tooltip");
    if (tooltip) tooltip.remove();
  }

  // make tooltip
  function makeTooltip(link) {
    // get target element that link points to
    const source = getSource(link);

    // if source can't be found, exit
    if (!source) return;

    // create new tooltip
    const tooltip = document.createElement("div");
    tooltip.id = "tooltip";
    const tooltipContent = document.createElement("div");
    tooltipContent.id = "tooltip_content";
    tooltip.appendChild(tooltipContent);

    // make copy of source node and put in tooltip
    const sourceCopy = makeCopy(source);
    tooltipContent.appendChild(sourceCopy);

    // attach mouse event listeners
    tooltip.addEventListener("click", onTooltipClick);
    tooltip.addEventListener("mousedown", onTooltipClick);
    tooltip.addEventListener("touchstart", onTooltipClick);
    tooltip.addEventListener("mouseleave", onTooltipUnhover);

    // (for interaction with lightbox plugin)
    // transfer click on tooltip copied img to original img
    const sourceImg = source.querySelector("img");
    const sourceCopyImg = sourceCopy.querySelector("img");
    if (sourceImg && sourceCopyImg) {
      const clickImg = function () {
        sourceImg.click();
        closeTooltip();
      };
      sourceCopyImg.addEventListener("click", clickImg);
    }

    return tooltip;
  }

  // make carbon copy of html dom element
  function makeCopy(source) {
    const sourceCopy = source.cloneNode(true);

    // delete elements marked with ignore (eg anchor and jump buttons)
    const deleteFromCopy = sourceCopy.querySelectorAll('[data-ignore="true"]');
    for (const element of deleteFromCopy) element.remove();

    // delete certain element attributes
    const attributes = [
      "id",
      "data-collapsed",
      "data-selected",
      "data-highlighted",
      "data-glow",
      "class",
    ];
    for (const attribute of attributes) {
      sourceCopy.removeAttribute(attribute);
      const elements = sourceCopy.querySelectorAll("[" + attribute + "]");
      for (const element of elements) element.removeAttribute(attribute);
    }

    return sourceCopy;
  }

  // when tooltip is clicked
  function onTooltipClick(event) {
    // when user clicks on tooltip, stop click from transferring
    // outside of tooltip (eg, click off to close tooltip, or eg click
    // off to unhighlight same refs)
    event.stopPropagation();
  }

  // when tooltip is unhovered
  function onTooltipUnhover(event) {
    if (clickClose === "true") return;

    // make sure new mouse/touch/focus no longer over tooltip or any
    // element within it
    const tooltip = document.getElementById("tooltip");
    if (!tooltip) return;
    if (this.contains(event.relatedTarget)) return;

    closeTooltip();
  }

  // make nav bar to go betwen prev/next instances of same reference
  function makeNavBar(link) {
    // find other links to the same source
    const sameLinks = getSameLinks(link);

    // don't show nav bar when singular reference
    if (sameLinks.total <= 1) return;

    // find prev/next links with same target
    const prevLink = getPrevLink(link, sameLinks);
    const nextLink = getNextLink(link, sameLinks);

    // create nav bar
    const navBar = document.createElement("div");
    navBar.id = "tooltip_nav_bar";
    const text = sameLinks.index + 1 + " of " + sameLinks.total;

    // create nav bar prev/next buttons
    const prevButton = document.createElement("button");
    const nextButton = document.createElement("button");
    prevButton.id = "tooltip_prev_button";
    nextButton.id = "tooltip_next_button";
    prevButton.title =
      "Jump to the previous occurence of this item in the document [←]";
    nextButton.title =
      "Jump to the next occurence of this item in the document [→]";
    prevButton.classList.add("icon_button");
    nextButton.classList.add("icon_button");
    prevButton.innerHTML = document.querySelector(".icon_caret_left").innerHTML;
    nextButton.innerHTML =
      document.querySelector(".icon_caret_right").innerHTML;
    navBar.appendChild(prevButton);
    navBar.appendChild(document.createTextNode(text));
    navBar.appendChild(nextButton);

    // attach click listeners to buttons
    prevButton.addEventListener("click", function () {
      onPrevNextClick(link, prevLink);
    });
    nextButton.addEventListener("click", function () {
      onPrevNextClick(link, nextLink);
    });

    return navBar;
  }

  // get previous link with same target
  function getPrevLink(link, sameLinks) {
    if (!sameLinks) sameLinks = getSameLinks(link);
    // wrap index to other side if < 1
    let index;
    if (sameLinks.index - 1 >= 0) index = sameLinks.index - 1;
    else index = sameLinks.total - 1;
    return sameLinks.elements[index];
  }

  // get next link with same target
  function getNextLink(link, sameLinks) {
    if (!sameLinks) sameLinks = getSameLinks(link);
    // wrap index to other side if > total
    let index;
    if (sameLinks.index + 1 <= sameLinks.total - 1) index = sameLinks.index + 1;
    else index = 0;
    return sameLinks.elements[index];
  }

  // get element that is target of link or url hash
  function getSource(link) {
    const hash = link ? link.hash : window.location.hash;
    const id = hash.slice(1);
    let target = document.querySelector('[id="' + id + '"]');
    if (!target) return;

    // if ref or figure, modify target to get expected element
    if (id.indexOf("ref-") === 0) target = target.querySelector(":nth-child(2)");
    else if (id.indexOf("fig:") === 0) target = target.querySelector("figure");

    return target;
  }

  // when prev/next arrow button is clicked
  function onPrevNextClick(link, prevNextLink) {
    if (link && prevNextLink)
      goToElement(prevNextLink, window.innerHeight * 0.5);
  }

  // scroll to and focus element
  function goToElement(element, offset) {
    // expand accordion section if collapsed
    expandElement(element);
    const y =
      getRectInView(element).top -
      getRectInView(document.documentElement).top -
      (offset || 0);
    // trigger any function listening for "onscroll" event
    window.dispatchEvent(new Event("scroll"));
    window.scrollTo(0, y);
    document.activeElement.blur();
    element.focus();
  }

  // determine position to place tooltip based on link position in
  // viewport and tooltip size
  function positionTooltip(link, left, top) {
    const tooltipElement = document.getElementById("tooltip");
    if (!tooltipElement) return;

    // get convenient vars for position/dimensions of
    // link/tooltip/page/view
    link = getRectInPage(link);
    const tooltip = getRectInPage(tooltipElement);
    const view = getRectInPage();

    // horizontal positioning
    if (left)
      // use explicit value
      left = left;
    else if (link.left + tooltip.width < view.right)
      // fit tooltip to right of link
      left = link.left;
    else if (link.right - tooltip.width > view.left)
      // fit tooltip to left of link
      left = link.right - tooltip.width;
    // center tooltip in view
    else left = (view.right - view.left) / 2 - tooltip.width / 2;

    // vertical positioning
    if (top)
      // use explicit value
      top = top;
    else if (link.top - tooltip.height > view.top)
      // fit tooltip above link
      top = link.top - tooltip.height;
    else if (link.bottom + tooltip.height < view.bottom)
      // fit tooltip below link
      top = link.bottom;
    else {
      // center tooltip in view
      top = view.top + view.height / 2 - tooltip.height / 2;
      // nudge off of link to left/right if possible
      if (link.right + tooltip.width < view.right) left = link.right;
      else if (link.left - tooltip.width > view.left)
        left = link.left - tooltip.width;
    }

    tooltipElement.style.left = left + "px";
    tooltipElement.style.top = top + "px";
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- caret left icon -->

<template class="icon_caret_left">
  <!-- modified from: https://fontawesome.com/icons/caret-left -->
  <svg width="16" height="16" viewBox="0 0 192 512">
    <path
      fill="currentColor"
      d="M192 127.338v257.324c0 17.818-21.543 26.741-34.142 14.142L29.196 270.142c-7.81-7.81-7.81-20.474 0-28.284l128.662-128.662c12.599-12.6 34.142-3.676 34.142 14.142z"
    ></path>
  </svg>
</template>

<!-- caret right icon -->

<template class="icon_caret_right">
  <!-- modified from: https://fontawesome.com/icons/caret-right -->
  <svg width="16" height="16" viewBox="0 0 192 512">
    <path
      fill="currentColor"
      d="M0 384.662V127.338c0-17.818 21.543-26.741 34.142-14.142l128.662 128.662c7.81 7.81 7.81 20.474 0 28.284L34.142 398.804C21.543 411.404 0 402.48 0 384.662z"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* tooltip container */
    #tooltip {
      position: absolute;
      width: 50%;
      min-width: 240px;
      max-width: 75%;
      z-index: 1;
    }

    /* tooltip content */
    #tooltip_content {
      margin-bottom: 5px;
      padding: 20px;
      border-radius: 5px;
      border: solid 1px #bdbdbd;
      box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
      background: #ffffff;
      overflow-wrap: break-word;
    }

    /* tooltip copy of paragraphs and figures */
    #tooltip_content > p,
    #tooltip_content > figure {
      margin: 0;
      max-height: 320px;
      overflow-y: auto;
    }

    /* tooltip copy of <img> */
    #tooltip_content > figure > img,
    #tooltip_content > figure > svg {
      max-height: 260px;
    }

    /* navigation bar */
    #tooltip_nav_bar {
      margin-top: 10px;
      text-align: center;
    }

    /* navigation bar previous/next buton */
    #tooltip_nav_bar > .icon_button {
      position: relative;
      top: 3px;
    }

    /* navigation bar previous button */
    #tooltip_nav_bar > .icon_button:first-of-type {
      margin-right: 5px;
    }

    /* navigation bar next button */
    #tooltip_nav_bar > .icon_button:last-of-type {
      margin-left: 5px;
    }
  }

  /* always hide tooltip on print */
  @media only print {
    #tooltip {
      display: none;
    }
  }
</style>
<!--
  Analytics Plugin (third-party) 
  
  Copy and paste code from Google Analytics or similar service here.
-->
<!-- 
  Annotations Plugin

  Allows public annotation of the  manuscript. See https://web.hypothes.is/.
-->

<script type="module">
  // configuration
  window.hypothesisConfig = function () {
    return {
      branding: {
        accentColor: "#2196f3",
        appBackgroundColor: "#f8f8f8",
        ctaBackgroundColor: "#f8f8f8",
        ctaTextColor: "#000000",
        selectionFontFamily: "Open Sans, Helvetica, sans serif",
        annotationFontFamily: "Open Sans, Helvetica, sans serif",
      },
    };
  };

  // hypothesis client script
  const embed = "https://hypothes.is/embed.js";
  // hypothesis annotation count query url
  const query = "https://api.hypothes.is/api/search?limit=0&url=";

  // start script
  function start() {
    const button = makeButton();
    document.body.insertBefore(button, document.body.firstChild);
    insertCount(button);
  }

  // make button
  function makeButton() {
    // create button
    const button = document.createElement("button");
    button.id = "hypothesis_button";
    button.innerHTML = document.querySelector(".icon_hypothesis").innerHTML;
    button.title = "Hypothesis annotations";
    button.classList.add("icon_button");

    function onClick(event) {
      onButtonClick(event, button);
    }

    // attach click listeners
    button.addEventListener("click", onClick);

    return button;
  }

  // insert annotations count
  async function insertCount(button) {
    // get annotation count from Hypothesis based on url
    let count = "-";
    try {
      const canonical = document.querySelector('link[rel="canonical"]');
      const location = window.location;
      const url = encodeURIComponent((canonical || location).href);
      const response = await fetch(query + url);
      const json = await response.json();
      count = json.total || "-";
    } catch (error) {
      console.log(error);
    }

    // put count into button
    const counter = document.createElement("span");
    counter.id = "hypothesis_count";
    counter.innerHTML = count;
    button.title = "View " + count + " Hypothesis annotations";
    button.append(counter);
  }

  // when button is clicked
  function onButtonClick(event, button) {
    const script = document.createElement("script");
    script.src = embed;
    document.body.append(script);
    button.remove();
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- hypothesis icon -->

<template class="icon_hypothesis">
  <!-- modified from: https://simpleicons.org/icons/hypothesis.svg / https://git.io/Jf1VB -->
  <svg width="16" height="16" viewBox="0 0 24 24" tabindex="-1">
    <path
      fill="currentColor"
      d="M3.43 0C2.5 0 1.72 .768 1.72 1.72V18.86C1.72 19.8 2.5 20.57 3.43 20.57H9.38L12 24L14.62 20.57H20.57C21.5 20.57 22.29 19.8 22.29 18.86V1.72C22.29 .77 21.5 0 20.57 0H3.43M5.14 3.43H7.72V9.43S8.58 7.72 10.28 7.72C12 7.72 13.74 8.57 13.74 11.24V17.14H11.16V12C11.16 10.61 10.28 10.07 9.43 10.29C8.57 10.5 7.72 11.41 7.72 13.29V17.14H5.14V3.43M18 13.72C18.95 13.72 19.72 14.5 19.72 15.42A1.71 1.71 0 0 1 18 17.13A1.71 1.71 0 0 1 16.29 15.42C16.29 14.5 17.05 13.71 18 13.71Z"
      tabindex="-1"
    ></path>
  </svg>
</template>

<style>
  /* hypothesis activation button */
  #hypothesis_button {
    box-sizing: border-box;
    position: fixed;
    top: 0;
    right: 0;
    width: 60px;
    height: 60px;
    background: #ffffff;
    border-radius: 0;
    border-left: solid 1px #bdbdbd;
    border-bottom: solid 1px #bdbdbd;
    box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
    z-index: 2;
  }

  /* hypothesis button svg */
  #hypothesis_button > svg {
    position: relative;
    top: -4px;
  }

  /* hypothesis annotation count */
  #hypothesis_count {
    position: absolute;
    left: 0;
    right: 0;
    bottom: 5px;
  }

  /* side panel */
  .annotator-frame {
    width: 280px !important;
  }

  /* match highlight color to rest of theme */
  .annotator-highlights-always-on .annotator-hl {
    background-color: #ffeb3b !important;
  }

  /* match focused color to rest of theme */
  .annotator-hl.annotator-hl-focused {
    background-color: #ff8a65 !important;
  }

  /* match bucket bar color to rest of theme */
  .annotator-bucket-bar {
    background: #f5f5f5 !important;
  }

  /* always hide button, toolbar, and tooltip on print */
  @media only print {
    #hypothesis_button {
      display: none;
    }

    .annotator-frame {
      display: none !important;
    }

    hypothesis-adder {
      display: none !important;
    }
  }
</style>
<!-- 
  Mathjax Plugin (third-party) 

  Allows the proper rendering of math/equations written in LaTeX.
  See https://www.mathjax.org/.
-->

<script type="text/x-mathjax-config">
  // configuration
  MathJax.Hub.Config({
    "CommonHTML": { linebreaks: { automatic: true } },
    "HTML-CSS": { linebreaks: { automatic: true } },
    "SVG": { linebreaks: { automatic: true } },
    "fast-preview": { disabled: true }
  });
</script>

<script
  src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_CHTML"
  integrity="sha512-0doc9hKxR3PYwso42RD1p5ySZpzzuDiOwMrdCEh2WdJZCjcmFKc/wEnL+z8fBQrnHoiNWbo+3fiGkOYXBdQp4A=="
  crossorigin="anonymous"
></script>

<style>
  /* mathjax containers */
  .math.display > span:not(.MathJax_Preview) {
    /* turn inline element (no dimensions) into block (allows fixed width and thus scrolling) */
    display: flex !important;
    overflow-x: auto !important;
    overflow-y: hidden !important;
    justify-content: center;
    align-items: center;
    margin: 0 !important;
  }

  /* right click menu */
  .MathJax_Menu {
    border-radius: 5px !important;
    border: solid 1px #bdbdbd !important;
    box-shadow: none !important;
  }

  /* equation auto-number */
  span[id^="eq:"] > span.math.display + span {
    font-weight: 600;
  }

  /* equation */
  span[id^="eq:"] > span.math.display > span {
    /* nudge to make room for equation auto-number and anchor */
    margin-right: 60px !important;
  }
</style>
</body>
</html>
